{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": [],
      "toc_visible": true
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "source": [
        "# Homework 4: Named Entity Recognization[5 points]\n",
        "\n",
        "\n",
        "### HIDS 7006, Spring 2024\n",
        "\n",
        "### Due: Wednesday, May 1, 2024 11:59 pm E.T.\n",
        "\n",
        "This homework will test your knowledge about extracting named entities from biomedical text (PubMed articles) that you learned in lecture 12. Fill in the code/answers for the questions as indicated below.\n",
        "\n",
        "Please edit this document directly using Jupyter or Google Colab and answer each of the questions in-line as indicated.\n",
        "\n",
        "Turn in a single document i.e. the Notebook showing all of your code and output for the entire assignment, with each question clearly demarcated. Submit your completed assignment through Canvas. The notebook can be downloaded by clicking the `File` option (top left) and clicking `Download .ipynb` in the drop down menu."
      ],
      "metadata": {
        "id": "R4D4aJNiTIxZ"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "## 1. Biomedical Named Entity Extraction  [5 points]"
      ],
      "metadata": {
        "id": "mJPyEvUWUdqK"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "**Dataset**\n",
        "\n",
        "You have been provided with a list of 100 PMIDs (PubMed Identifiers) in a CSV. You need to randomly select 25 pmids from the CSV files and extract the biomedical entities (genes, disease, mutations, and chemicals) mentioned in the selected 25 pmids using PubTator APIs.\n",
        "\n"
      ],
      "metadata": {
        "id": "VpR9YlSjczsX"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "**Steps**\n",
        "\n",
        "1. Read the input pmids.csv files (which contains 100 pmids) using Pandas or Python file IO operations.\n",
        "\n",
        "2. Randomly select 25 pmids from the input file and store them in a python list. Hint: If you are using Pandas to read the input file, you can use the function `sample` to randomly select a subset of rows in dataframe.\n",
        "\n",
        "    **Note:** The 25 pmids should be selected randomly and not the first/last 25\n",
        "    \n",
        "    `sample` function documentation: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.sample.html\n",
        "\n",
        "3. Use Pubtator APIs to create a dataframe of the different biomedical entities mentioned in the selected 25 pmids. The dataframe should **atleast** contain the following columns: `[\"pmid\", \"entity_mention\", \"entity_type\", \"entity_id\"]`\n",
        "\n",
        "    **Hint:** Please refer to the section \"*Extracting biomedical entities from PubMed abstracts*\" notebook from lecture 12 (`12_named_entity_recognization_solved`) for details on how to use the APIs and create the dataframe of entities.\n",
        "\n",
        "4. What are the top five diseases (by count) detected by Pubtator in your list of 25 pmids? Based on these top five diseases, what disease topic or query do you think was used to obtain the initial list of pmids.\n",
        "\n",
        "\n",
        "5. What are the top five genes (by count) detected by Pubtator in your list of 25 pmids?"
      ],
      "metadata": {
        "id": "tQ4Q3WlrgvqM"
      }
    },
    {
      "cell_type": "markdown",
      "source": [],
      "metadata": {
        "id": "sMAsuY7M7pph"
      }
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "Vq3zeMxiTFSo"
      },
      "outputs": [],
      "source": [
        "##Code here; Use as many code cells as required"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# 1) Read the input pmids.csv files (which contains 100 pmids) using Pandas or Python file IO operations."
      ],
      "metadata": {
        "id": "X_NLS5ha7qYE"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "import pandas as pd"
      ],
      "metadata": {
        "id": "3f3teYbR7tqq"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "from google.colab import drive\n",
        "drive.mount('/content/drive')"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "PfWjq8MP98eE",
        "outputId": "4e17e8f4-4e3b-415a-fb30-7d97808b5a3d"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Mounted at /content/drive\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "pmids=pd.read_csv('/content/drive/MyDrive/HIDS_7006/Assignment/pmids.csv')\n",
        "pmids"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 424
        },
        "id": "-s2PuzC5-CPT",
        "outputId": "d54ad3f2-3ce6-4c53-d46c-1ede565a264e"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "        PMID\n",
              "0   34534430\n",
              "1   32846060\n",
              "2   30980071\n",
              "3   30718506\n",
              "4   30413378\n",
              "..       ...\n",
              "95  25668228\n",
              "96  25667280\n",
              "97  25470694\n",
              "98  25393796\n",
              "99  25351743\n",
              "\n",
              "[100 rows x 1 columns]"
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-98e7dde5-60d3-472a-8fd4-67d62913a3b5\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>PMID</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>34534430</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>32846060</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>30980071</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>30718506</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>30413378</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>...</th>\n",
              "      <td>...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>95</th>\n",
              "      <td>25668228</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>96</th>\n",
              "      <td>25667280</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>97</th>\n",
              "      <td>25470694</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>98</th>\n",
              "      <td>25393796</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>99</th>\n",
              "      <td>25351743</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>100 rows Ã— 1 columns</p>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-98e7dde5-60d3-472a-8fd4-67d62913a3b5')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-98e7dde5-60d3-472a-8fd4-67d62913a3b5 button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-98e7dde5-60d3-472a-8fd4-67d62913a3b5');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-07f7f750-6a8d-4d15-b86a-907cbebb426c\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-07f7f750-6a8d-4d15-b86a-907cbebb426c')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-07f7f750-6a8d-4d15-b86a-907cbebb426c button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "variable_name": "pmids",
              "summary": "{\n  \"name\": \"pmids\",\n  \"rows\": 100,\n  \"fields\": [\n    {\n      \"column\": \"PMID\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 1500905,\n        \"min\": 25351743,\n        \"max\": 34534430,\n        \"num_unique_values\": 100,\n        \"samples\": [\n          26027660,\n          26771872,\n          26515464\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"
            }
          },
          "metadata": {},
          "execution_count": 3
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# 2) Randomly select 25 pmids from the input file and store them in a python list. Hint: If you are using Pandas to read the input file, you can use the function sample to randomly select a subset of rows in dataframe."
      ],
      "metadata": {
        "id": "_oWbER8B-Ouj"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "pmids_25=pmids.sample(n=25, random_state=42)\n",
        "pmids_25"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 677
        },
        "id": "xQG0EtqQ-L_3",
        "outputId": "fb56bfcf-af1c-434d-c68a-2a259bae9e79"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "        PMID\n",
              "83  26027660\n",
              "53  26771872\n",
              "70  26515464\n",
              "45  26899019\n",
              "44  26918046\n",
              "39  27022118\n",
              "22  27876675\n",
              "80  26124334\n",
              "10  28602779\n",
              "0   34534430\n",
              "18  28259530\n",
              "30  27577079\n",
              "73  26439803\n",
              "33  27315356\n",
              "90  25765070\n",
              "4   30413378\n",
              "76  26287849\n",
              "77  26209642\n",
              "12  28525386\n",
              "31  27544060\n",
              "55  26758680\n",
              "88  25891174\n",
              "26  27694386\n",
              "42  26938871\n",
              "69  26554404"
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-eee12e87-053d-43de-9ddd-1978b0060f5a\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>PMID</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>83</th>\n",
              "      <td>26027660</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>53</th>\n",
              "      <td>26771872</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>70</th>\n",
              "      <td>26515464</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>45</th>\n",
              "      <td>26899019</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>44</th>\n",
              "      <td>26918046</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>39</th>\n",
              "      <td>27022118</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>22</th>\n",
              "      <td>27876675</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>80</th>\n",
              "      <td>26124334</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>10</th>\n",
              "      <td>28602779</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>34534430</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>18</th>\n",
              "      <td>28259530</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>30</th>\n",
              "      <td>27577079</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>73</th>\n",
              "      <td>26439803</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>33</th>\n",
              "      <td>27315356</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>90</th>\n",
              "      <td>25765070</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>30413378</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>76</th>\n",
              "      <td>26287849</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>77</th>\n",
              "      <td>26209642</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>12</th>\n",
              "      <td>28525386</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>31</th>\n",
              "      <td>27544060</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>55</th>\n",
              "      <td>26758680</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>88</th>\n",
              "      <td>25891174</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>26</th>\n",
              "      <td>27694386</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>42</th>\n",
              "      <td>26938871</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>69</th>\n",
              "      <td>26554404</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-eee12e87-053d-43de-9ddd-1978b0060f5a')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-eee12e87-053d-43de-9ddd-1978b0060f5a button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-eee12e87-053d-43de-9ddd-1978b0060f5a');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-f198e295-b896-47be-9ab9-000847b0c27e\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-f198e295-b896-47be-9ab9-000847b0c27e')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-f198e295-b896-47be-9ab9-000847b0c27e button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "variable_name": "pmids_25",
              "summary": "{\n  \"name\": \"pmids_25\",\n  \"rows\": 25,\n  \"fields\": [\n    {\n      \"column\": \"PMID\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 1814006,\n        \"min\": 25765070,\n        \"max\": 34534430,\n        \"num_unique_values\": 25,\n        \"samples\": [\n          28602779,\n          26287849,\n          26027660\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"
            }
          },
          "metadata": {},
          "execution_count": 4
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# 3) Use Pubtator APIs to create a dataframe of the different biomedical entities mentioned in the selected 25 pmids. The dataframe should atleast contain the following columns: [\"pmid\", \"entity_mention\", \"entity_type\", \"entity_id\"]\n",
        "\n",
        "Hint: Please refer to the section \"Extracting biomedical entities from PubMed abstracts\" notebook from lecture 12 (12_named_entity_recognization_solved) for details on how to use the APIs and create the dataframe of entities."
      ],
      "metadata": {
        "id": "b0hpyZoU-5hb"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "def process_pmids(input_pmids, format, bioconcept):\n",
        "  json = {}\n",
        "\n",
        "  #load pmids\n",
        "  json = {\"pmids\": [pmid.strip() for pmid in input_pmids]}\n",
        "\n",
        "  #load bioconcepts\n",
        "  if bioconcept != \"\":\n",
        "    json[\"concepts\"]=bioconcept.split(\",\")\n",
        "\n",
        "  ###process restful API\n",
        "  r = requests.post(\"https://www.ncbi.nlm.nih.gov/research/pubtator-api/publications/export/\"+format , json = json)\n",
        "\n",
        "  if r.status_code != 200:\n",
        "    print (\"[Error]: HTTP code \"+ str(r.status_code))\n",
        "  else:\n",
        "    return r.text"
      ],
      "metadata": {
        "id": "5rp4WWW--y0D"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "selected_pmids = pmids_25[\"PMID\"].tolist()\n",
        "selected_pmids"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Pl1KhmbcEbsv",
        "outputId": "40452bbb-fdb4-49bc-b403-c94f35fbbb2d"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "[26027660,\n",
              " 26771872,\n",
              " 26515464,\n",
              " 26899019,\n",
              " 26918046,\n",
              " 27022118,\n",
              " 27876675,\n",
              " 26124334,\n",
              " 28602779,\n",
              " 34534430,\n",
              " 28259530,\n",
              " 27577079,\n",
              " 26439803,\n",
              " 27315356,\n",
              " 25765070,\n",
              " 30413378,\n",
              " 26287849,\n",
              " 26209642,\n",
              " 28525386,\n",
              " 27544060,\n",
              " 26758680,\n",
              " 25891174,\n",
              " 27694386,\n",
              " 26938871,\n",
              " 26554404]"
            ]
          },
          "metadata": {},
          "execution_count": 34
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "selected_pmids_str=[str(pmid) for pmid in selected_pmids]"
      ],
      "metadata": {
        "id": "QaOrAUfLOfYe"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "import requests"
      ],
      "metadata": {
        "id": "LBOYEW95FLOQ"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "result = process_pmids(selected_pmids_str, \"pubtator\", \"\")"
      ],
      "metadata": {
        "id": "MN9FbiwJEwvK"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "result"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 209
        },
        "id": "EHFSzXx5FQw8",
        "outputId": "cdbe7d8a-1501-4c6e-dc56-f844563d728e"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "'25765070|t|Cancer immunology. Mutational landscape determines sensitivity to PD-1 blockade in non-small cell lung cancer.\\n25765070|a|Immune checkpoint inhibitors, which unleash a patient\\'s own T cells to kill tumors, are revolutionizing cancer treatment. To unravel the genomic determinants of response to this therapy, we used whole-exome sequencing of non-small cell lung cancers treated with pembrolizumab, an antibody targeting programmed cell death-1 (PD-1). In two independent cohorts, higher nonsynonymous mutation burden in tumors was associated with improved objective response, durable clinical benefit, and progression-free survival. Efficacy also correlated with the molecular smoking signature, higher neoantigen burden, and DNA repair pathway mutations; each factor was also associated with mutation burden. In one responder, neoantigen-specific CD8+ T cell responses paralleled tumor regression, suggesting that anti-PD-1 therapy enhances neoantigen-specific T cell reactivity. Our results suggest that the genomic landscape of lung cancers shapes response to anti-PD-1 therapy. \\n25765070\\t0\\t6\\tCancer\\tDisease\\tMESH:D009369\\n25765070\\t157\\t164\\tpatient\\tSpecies\\t9606\\n25765070\\t187\\t193\\ttumors\\tDisease\\tMESH:D009369\\n25765070\\t347\\t359\\tlung cancers\\tDisease\\tMESH:D008175\\n25765070\\t373\\t386\\tpembrolizumab\\tChemical\\tMESH:C582435\\n25765070\\t510\\t516\\ttumors\\tDisease\\tMESH:D009369\\n25765070\\t843\\t849\\tT cell\\tCellLine\\tT cell\\n25765070\\t871\\t876\\ttumor\\tDisease\\tMESH:D009369\\n25765070\\t952\\t958\\tT cell\\tCellLine\\tT cell\\n25765070\\t1021\\t1033\\tlung cancers\\tDisease\\tMESH:D008175\\n\\n25891174|t|Pembrolizumab for the treatment of non-small-cell lung cancer.\\n25891174|a|BACKGROUND: We assessed the efficacy and safety of programmed cell death 1 (PD-1) inhibition with pembrolizumab in patients with advanced non-small-cell lung cancer enrolled in a phase 1 study. We also sought to define and validate an expression level of the PD-1 ligand 1 (PD-L1) that is associated with the likelihood of clinical benefit. METHODS: We assigned 495 patients receiving pembrolizumab (at a dose of either 2 mg or 10 mg per kilogram of body weight every 3 weeks or 10 mg per kilogram every 2 weeks) to either a training group (182 patients) or a validation group (313 patients). We assessed PD-L1 expression in tumor samples using immunohistochemical analysis, with results reported as the percentage of neoplastic cells with staining for membranous PD-L1 (proportion score). Response was assessed every 9 weeks by central review. RESULTS: Common side effects that were attributed to pembrolizumab were fatigue, pruritus, and decreased appetite, with no clear difference according to dose or schedule. Among all the patients, the objective response rate was 19.4%, and the median duration of response was 12.5 months. The median duration of progression-free survival was 3.7 months, and the median duration of overall survival was 12.0 months. PD-L1 expression in at least 50% of tumor cells was selected as the cutoff from the training group. Among patients with a proportion score of at least 50% in the validation group, the response rate was 45.2%. Among all the patients with a proportion score of at least 50%, median progression-free survival was 6.3 months; median overall survival was not reached. CONCLUSIONS: Pembrolizumab had an acceptable side-effect profile and showed antitumor activity in patients with advanced non-small-cell lung cancer. PD-L1 expression in at least 50% of tumor cells correlated with improved efficacy of pembrolizumab. (Funded by Merck; KEYNOTE-001 ClinicalTrials.gov number, NCT01295827.).\\n25891174\\t0\\t13\\tPembrolizumab\\tChemical\\tMESH:C582435\\n25891174\\t50\\t61\\tlung cancer\\tDisease\\tMESH:D008175\\n25891174\\t114\\t137\\tprogrammed cell death 1\\tGene\\t5133\\n25891174\\t139\\t143\\tPD-1\\tGene\\t5133\\n25891174\\t161\\t174\\tpembrolizumab\\tChemical\\tMESH:C582435\\n25891174\\t178\\t186\\tpatients\\tSpecies\\t9606\\n25891174\\t216\\t227\\tlung cancer\\tDisease\\tMESH:D008175\\n25891174\\t322\\t335\\tPD-1 ligand 1\\tGene\\t29126\\n25891174\\t337\\t342\\tPD-L1\\tGene\\t29126\\n25891174\\t429\\t437\\tpatients\\tSpecies\\t9606\\n25891174\\t448\\t461\\tpembrolizumab\\tChemical\\tMESH:C582435\\n25891174\\t608\\t616\\tpatients\\tSpecies\\t9606\\n25891174\\t645\\t653\\tpatients\\tSpecies\\t9606\\n25891174\\t668\\t673\\tPD-L1\\tGene\\t29126\\n25891174\\t688\\t693\\ttumor\\tDisease\\tMESH:D009369\\n25891174\\t827\\t832\\tPD-L1\\tGene\\t29126\\n25891174\\t961\\t974\\tpembrolizumab\\tChemical\\tMESH:C582435\\n25891174\\t980\\t987\\tfatigue\\tDisease\\tMESH:D005221\\n25891174\\t989\\t997\\tpruritus\\tDisease\\tMESH:D011537\\n25891174\\t1093\\t1101\\tpatients\\tSpecies\\t9606\\n25891174\\t1321\\t1326\\tPD-L1\\tGene\\t29126\\n25891174\\t1357\\t1362\\ttumor\\tDisease\\tMESH:D009369\\n25891174\\t1427\\t1435\\tpatients\\tSpecies\\t9606\\n25891174\\t1544\\t1552\\tpatients\\tSpecies\\t9606\\n25891174\\t1697\\t1710\\tPembrolizumab\\tChemical\\tMESH:C582435\\n25891174\\t1782\\t1790\\tpatients\\tSpecies\\t9606\\n25891174\\t1820\\t1831\\tlung cancer\\tDisease\\tMESH:D008175\\n25891174\\t1833\\t1838\\tPD-L1\\tGene\\t29126\\n25891174\\t1869\\t1874\\ttumor\\tDisease\\tMESH:D009369\\n25891174\\t1918\\t1931\\tpembrolizumab\\tChemical\\tMESH:C582435\\n\\n26027660|t|LKB1 gene inactivation does not sensitize non-small cell lung cancer cells to mTOR inhibitors in vitro.\\n26027660|a|AIM: Previous study has shown that endometrial cancers with LKB1 inactivation are highly responsive to mTOR inhibitors. In this study we examined the effect of LKB1 gene status on mTOR inhibitor responses in non-small cell lung cancer (NSCLC) cells. METHODS: Lung cancer cell lines Calu-1, H460, H1299, H1792, and A549 were treated with the mTOR inhibitors rapamycin or everolimus (RAD001). The mTOR activity was evaluated by measuring the phosphorylation of 4EBP1 and S6K, the two primary mTOR substrates. Cells proliferation was measured by MTS or sulforhodamine B assays. RESULTS: The basal level of mTOR activity in LKB1 mutant A549 and H460 cells was significantly higher than that in LKB1 wild-type Calu-1 and H1792 cells. However, the LKB1 mutant A549 and H460 cells were not more sensitive to the mTOR inhibitors than the LKB1 wild-type Calu-1 and H1792 cells. Moreover, knockdown of LKB1 gene in H1299 cells did not increase the sensitivity to the mTOR inhibitors. Treatment with rapamycin or RAD001 significantly increased the phosphorylation of AKT in both LKB1 wild-type and LKB1 mutant NSCLC cells, which was attenuated by the PI3K inhibitor LY294002. Furthermore, RAD001 combined with LY294002 markedly enhanced the growth inhibition on LKB1 wild-type H1792 cells and LKB1 mutant A549 cells. CONCLUSION: LKB1 gene inactivation in NSCLC cells does not increase the sensitivity to the mTOR inhibitors. The negative feedback activation of AKT by mTOR inhibition may contribute to the resistance of NSCLC cells to mTOR inhibitors.\\n26027660\\t0\\t4\\tLKB1\\tGene\\t6794\\n26027660\\t57\\t68\\tlung cancer\\tDisease\\tMESH:D008175\\n26027660\\t78\\t82\\tmTOR\\tGene\\t2475\\n26027660\\t139\\t158\\tendometrial cancers\\tDisease\\tMESH:D016889\\n26027660\\t164\\t168\\tLKB1\\tGene\\t6794\\n26027660\\t207\\t211\\tmTOR\\tGene\\t2475\\n26027660\\t264\\t268\\tLKB1\\tGene\\t6794\\n26027660\\t284\\t288\\tmTOR\\tGene\\t2475\\n26027660\\t327\\t338\\tlung cancer\\tDisease\\tMESH:D008175\\n26027660\\t340\\t345\\tNSCLC\\tDisease\\tMESH:D002289\\n26027660\\t363\\t374\\tLung cancer\\tDisease\\tMESH:D008175\\n26027660\\t394\\t398\\tH460\\tCellLine\\tCVCL:0459\\n26027660\\t400\\t405\\tH1299\\tCellLine\\tCVCL:0060\\n26027660\\t407\\t412\\tH1792\\tCellLine\\tCVCL:1495\\n26027660\\t418\\t422\\tA549\\tCellLine\\tCVCL:0023\\n26027660\\t445\\t449\\tmTOR\\tGene\\t2475\\n26027660\\t461\\t470\\trapamycin\\tChemical\\tMESH:D020123\\n26027660\\t474\\t484\\teverolimus\\tChemical\\tMESH:D000068338\\n26027660\\t499\\t503\\tmTOR\\tGene\\t2475\\n26027660\\t563\\t568\\t4EBP1\\tGene\\t1978\\n26027660\\t573\\t576\\tS6K\\tGene\\t6198\\n26027660\\t594\\t598\\tmTOR\\tGene\\t2475\\n26027660\\t654\\t670\\tsulforhodamine B\\tChemical\\tMESH:C022027\\n26027660\\t707\\t711\\tmTOR\\tGene\\t2475\\n26027660\\t724\\t728\\tLKB1\\tGene\\t6794\\n26027660\\t736\\t740\\tA549\\tCellLine\\tCVCL:0023\\n26027660\\t745\\t749\\tH460\\tCellLine\\tCVCL:0459\\n26027660\\t794\\t798\\tLKB1\\tGene\\t6794\\n26027660\\t820\\t825\\tH1792\\tCellLine\\tCVCL:1495\\n26027660\\t846\\t850\\tLKB1\\tGene\\t6794\\n26027660\\t858\\t862\\tA549\\tCellLine\\tCVCL:0023\\n26027660\\t867\\t871\\tH460\\tCellLine\\tCVCL:0459\\n26027660\\t909\\t913\\tmTOR\\tGene\\t2475\\n26027660\\t934\\t938\\tLKB1\\tGene\\t6794\\n26027660\\t960\\t965\\tH1792\\tCellLine\\tCVCL:1495\\n26027660\\t996\\t1000\\tLKB1\\tGene\\t6794\\n26027660\\t1009\\t1014\\tH1299\\tCellLine\\tCVCL:0060\\n26027660\\t1061\\t1065\\tmTOR\\tGene\\t2475\\n26027660\\t1093\\t1102\\trapamycin\\tChemical\\tMESH:D020123\\n26027660\\t1106\\t1112\\tRAD001\\tChemical\\tMESH:D000068338\\n26027660\\t1160\\t1163\\tAKT\\tGene\\t207\\n26027660\\t1172\\t1176\\tLKB1\\tGene\\t6794\\n26027660\\t1191\\t1195\\tLKB1\\tGene\\t6794\\n26027660\\t1203\\t1208\\tNSCLC\\tDisease\\tMESH:D002289\\n26027660\\t1259\\t1267\\tLY294002\\tChemical\\tMESH:C085911\\n26027660\\t1303\\t1311\\tLY294002\\tChemical\\tMESH:C085911\\n26027660\\t1355\\t1359\\tLKB1\\tGene\\t6794\\n26027660\\t1370\\t1375\\tH1792\\tCellLine\\tCVCL:1495\\n26027660\\t1386\\t1390\\tLKB1\\tGene\\t6794\\n26027660\\t1398\\t1402\\tA549\\tCellLine\\tCVCL:0023\\n26027660\\t1422\\t1426\\tLKB1\\tGene\\t6794\\n26027660\\t1448\\t1453\\tNSCLC\\tDisease\\tMESH:D002289\\n26027660\\t1501\\t1505\\tmTOR\\tGene\\t2475\\n26027660\\t1554\\t1557\\tAKT\\tGene\\t207\\n26027660\\t1561\\t1565\\tmTOR\\tGene\\t2475\\n26027660\\t1613\\t1618\\tNSCLC\\tDisease\\tMESH:D002289\\n26027660\\t1628\\t1632\\tmTOR\\tGene\\t2475\\n\\n26124334|t|Effectiveness of Tyrosine Kinase Inhibitors in Japanese Patients with Non-small Cell Lung Cancer Harboring Minor Epidermal Growth Factor Receptor Mutations: Results from a Multicenter Retrospective Study (HANSHIN Oncology Group 0212).\\n26124334|a|AIM: Non-small cell lung cancer (NSCLC) with minor mutations in the epidermal growth factor receptor (EGFR) gene, except for the common 15 base-pair deletions in exon 19 and the L858R mutation in exon 21, is rare, and only few data exist on this patient population. The aim of the present study was to describe the clinical characteristics and to clarify the efficacy of EGFR-tyrosine kinase inhibitors (TKIs) in patients with NSCLC harboring minor mutations of the EGFR gene. PATIENTS AND METHODS: This was a multicenter, retrospective study that analyzed specimens from patients with NSCLC who had minor EGFR gene mutations and were treated with EGFR-TKIs between June 2002 and March 2012. RESULTS: Out of 56 patients with minor mutations of the EGFR gene, 44 were treated with either gefitinib or erlotinib. Mutation sites were G719X in exon 18 (n=35), L861Q in exon 21 (n=11), and G874S in exon 21 (n=1). Three patients had both the G719S and the L861Q mutation. The response rate to TKI treatment was 29.5%, and the disease control rate was 63.6%. The median progression-free survival (PFS) was 6.7 months [95% confidence interval (CI)=2.06-8.66 months]. The median PFS was 7.2 months (95% CI=4.23-12.3 months) in 32 patients who received first- or second-line treatment with EGFR-TKIs, whereas the median PFS was 1.57 months (95% CI=0.73-3.8 months) in 12 patients treated with EGFR-TKIs as a third-line or later treatment. In multivariate Cox analysis, erlotinib therapy was associated with a longer PFS than gefitinib (p=0.025). CONCLUSION: Patients with NSCLC harboring minor mutations of the EGFR gene exhibited a modest response to EGFR-TKI treatment. Treatment with first-generation EGFR-TKIs, in particular erlotinib, may be considered a first- or second-line option for patients with NSCLC with minor EGFR mutations.\\n26124334\\t56\\t64\\tPatients\\tSpecies\\t9606\\n26124334\\t70\\t96\\tNon-small Cell Lung Cancer\\tDisease\\tMESH:D002289\\n26124334\\t113\\t145\\tEpidermal Growth Factor Receptor\\tGene\\t1956\\n26124334\\t240\\t266\\tNon-small cell lung cancer\\tDisease\\tMESH:D002289\\n26124334\\t268\\t273\\tNSCLC\\tDisease\\tMESH:D002289\\n26124334\\t303\\t335\\tepidermal growth factor receptor\\tGene\\t1956\\n26124334\\t337\\t341\\tEGFR\\tGene\\t1956\\n26124334\\t413\\t418\\tL858R\\tProteinMutation\\ttmVar:p|SUB|L|858|R;HGVS:p.L858R;VariantGroup:0;CorrespondingGene:1956;RS#:121434568;CA#:126713\\n26124334\\t481\\t488\\tpatient\\tSpecies\\t9606\\n26124334\\t606\\t610\\tEGFR\\tGene\\t1956\\n26124334\\t648\\t656\\tpatients\\tSpecies\\t9606\\n26124334\\t662\\t667\\tNSCLC\\tDisease\\tMESH:D002289\\n26124334\\t701\\t705\\tEGFR\\tGene\\t1956\\n26124334\\t712\\t720\\tPATIENTS\\tSpecies\\t9606\\n26124334\\t807\\t815\\tpatients\\tSpecies\\t9606\\n26124334\\t821\\t826\\tNSCLC\\tDisease\\tMESH:D002289\\n26124334\\t841\\t845\\tEGFR\\tGene\\t1956\\n26124334\\t883\\t887\\tEGFR\\tGene\\t1956\\n26124334\\t946\\t954\\tpatients\\tSpecies\\t9606\\n26124334\\t983\\t987\\tEGFR\\tGene\\t1956\\n26124334\\t1022\\t1031\\tgefitinib\\tChemical\\tMESH:D000077156\\n26124334\\t1035\\t1044\\terlotinib\\tChemical\\tMESH:D000069347\\n26124334\\t1066\\t1071\\tG719X\\tProteinMutation\\ttmVar:p|SUB|G|719|X;HGVS:p.G719X;VariantGroup:1;CorrespondingGene:1956;RS#:28929495\\n26124334\\t1091\\t1096\\tL861Q\\tProteinMutation\\ttmVar:p|SUB|L|861|Q;HGVS:p.L861Q;VariantGroup:2;CorrespondingGene:1956;RS#:121913444;CA#:176021\\n26124334\\t1120\\t1125\\tG874S\\tProteinMutation\\ttmVar:p|SUB|G|874|S;HGVS:p.G874S;VariantGroup:3;CorrespondingGene:1956\\n26124334\\t1150\\t1158\\tpatients\\tSpecies\\t9606\\n26124334\\t1172\\t1177\\tG719S\\tProteinMutation\\ttmVar:p|SUB|G|719|S;HGVS:p.G719S;VariantGroup:1;CorrespondingGene:1956;RS#:28929495;CA#:126722\\n26124334\\t1186\\t1191\\tL861Q\\tProteinMutation\\ttmVar:p|SUB|L|861|Q;HGVS:p.L861Q;VariantGroup:2;CorrespondingGene:1956;RS#:121913444;CA#:176021\\n26124334\\t1457\\t1465\\tpatients\\tSpecies\\t9606\\n26124334\\t1516\\t1520\\tEGFR\\tGene\\t1956\\n26124334\\t1597\\t1605\\tpatients\\tSpecies\\t9606\\n26124334\\t1619\\t1623\\tEGFR\\tGene\\t1956\\n26124334\\t1695\\t1704\\terlotinib\\tChemical\\tMESH:D000069347\\n26124334\\t1751\\t1760\\tgefitinib\\tChemical\\tMESH:D000077156\\n26124334\\t1784\\t1792\\tPatients\\tSpecies\\t9606\\n26124334\\t1798\\t1803\\tNSCLC\\tDisease\\tMESH:D002289\\n26124334\\t1837\\t1841\\tEGFR\\tGene\\t1956\\n26124334\\t1878\\t1882\\tEGFR\\tGene\\t1956\\n26124334\\t1930\\t1934\\tEGFR\\tGene\\t1956\\n26124334\\t1955\\t1964\\terlotinib\\tChemical\\tMESH:D000069347\\n26124334\\t2019\\t2027\\tpatients\\tSpecies\\t9606\\n26124334\\t2033\\t2038\\tNSCLC\\tDisease\\tMESH:D002289\\n26124334\\t2050\\t2054\\tEGFR\\tGene\\t1956\\n\\n26209642|t|Value of KRAS as prognostic or predictive marker in NSCLC: results from the TAILOR trial.\\n26209642|a|BACKGROUND: The prognostic and predictive role of KRAS mutations in advanced nonsmall-cell lung cancer (NSCLC) is still unclear. TAILOR prospectively assessed the prognostic and predictive value of KRAS mutations in NSCLC patients treated with erlotinib or docetaxel in second line. PATIENTS AND METHODS: NSCLC patients from 52 Italian hospitals were genotyped for KRAS and EGFR mutational status in two independent laboratories. Wild-type EGFR patients (N = 218) received first-line platinum-based chemotherapy and were randomly allocated at progression to erlotinib or docetaxel. Overall survival (OS) according to KRAS mutational status was the primary end point. RESULTS: KRAS mutations were present in 23% of TAILOR randomized cases. The presence of a KRAS mutation did not adversely affect progression-free (PFS) or overall (OS) survival [hazard ratio (HR) PFS = 1.01, 95% confidence interval (CI) 0.71-1.41, P = 0.977; OS = 1.24, 95% CI 0.87-1.77, P = 0.233], nor influenced treatment outcome (test for interaction: OS P = 0.965; PFS P = 0.417). Patients randomized to docetaxel treatment experienced longer survival independently from the KRAS mutational status of their tumors (HR: mutated KRAS 0.81, 95% CI 0.45-1.47; wild-type KRAS 0.79, 95% CI 0.57-1.10). CONCLUSION: In TAILOR, KRAS was neither prognostic nor predictive of benefit for either docetaxel or erlotinib. Docetaxel remains superior independently from KRAS status for second-line treatment in EGFR wild-type advanced NSCLC patients. CLINICAL TRIAL REGISTRATION: NCT00637910.\\n26209642\\t9\\t13\\tKRAS\\tGene\\t3845\\n26209642\\t52\\t57\\tNSCLC\\tDisease\\tMESH:D002289\\n26209642\\t140\\t144\\tKRAS\\tGene\\t3845\\n26209642\\t167\\t192\\tnonsmall-cell lung cancer\\tDisease\\tMESH:D002289\\n26209642\\t194\\t199\\tNSCLC\\tDisease\\tMESH:D002289\\n26209642\\t288\\t292\\tKRAS\\tGene\\t3845\\n26209642\\t306\\t311\\tNSCLC\\tDisease\\tMESH:D002289\\n26209642\\t312\\t320\\tpatients\\tSpecies\\t9606\\n26209642\\t334\\t343\\terlotinib\\tChemical\\tMESH:D000069347\\n26209642\\t347\\t356\\tdocetaxel\\tChemical\\tMESH:D000077143\\n26209642\\t373\\t381\\tPATIENTS\\tSpecies\\t9606\\n26209642\\t395\\t400\\tNSCLC\\tDisease\\tMESH:D002289\\n26209642\\t401\\t409\\tpatients\\tSpecies\\t9606\\n26209642\\t455\\t459\\tKRAS\\tGene\\t3845\\n26209642\\t464\\t468\\tEGFR\\tGene\\t1956\\n26209642\\t530\\t534\\tEGFR\\tGene\\t1956\\n26209642\\t535\\t543\\tpatients\\tSpecies\\t9606\\n26209642\\t574\\t582\\tplatinum\\tChemical\\tMESH:D010984\\n26209642\\t648\\t657\\terlotinib\\tChemical\\tMESH:D000069347\\n26209642\\t661\\t670\\tdocetaxel\\tChemical\\tMESH:D000077143\\n26209642\\t707\\t711\\tKRAS\\tGene\\t3845\\n26209642\\t766\\t770\\tKRAS\\tGene\\t3845\\n26209642\\t847\\t851\\tKRAS\\tGene\\t3845\\n26209642\\t1143\\t1151\\tPatients\\tSpecies\\t9606\\n26209642\\t1166\\t1175\\tdocetaxel\\tChemical\\tMESH:D000077143\\n26209642\\t1237\\t1241\\tKRAS\\tGene\\t3845\\n26209642\\t1269\\t1275\\ttumors\\tDisease\\tMESH:D009369\\n26209642\\t1289\\t1293\\tKRAS\\tGene\\t3845\\n26209642\\t1328\\t1332\\tKRAS\\tGene\\t3845\\n26209642\\t1381\\t1385\\tKRAS\\tGene\\t3845\\n26209642\\t1446\\t1455\\tdocetaxel\\tChemical\\tMESH:D000077143\\n26209642\\t1459\\t1468\\terlotinib\\tChemical\\tMESH:D000069347\\n26209642\\t1470\\t1479\\tDocetaxel\\tChemical\\tMESH:D000077143\\n26209642\\t1516\\t1520\\tKRAS\\tGene\\t3845\\n26209642\\t1557\\t1561\\tEGFR\\tGene\\t1956\\n26209642\\t1581\\t1586\\tNSCLC\\tDisease\\tMESH:D002289\\n26209642\\t1587\\t1595\\tpatients\\tSpecies\\t9606\\n\\n26287849|t|Vemurafenib in Multiple Nonmelanoma Cancers with BRAF V600 Mutations.\\n26287849|a|BACKGROUND: BRAF V600 mutations occur in various nonmelanoma cancers. We undertook a histology-independent phase 2 \"basket\" study of vemurafenib in BRAF V600 mutation-positive nonmelanoma cancers. METHODS: We enrolled patients in six prespecified cancer cohorts; patients with all other tumor types were enrolled in a seventh cohort. A total of 122 patients with BRAF V600 mutation-positive cancer were treated, including 27 patients with colorectal cancer who received vemurafenib and cetuximab. The primary end point was the response rate; secondary end points included progression-free and overall survival. RESULTS: In the cohort with non-small-cell lung cancer, the response rate was 42% (95% confidence interval [CI], 20 to 67) and median progression-free survival was 7.3 months (95% CI, 3.5 to 10.8). In the cohort with Erdheim-Chester disease or Langerhans\\'-cell histiocytosis, the response rate was 43% (95% CI, 18 to 71); the median treatment duration was 5.9 months (range, 0.6 to 18.6), and no patients had disease progression during therapy. There were anecdotal responses among patients with pleomorphic xanthoastrocytoma, anaplastic thyroid cancer, cholangiocarcinoma, salivary-duct cancer, ovarian cancer, and clear-cell sarcoma and among patients with colorectal cancer who received vemurafenib and cetuximab. Safety was similar to that in prior studies of vemurafenib for melanoma. CONCLUSIONS: BRAF V600 appears to be a targetable oncogene in some, but not all, nonmelanoma cancers. Preliminary vemurafenib activity was observed in non-small-cell lung cancer and in Erdheim-Chester disease and Langerhans\\'-cell histiocytosis. The histologic context is an important determinant of response in BRAF V600-mutated cancers. (Funded by F. Hoffmann-La Roche/Genentech; ClinicalTrials.gov number, NCT01524978.).\\n26287849\\t0\\t11\\tVemurafenib\\tChemical\\tMESH:D000077484\\n26287849\\t15\\t43\\tMultiple Nonmelanoma Cancers\\tDisease\\tMESH:D009369\\n26287849\\t49\\t53\\tBRAF\\tGene\\t673\\n26287849\\t82\\t86\\tBRAF\\tGene\\t673\\n26287849\\t119\\t138\\tnonmelanoma cancers\\tDisease\\tMESH:D009369\\n26287849\\t203\\t214\\tvemurafenib\\tChemical\\tMESH:D000077484\\n26287849\\t218\\t222\\tBRAF\\tGene\\t673\\n26287849\\t246\\t265\\tnonmelanoma cancers\\tDisease\\tMESH:D009369\\n26287849\\t288\\t296\\tpatients\\tSpecies\\t9606\\n26287849\\t317\\t323\\tcancer\\tDisease\\tMESH:D009369\\n26287849\\t333\\t341\\tpatients\\tSpecies\\t9606\\n26287849\\t357\\t362\\ttumor\\tDisease\\tMESH:D009369\\n26287849\\t419\\t427\\tpatients\\tSpecies\\t9606\\n26287849\\t433\\t437\\tBRAF\\tGene\\t673\\n26287849\\t461\\t467\\tcancer\\tDisease\\tMESH:D009369\\n26287849\\t495\\t503\\tpatients\\tSpecies\\t9606\\n26287849\\t509\\t526\\tcolorectal cancer\\tDisease\\tMESH:D015179\\n26287849\\t540\\t551\\tvemurafenib\\tChemical\\tMESH:D000077484\\n26287849\\t556\\t565\\tcetuximab\\tChemical\\tMESH:D000068818\\n26287849\\t724\\t735\\tlung cancer\\tDisease\\tMESH:D008175\\n26287849\\t898\\t921\\tErdheim-Chester disease\\tDisease\\tMESH:D031249\\n26287849\\t1077\\t1085\\tpatients\\tSpecies\\t9606\\n26287849\\t1163\\t1171\\tpatients\\tSpecies\\t9606\\n26287849\\t1177\\t1233\\tpleomorphic xanthoastrocytoma, anaplastic thyroid cancer\\tDisease\\tMESH:D065646\\n26287849\\t1235\\t1253\\tcholangiocarcinoma\\tDisease\\tMESH:D018281\\n26287849\\t1255\\t1275\\tsalivary-duct cancer\\tDisease\\tMESH:D009369\\n26287849\\t1277\\t1291\\tovarian cancer\\tDisease\\tMESH:D010051\\n26287849\\t1297\\t1315\\tclear-cell sarcoma\\tDisease\\tMESH:D018227\\n26287849\\t1326\\t1334\\tpatients\\tSpecies\\t9606\\n26287849\\t1340\\t1357\\tcolorectal cancer\\tDisease\\tMESH:D015179\\n26287849\\t1371\\t1382\\tvemurafenib\\tChemical\\tMESH:D000077484\\n26287849\\t1387\\t1396\\tcetuximab\\tChemical\\tMESH:D000068818\\n26287849\\t1445\\t1456\\tvemurafenib\\tChemical\\tMESH:D000077484\\n26287849\\t1461\\t1469\\tmelanoma\\tDisease\\tMESH:D008545\\n26287849\\t1484\\t1488\\tBRAF\\tGene\\t673\\n26287849\\t1552\\t1571\\tnonmelanoma cancers\\tDisease\\tMESH:D009369\\n26287849\\t1637\\t1648\\tlung cancer\\tDisease\\tMESH:D008175\\n26287849\\t1656\\t1679\\tErdheim-Chester disease\\tDisease\\tMESH:D031249\\n26287849\\t1782\\t1786\\tBRAF\\tGene\\t673\\n26287849\\t1800\\t1807\\tcancers\\tDisease\\tMESH:D009369\\n\\n26439803|t|Low EGFR/MET ratio is associated with resistance to EGFR inhibitors in non-small cell lung cancer.\\n26439803|a|PURPOSE: Although activating mutations in the epidermal growth factor receptor (EGFR) gene are predictive markers for response to EGFR inhibitors, 30-40% of EGFR-mutant non-small cell lung cancer (NSCLC) patients are de novo non-responders. Hence, we sought to explore additional biomarkers of response. METHODS: We conducted a prospective pilot study to characterize the expression and/or activation of key receptor tyrosine kinases (RTKs) in stage IIIB-IV NSCLC tumors. A total of 37 patients were enrolled and 34 underwent EGFR inhibitor treatment. RESULTS: As expected, patients bearing activating EGFR mutations showed increased progression free survival (PFS) compared to patients with wild-type EGFR status (9.3 vs 1.4 months, p = 0.0629). Analysis of baseline tumor RTK profiles revealed that, regardless of EGFR mutation status, higher levels of EGFR relative to MET correlated with longer PFS. At multiple EGFR/MET ratio cut-offs, including 1, 2 and 3, median PFS according to below vs. above cut-offs were 0.4 vs. 6.1 (p = 0.0001), 0.5 vs. 9.3 (p = 0.0006) and 1.0 vs. 11.2 months (p = 0.0008), respectively. CONCLUSION: The EGFR/MET ratio measured in tumors at baseline may help identify NSCLC patients most likely to benefit from prolonged PFS when treated with EGFR inhibitors.\\n26439803\\t4\\t8\\tEGFR\\tGene\\t1956\\n26439803\\t9\\t12\\tMET\\tGene\\t79811\\n26439803\\t52\\t56\\tEGFR\\tGene\\t1956\\n26439803\\t86\\t97\\tlung cancer\\tDisease\\tMESH:D008175\\n26439803\\t145\\t177\\tepidermal growth factor receptor\\tGene\\t1956\\n26439803\\t179\\t183\\tEGFR\\tGene\\t1956\\n26439803\\t229\\t233\\tEGFR\\tGene\\t1956\\n26439803\\t256\\t260\\tEGFR\\tGene\\t1956\\n26439803\\t283\\t294\\tlung cancer\\tDisease\\tMESH:D008175\\n26439803\\t303\\t311\\tpatients\\tSpecies\\t9606\\n26439803\\t563\\t569\\ttumors\\tDisease\\tMESH:D009369\\n26439803\\t585\\t593\\tpatients\\tSpecies\\t9606\\n26439803\\t625\\t629\\tEGFR\\tGene\\t1956\\n26439803\\t673\\t681\\tpatients\\tSpecies\\t9606\\n26439803\\t701\\t705\\tEGFR\\tGene\\t1956\\n26439803\\t777\\t785\\tpatients\\tSpecies\\t9606\\n26439803\\t801\\t805\\tEGFR\\tGene\\t1956\\n26439803\\t867\\t872\\ttumor\\tDisease\\tMESH:D009369\\n26439803\\t915\\t919\\tEGFR\\tGene\\t1956\\n26439803\\t954\\t958\\tEGFR\\tGene\\t1956\\n26439803\\t971\\t974\\tMET\\tGene\\t79811\\n26439803\\t1015\\t1019\\tEGFR\\tGene\\t1956\\n26439803\\t1020\\t1023\\tMET\\tGene\\t79811\\n26439803\\t1235\\t1239\\tEGFR\\tGene\\t1956\\n26439803\\t1240\\t1243\\tMET\\tGene\\t79811\\n26439803\\t1262\\t1268\\ttumors\\tDisease\\tMESH:D009369\\n26439803\\t1305\\t1313\\tpatients\\tSpecies\\t9606\\n26439803\\t1374\\t1378\\tEGFR\\tGene\\t1956\\n\\n26515464|t|In vitro modeling to determine mutation specificity of EGFR tyrosine kinase inhibitors against clinically relevant EGFR mutants in non-small-cell lung cancer.\\n26515464|a|EGFR mutated lung cancer accounts for a significant subgroup of non-small-cell lung cancer (NSCLC). Over the last decade, multiple EGFR tyrosine kinase inhibitors (EGFR-TKIs) have been developed to target mutated EGFR. However, there is little information regarding mutation specific potency of EGFR-TKIs against various types of EGFR mutations. The purpose of this study is to establish an in vitro model to determine the \"therapeutic window\" of EGFR-TKIs against various types of EGFR mutations, including EGFR exon 20 insertion mutations. The potency of 1st (erlotinib), 2nd (afatinib) and 3rd (osimertinib and rociletinib) generation EGFR-TKIs was compared in vitro for human lung cancer cell lines and Ba/F3 cells, which exogenously express mutated or wild type EGFR. An in vitro model of mutation specificity was created by calculating the ratio of IC50 values between mutated and wild type EGFR. The in vitro model identified a wide therapeutic window of afatinib for exon 19 deletions and L858R and of osimertinib and rociletinib for T790M positive mutations. The results obtained with our models matched well with previously reported preclinical and clinical data. Interestingly, for EGFR exon 20 insertion mutations, most of which are known to be resistant to 1st and 2nd generation EGFR-TKIS, osimertinib was potent and presented a wide therapeutic window. To our knowledge, this is the first report that has identified the therapeutic window of osimertinib for EGFR exon 20 insertion mutations. In conclusion, this model will provide a preclinical rationale for proper selection of EGFR-TKIs against clinically-relevant EGFR mutations. \\n26515464\\t115\\t119\\tEGFR\\tGene\\t1956\\n26515464\\t146\\t157\\tlung cancer\\tDisease\\tMESH:D008175\\n26515464\\t159\\t163\\tEGFR\\tGene\\t1956\\n26515464\\t172\\t183\\tlung cancer\\tDisease\\tMESH:D008175\\n26515464\\t238\\t249\\tlung cancer\\tDisease\\tMESH:D008175\\n26515464\\t323\\t327\\tEGFR\\tGene\\t1956\\n26515464\\t372\\t376\\tEGFR\\tGene\\t1956\\n26515464\\t454\\t458\\tEGFR\\tGene\\t1956\\n26515464\\t489\\t493\\tEGFR\\tGene\\t1956\\n26515464\\t606\\t610\\tEGFR\\tGene\\t1956\\n26515464\\t641\\t645\\tEGFR\\tGene\\t1956\\n26515464\\t667\\t671\\tEGFR\\tGene\\t1956\\n26515464\\t721\\t730\\terlotinib\\tChemical\\tMESH:D000069347\\n26515464\\t738\\t746\\tafatinib\\tChemical\\tMESH:D000077716\\n26515464\\t757\\t768\\tosimertinib\\tChemical\\t-\\n26515464\\t773\\t784\\trociletinib\\tChemical\\tMESH:C000589977\\n26515464\\t797\\t801\\tEGFR\\tGene\\t1956\\n26515464\\t833\\t838\\thuman\\tSpecies\\t9606\\n26515464\\t839\\t850\\tlung cancer\\tDisease\\tMESH:D008175\\n26515464\\t866\\t871\\tBa/F3\\tCellLine\\tCVCL_0161;NCBITaxID:10090\\n26515464\\t926\\t930\\tEGFR\\tGene\\t13649\\n26515464\\t1056\\t1060\\tEGFR\\tGene\\t13649\\n26515464\\t1121\\t1129\\tafatinib\\tChemical\\tMESH:D000077716\\n26515464\\t1156\\t1161\\tL858R\\tProteinMutation\\ttmVar:p|SUB|L|858|R;HGVS:p.L858R;VariantGroup:1;CorrespondingGene:1956;RS#:121434568;CA#:126713\\n26515464\\t1169\\t1180\\tosimertinib\\tChemical\\t-\\n26515464\\t1185\\t1196\\trociletinib\\tChemical\\tMESH:C000589977\\n26515464\\t1201\\t1206\\tT790M\\tProteinMutation\\ttmVar:p|SUB|T|790|M;HGVS:p.T790M;VariantGroup:0;CorrespondingGene:1956;RS#:121434569;CA#:90928\\n26515464\\t1352\\t1356\\tEGFR\\tGene\\t13649\\n26515464\\t1452\\t1456\\tEGFR\\tGene\\t13649\\n26515464\\t1463\\t1474\\tosimertinib\\tChemical\\t-\\n26515464\\t1616\\t1627\\tosimertinib\\tChemical\\t-\\n26515464\\t1632\\t1636\\tEGFR\\tGene\\t13649\\n26515464\\t1753\\t1757\\tEGFR\\tGene\\t13649\\n26515464\\t1791\\t1795\\tEGFR\\tGene\\t13649\\n\\n26554404|t|The ALK/ROS1 Inhibitor PF-06463922 Overcomes Primary Resistance to Crizotinib in ALK-Driven Neuroblastoma.\\n26554404|a|UNLABELLED: Neuroblastomas harboring activating point mutations in anaplastic lymphoma kinase (ALK) are differentially sensitive to the ALK inhibitor crizotinib, with certain mutations conferring intrinsic crizotinib resistance. To overcome this clinical obstacle, our goal was to identify inhibitors with improved potency that can target intractable ALK variants such as F1174L. We find that PF-06463922 has high potency across ALK variants and inhibits ALK more effectively than crizotinib in vitro. Most importantly, PF-06463922 induces complete tumor regression in both crizotinib-resistant and crizotinib-sensitive xenograft mouse models of neuroblastoma, as well as in patient-derived xenografts harboring the crizotinib-resistant F1174L or F1245C mutations. These studies demonstrate that PF-06463922 has the potential to overcome crizotinib resistance and exerts unprecedented activity as a single targeted agent against F1174L and F1245C ALK-mutated xenograft tumors, while also inducing responses in an R1275Q xenograft model. Taken together, these results provide the rationale to move PF-06463922 into clinical trials for treatment of patients with ALK-mutated neuroblastoma. SIGNIFICANCE: The next-generation ALK/ROS1 inhibitor PF-06463922 exerts unparalleled activity in ALK-driven neuroblastoma models with primary crizotinib resistance. Our biochemical and in vivo data provide the preclinical rationale for fast-tracking the development of this agent in children with relapsed/refractory ALK-mutant neuroblastoma.\\n26554404\\t4\\t7\\tALK\\tGene\\t238\\n26554404\\t8\\t12\\tROS1\\tGene\\t6098\\n26554404\\t23\\t34\\tPF-06463922\\tChemical\\tMESH:C000590786\\n26554404\\t67\\t77\\tCrizotinib\\tChemical\\tMESH:D000077547\\n26554404\\t81\\t84\\tALK\\tGene\\t238\\n26554404\\t92\\t105\\tNeuroblastoma\\tDisease\\tMESH:D009447\\n26554404\\t119\\t133\\tNeuroblastomas\\tDisease\\tMESH:D009447\\n26554404\\t174\\t200\\tanaplastic lymphoma kinase\\tGene\\t238\\n26554404\\t202\\t205\\tALK\\tGene\\t238\\n26554404\\t243\\t246\\tALK\\tGene\\t238\\n26554404\\t257\\t267\\tcrizotinib\\tChemical\\tMESH:D000077547\\n26554404\\t313\\t323\\tcrizotinib\\tChemical\\tMESH:D000077547\\n26554404\\t458\\t461\\tALK\\tGene\\t238\\n26554404\\t479\\t485\\tF1174L\\tProteinMutation\\ttmVar:p|SUB|F|1174|L;HGVS:p.F1174L;VariantGroup:0;CorrespondingGene:238;RS#:863225281;CA#:279616\\n26554404\\t500\\t511\\tPF-06463922\\tChemical\\tMESH:C000590786\\n26554404\\t536\\t539\\tALK\\tGene\\t238\\n26554404\\t562\\t565\\tALK\\tGene\\t238\\n26554404\\t588\\t598\\tcrizotinib\\tChemical\\tMESH:D000077547\\n26554404\\t627\\t638\\tPF-06463922\\tChemical\\tMESH:C000590786\\n26554404\\t656\\t661\\ttumor\\tDisease\\tMESH:D009369\\n26554404\\t681\\t691\\tcrizotinib\\tChemical\\tMESH:D000077547\\n26554404\\t706\\t716\\tcrizotinib\\tChemical\\tMESH:D000077547\\n26554404\\t737\\t742\\tmouse\\tSpecies\\t10090\\n26554404\\t753\\t766\\tneuroblastoma\\tDisease\\tMESH:D009447\\n26554404\\t782\\t789\\tpatient\\tSpecies\\t9606\\n26554404\\t823\\t833\\tcrizotinib\\tChemical\\tMESH:D000077547\\n26554404\\t844\\t850\\tF1174L\\tProteinMutation\\ttmVar:p|SUB|F|1174|L;HGVS:p.F1174L;VariantGroup:0;CorrespondingGene:238;RS#:863225281;CA#:279616\\n26554404\\t854\\t860\\tF1245C\\tProteinMutation\\ttmVar:p|SUB|F|1245|C;HGVS:p.F1245C;VariantGroup:1;CorrespondingGene:238;RS#:863225283;CA#:279597\\n26554404\\t903\\t914\\tPF-06463922\\tChemical\\tMESH:C000590786\\n26554404\\t945\\t955\\tcrizotinib\\tChemical\\tMESH:D000077547\\n26554404\\t1036\\t1042\\tF1174L\\tProteinMutation\\ttmVar:p|SUB|F|1174|L;HGVS:p.F1174L;VariantGroup:0;CorrespondingGene:238;RS#:863225281;CA#:279616\\n26554404\\t1047\\t1053\\tF1245C\\tProteinMutation\\ttmVar:p|SUB|F|1245|C;HGVS:p.F1245C;VariantGroup:1;CorrespondingGene:238;RS#:863225283;CA#:279597\\n26554404\\t1054\\t1057\\tALK\\tGene\\t238\\n26554404\\t1076\\t1082\\ttumors\\tDisease\\tMESH:D009369\\n26554404\\t1120\\t1126\\tR1275Q\\tProteinMutation\\ttmVar:p|SUB|R|1275|Q;HGVS:p.R1275Q;VariantGroup:2;CorrespondingGene:238;RS#:113994087;CA#:341482\\n26554404\\t1204\\t1215\\tPF-06463922\\tChemical\\tMESH:C000590786\\n26554404\\t1254\\t1262\\tpatients\\tSpecies\\t9606\\n26554404\\t1268\\t1271\\tALK\\tGene\\t238\\n26554404\\t1280\\t1293\\tneuroblastoma\\tDisease\\tMESH:D009447\\n26554404\\t1329\\t1332\\tALK\\tGene\\t238\\n26554404\\t1333\\t1337\\tROS1\\tGene\\t6098\\n26554404\\t1348\\t1359\\tPF-06463922\\tChemical\\tMESH:C000590786\\n26554404\\t1392\\t1395\\tALK\\tGene\\t238\\n26554404\\t1403\\t1416\\tneuroblastoma\\tDisease\\tMESH:D009447\\n26554404\\t1437\\t1447\\tcrizotinib\\tChemical\\tMESH:D000077547\\n26554404\\t1578\\t1586\\tchildren\\tSpecies\\t9606\\n26554404\\t1612\\t1615\\tALK\\tGene\\t238\\n26554404\\t1623\\t1636\\tneuroblastoma\\tDisease\\tMESH:D009447\\n\\n26758680|t|Somatically mutated ABL1 is an actionable and essential NSCLC survival gene.\\n26758680|a|The lack of actionable mutations in patients with non-small cell lung cancer (NSCLC) presents a significant hurdle in the design of targeted therapies for this disease. Here, we identify somatically mutated ABL1 as a genetic dependency that is required to maintain NSCLC cell survival. We demonstrate that NSCLC cells with ABL1 mutations are sensitive to ABL inhibitors and we verify that the drug-induced effects on cell viability are specific to pharmacological inhibition of the ABL1 kinase. Furthermore, we confirm that imatinib suppresses lung tumor growth in vivo, specifically in lung cancer cells harboring a gain-of-function (GOF) mutation in ABL1. Consistent with structural modeling, we demonstrate that mutations in ABL1 identified in primary NSCLC tumors and a lung cancer cell line increase downstream pathway activation compared to wild-type ABL1. Finally, we observe that the ABL1 cancer mutants display an increased cytosolic localization, which is associated with the oncogenic properties of the ABL1 kinase. In summary, our results suggest that NSCLC patients with ABL1 mutations could be stratified for treatment with imatinib in combination with other therapies. \\n26758680\\t20\\t24\\tABL1\\tGene\\t25\\n26758680\\t56\\t61\\tNSCLC\\tDisease\\tMESH:D002289\\n26758680\\t113\\t121\\tpatients\\tSpecies\\t9606\\n26758680\\t127\\t153\\tnon-small cell lung cancer\\tDisease\\tMESH:D002289\\n26758680\\t155\\t160\\tNSCLC\\tDisease\\tMESH:D002289\\n26758680\\t284\\t288\\tABL1\\tGene\\t25\\n26758680\\t342\\t347\\tNSCLC\\tDisease\\tMESH:D002289\\n26758680\\t383\\t388\\tNSCLC\\tDisease\\tMESH:D002289\\n26758680\\t400\\t404\\tABL1\\tGene\\t25\\n26758680\\t432\\t435\\tABL\\tGene\\t25\\n26758680\\t559\\t563\\tABL1\\tGene\\t25\\n26758680\\t601\\t609\\timatinib\\tChemical\\tMESH:D000068877\\n26758680\\t621\\t631\\tlung tumor\\tDisease\\tMESH:D008175\\n26758680\\t664\\t675\\tlung cancer\\tDisease\\tMESH:D008175\\n26758680\\t729\\t733\\tABL1\\tGene\\t25\\n26758680\\t805\\t809\\tABL1\\tGene\\t25\\n26758680\\t832\\t837\\tNSCLC\\tDisease\\tMESH:D002289\\n26758680\\t851\\t862\\tlung cancer\\tDisease\\tMESH:D008175\\n26758680\\t934\\t938\\tABL1\\tGene\\t25\\n26758680\\t969\\t973\\tABL1\\tGene\\t25\\n26758680\\t974\\t980\\tcancer\\tDisease\\tMESH:D009369\\n26758680\\t1091\\t1095\\tABL1\\tGene\\t25\\n26758680\\t1141\\t1146\\tNSCLC\\tDisease\\tMESH:D002289\\n26758680\\t1147\\t1155\\tpatients\\tSpecies\\t9606\\n26758680\\t1161\\t1165\\tABL1\\tGene\\t25\\n26758680\\t1215\\t1223\\timatinib\\tChemical\\tMESH:D000068877\\n\\n26771872|t|Predictive relevance of PD-L1 expression combined with CD8+ TIL density in stage III non-small cell lung cancer patients receiving concurrent chemoradiotherapy.\\n26771872|a|BACKGROUND: Expression of programmed cell death-ligand 1 (PD-L1) is known to be a mechanism whereby cancer can escape immune surveillance, but little is known about factors predictive of efficacy in patients with locally advanced non-small cell lung cancer (NSCLC). We investigated the predictive relevance of PD-L1 expression and CD8+ tumour-infiltrating lymphocytes (TILs) density in patients with locally advanced NSCLC receiving concurrent chemoradiotherapy (CCRT). METHODS: We retrospectively reviewed 74 consecutive patients with stage III NSCLC who had received CCRT. PD-L1 expression and CD8+ TIL density were evaluated by immunohistochemical analysis. RESULTS: Univariate and multivariate analyses demonstrated that CD8+ TIL density was an independent and significant predictive factor for progression-free survival (PFS) and OS, whereas PD-L1 expression was not correlated with PFS and OS. Sub-analysis revealed that the PD-L1+/CD8 low group had the shortest PFS (8.6 months, p = 0.02) and OS (13.9 months, p = 0.11), and that the PD-L1-/CD8 high group had the longest prognosis (median PFS and OS were not reached) by Kaplan-Meier curves of the four sub-groups. CONCLUSIONS: Among stage III NSCLC patients who received CCRT, there was a trend for poor survival in those who expressed PD-L1. Our analysis indicated that a combination of lack of PD-L1 expression and CD8+ TIL density was significantly associated with favourable survival in these patients. It is proposed that PD-L1 expression in combination with CD8+ TIL density could be a useful predictive biomarker in patients with stage III NSCLC.\\n26771872\\t24\\t29\\tPD-L1\\tGene\\t29126\\n26771872\\t55\\t58\\tCD8\\tGene\\t925\\n26771872\\t105\\t111\\tcancer\\tDisease\\tMESH:D009369\\n26771872\\t112\\t120\\tpatients\\tSpecies\\t9606\\n26771872\\t187\\t217\\tprogrammed cell death-ligand 1\\tGene\\t29126\\n26771872\\t219\\t224\\tPD-L1\\tGene\\t29126\\n26771872\\t261\\t267\\tcancer\\tDisease\\tMESH:D009369\\n26771872\\t360\\t368\\tpatients\\tSpecies\\t9606\\n26771872\\t391\\t417\\tnon-small cell lung cancer\\tDisease\\tMESH:D002289\\n26771872\\t419\\t424\\tNSCLC\\tDisease\\tMESH:D002289\\n26771872\\t471\\t476\\tPD-L1\\tGene\\t29126\\n26771872\\t492\\t495\\tCD8\\tGene\\t925\\n26771872\\t547\\t555\\tpatients\\tSpecies\\t9606\\n26771872\\t578\\t583\\tNSCLC\\tDisease\\tMESH:D002289\\n26771872\\t683\\t691\\tpatients\\tSpecies\\t9606\\n26771872\\t707\\t712\\tNSCLC\\tDisease\\tMESH:D002289\\n26771872\\t736\\t741\\tPD-L1\\tGene\\t29126\\n26771872\\t757\\t760\\tCD8\\tGene\\t925\\n26771872\\t886\\t889\\tCD8\\tGene\\t925\\n26771872\\t1008\\t1013\\tPD-L1\\tGene\\t29126\\n26771872\\t1092\\t1097\\tPD-L1\\tGene\\t29126\\n26771872\\t1099\\t1102\\tCD8\\tGene\\t925\\n26771872\\t1202\\t1207\\tPD-L1\\tGene\\t29126\\n26771872\\t1209\\t1212\\tCD8\\tGene\\t925\\n26771872\\t1363\\t1368\\tNSCLC\\tDisease\\tMESH:D002289\\n26771872\\t1369\\t1377\\tpatients\\tSpecies\\t9606\\n26771872\\t1456\\t1461\\tPD-L1\\tGene\\t29126\\n26771872\\t1516\\t1521\\tPD-L1\\tGene\\t29126\\n26771872\\t1537\\t1540\\tCD8\\tGene\\t925\\n26771872\\t1617\\t1625\\tpatients\\tSpecies\\t9606\\n26771872\\t1647\\t1652\\tPD-L1\\tGene\\t29126\\n26771872\\t1684\\t1687\\tCD8\\tGene\\t925\\n26771872\\t1743\\t1751\\tpatients\\tSpecies\\t9606\\n26771872\\t1767\\t1772\\tNSCLC\\tDisease\\tMESH:D002289\\n\\n26899019|t|Prognostic and Predictive Effect of TP53 Mutations in Patients with Non-Small Cell Lung Cancer from Adjuvant Cisplatin-Based Therapy Randomized Trials: A LACE-Bio Pooled Analysis.\\n26899019|a|INTRODUCTION: Tumor protein p53 gene (TP53) mutations are common in stage I through III non-small cell lung cancer, but clinical trials have shown inconsistent results regarding their relationship to the effects of adjuvant therapy. The objective is to clarify their putative prognostic and predictive effects. METHODS: A pooled analysis of TP53 mutations (exons 5-8) was conducted in four randomized trials (the International Adjuvant Lung Cancer Trial, J BRonchus 10, Cancer and Leukemia Group B-9633, and Adjuvant Navelbine International Trialist Association trial) of platinum-based adjuvant chemotherapy (ACT) versus observation (OBS). Hazard ratios (HRs) and 95% confidence intervals (CIs) of mutant versus wild-type (WT) TP53 for overall survival (OS) and disease-free survival (DFS) were estimated using a multivariable Cox model stratified on trial and adjusted on sex, age, and clinicopathological variables. Predictive value was evaluated with an interaction between treatment and TP53. RESULTS: A total of 1209 patients (median follow-up 5.5 years) were included. There were 573 deaths (47%) and 653 DFS events (54%). Mutations (434 [36%]) had no prognostic effect (OBS HROS = 0.99, 95% CI: 0.77-1.28, p = 0.95; HRDFS = 0.99, 95% CI: 0.78-1.25, p = 0.92) but were marginally predictive of benefit from ACT for OS (test for interaction: OS, p = 0.06; DFS, p = 0.11). Patients with WT TP53 had a tendency toward better outcomes with ACT than did those in the OBS group (HROS = 0.77, 95% CI: 0.62-0.95, p = 0.02; HRDFS = 0.75, 95% CI: 0.62-0.92, p = 0.005). In the ACT arm, a deleterious effect of mutant versus WT TP53 was observed (HROS = 1.40, 95% CI: 1.10-1.78, p = 0.006; HRDFS = 1.31, 95% CI: 1.04-1.64, p = 0.02). CONCLUSIONS: TP53 mutation had no prognostic effect but was marginally predictive for survival from ACT. In patients who received ACT, TP53 mutation tended to be associated with shorter survival than wild-type TP53.\\n26899019\\t36\\t40\\tTP53\\tGene\\t7157\\n26899019\\t54\\t62\\tPatients\\tSpecies\\t9606\\n26899019\\t68\\t94\\tNon-Small Cell Lung Cancer\\tDisease\\tMESH:D002289\\n26899019\\t109\\t118\\tCisplatin\\tChemical\\tMESH:D002945\\n26899019\\t218\\t222\\tTP53\\tGene\\t7157\\n26899019\\t283\\t294\\tlung cancer\\tDisease\\tMESH:D008175\\n26899019\\t521\\t525\\tTP53\\tGene\\t7157\\n26899019\\t616\\t627\\tLung Cancer\\tDisease\\tMESH:D008175\\n26899019\\t650\\t656\\tCancer\\tDisease\\tMESH:D009369\\n26899019\\t661\\t669\\tLeukemia\\tDisease\\tMESH:D007938\\n26899019\\t697\\t706\\tNavelbine\\tChemical\\tMESH:D000077235\\n26899019\\t752\\t760\\tplatinum\\tChemical\\tMESH:D010984\\n26899019\\t908\\t912\\tTP53\\tGene\\t7157\\n26899019\\t1172\\t1176\\tTP53\\tGene\\t7157\\n26899019\\t1203\\t1211\\tpatients\\tSpecies\\t9606\\n26899019\\t1271\\t1277\\tdeaths\\tDisease\\tMESH:D003643\\n26899019\\t1558\\t1566\\tPatients\\tSpecies\\t9606\\n26899019\\t1575\\t1579\\tTP53\\tGene\\t7157\\n26899019\\t1804\\t1808\\tTP53\\tGene\\t7157\\n26899019\\t1923\\t1927\\tTP53\\tGene\\t7157\\n26899019\\t2018\\t2026\\tpatients\\tSpecies\\t9606\\n26899019\\t2045\\t2049\\tTP53\\tGene\\t7157\\n26899019\\t2120\\t2124\\tTP53\\tGene\\t7157\\n\\n26918046|t|Association of Nuclear PIM1 Expression with Lymph Node Metastasis and Poor Prognosis in Patients with Lung Adenocarcinoma and Squamous Cell Carcinoma.\\n26918046|a|Increasing evidence indicates that aberrant expression of PIM1, p-STAT3 and c-MYC is involved in the pathogenesis of various solid tumors, but its prognostic value is still unclear in non-small cell lung cancer (NSCLC). Here, we sought to evaluate the expression and prognostic role of these markers in patients with lung adenocarcinoma (AD) and squamous cell carcinoma (SCC). Real time RT-PCR and Western blotting was used to analyze the mRNA and protein expression of PIM1 in NSCLC cell lines, respectively. The expression of PIM1, p-STAT3, and c-MYC was immunohistochemically tested in archival tumor samples from 194 lung AD and SCC patients. High nuclear PIM1 expression was detected in 43.3% of ADs and SCCs, and was significantly correlated with lymph node (LN) metastasis (P = 0.028) and histology (P = 0.003). High nuclear PIM1 expression (P = 0.034), locally advanced stage (P < 0.001), AD (P = 0.007) and poor pathologic differentiation (P = 0.002) were correlated with worse disease-free survival (DFS). High nuclear PIM1 expression (P = 0.009), advanced clinical stage (P < 0.001) and poor pathologic differentiation (P = 0.004) were independent unfavorable prognostic factors for overall survival (OS). High p-STAT3 expression was not associated with OS but significantly correlated with LN metastasis, while c-MYC was not significantly correlated with any clinicopathological parameter or survival. Therefore, in AD and SCC patients, nuclear PIM1 expression level is an independent factor for DFS and OS and it might serve as a predictive biomarker for outcome. \\n26918046\\t23\\t27\\tPIM1\\tGene\\t5292\\n26918046\\t88\\t96\\tPatients\\tSpecies\\t9606\\n26918046\\t102\\t149\\tLung Adenocarcinoma and Squamous Cell Carcinoma\\tDisease\\tMESH:D002294\\n26918046\\t209\\t213\\tPIM1\\tGene\\t5292\\n26918046\\t217\\t222\\tSTAT3\\tGene\\t6774\\n26918046\\t227\\t232\\tc-MYC\\tGene\\t4609\\n26918046\\t282\\t288\\ttumors\\tDisease\\tMESH:D009369\\n26918046\\t350\\t361\\tlung cancer\\tDisease\\tMESH:D008175\\n26918046\\t454\\t462\\tpatients\\tSpecies\\t9606\\n26918046\\t468\\t487\\tlung adenocarcinoma\\tDisease\\tMESH:D000077192\\n26918046\\t497\\t520\\tsquamous cell carcinoma\\tDisease\\tMESH:D002294\\n26918046\\t522\\t525\\tSCC\\tDisease\\tMESH:D002294\\n26918046\\t621\\t625\\tPIM1\\tGene\\t5292\\n26918046\\t679\\t683\\tPIM1\\tGene\\t5292\\n26918046\\t687\\t692\\tSTAT3\\tGene\\t6774\\n26918046\\t698\\t703\\tc-MYC\\tGene\\t4609\\n26918046\\t740\\t754\\tarchival tumor\\tDisease\\tMESH:D009369\\n26918046\\t784\\t787\\tSCC\\tDisease\\tMESH:D002294\\n26918046\\t788\\t796\\tpatients\\tSpecies\\t9606\\n26918046\\t811\\t815\\tPIM1\\tGene\\t5292\\n26918046\\t983\\t987\\tPIM1\\tGene\\t5292\\n26918046\\t1180\\t1184\\tPIM1\\tGene\\t5292\\n26918046\\t1375\\t1380\\tSTAT3\\tGene\\t6774\\n26918046\\t1474\\t1479\\tc-MYC\\tGene\\t4609\\n26918046\\t1586\\t1589\\tSCC\\tDisease\\tMESH:D002294\\n26918046\\t1590\\t1598\\tpatients\\tSpecies\\t9606\\n26918046\\t1608\\t1612\\tPIM1\\tGene\\t5292\\n\\n26938871|t|Rapid and dramatic response to alectinib in an anaplastic lymphoma kinase rearranged non-small-cell lung cancer patient who is critically ill.\\n26938871|a|Anaplastic lymphoma kinase (ALK) tyrosine kinase inhibitors (TKIs) have shown promising clinical activity in the treatment of non-small-cell lung cancer (NSCLC) that harbors ALK rearrangement. The next-generation ALK-TKI, alectinib, has been reported to have potent efficacy in ALK-positive NSCLC patients including on mutations that confer resistance to crizotinib, which was the first ALK-TKI approved for ALK-positive NSCLC. The efficacy and safety of ALK-TKIs, including crizotinib and alectinib, as the first-line treatment in critically ill patients is unclear. We report one ALK-positive NSCLC patient with poor performance status (PS) and disseminated intravascular coagulation because of respiratory failure and multiple metastases, and experienced the rapid and dramatic response to alectinib without adverse events that can lead to discontinuation and dose reduction of the drug. After a couple of months of treatment with alectinib, radiological review indicated a complete response. The present case is the first reported case of rapid and marked response to alectinib in ALK-positive NSCLC patients who had poor PS and severe organ dysfunction, such as disseminated intravascular coagulation. Further investigation of the safety and efficacy of ALK-TKI for ALK-positive NSCLC patients who are critically ill is warranted.\\n26938871\\t31\\t40\\talectinib\\tChemical\\tMESH:C582670\\n26938871\\t47\\t73\\tanaplastic lymphoma kinase\\tGene\\t238\\n26938871\\t105\\t111\\tcancer\\tDisease\\tMESH:D009369\\n26938871\\t112\\t119\\tpatient\\tSpecies\\t9606\\n26938871\\t127\\t141\\tcritically ill\\tDisease\\tMESH:D016638\\n26938871\\t143\\t169\\tAnaplastic lymphoma kinase\\tGene\\t238\\n26938871\\t171\\t174\\tALK\\tGene\\t238\\n26938871\\t284\\t295\\tlung cancer\\tDisease\\tMESH:D008175\\n26938871\\t317\\t320\\tALK\\tGene\\t238\\n26938871\\t356\\t359\\tALK\\tGene\\t238\\n26938871\\t365\\t374\\talectinib\\tChemical\\tMESH:C582670\\n26938871\\t421\\t424\\tALK\\tGene\\t238\\n26938871\\t440\\t448\\tpatients\\tSpecies\\t9606\\n26938871\\t498\\t508\\tcrizotinib\\tChemical\\tMESH:D000077547\\n26938871\\t530\\t533\\tALK\\tGene\\t238\\n26938871\\t551\\t554\\tALK\\tGene\\t238\\n26938871\\t598\\t601\\tALK\\tGene\\t238\\n26938871\\t618\\t628\\tcrizotinib\\tChemical\\tMESH:D000077547\\n26938871\\t633\\t642\\talectinib\\tChemical\\tMESH:C582670\\n26938871\\t675\\t689\\tcritically ill\\tDisease\\tMESH:D016638\\n26938871\\t690\\t698\\tpatients\\tSpecies\\t9606\\n26938871\\t725\\t728\\tALK\\tGene\\t238\\n26938871\\t744\\t751\\tpatient\\tSpecies\\t9606\\n26938871\\t790\\t828\\tdisseminated intravascular coagulation\\tDisease\\tMESH:D004211\\n26938871\\t840\\t859\\trespiratory failure\\tDisease\\tMESH:D012131\\n26938871\\t873\\t883\\tmetastases\\tDisease\\tMESH:D009362\\n26938871\\t936\\t945\\talectinib\\tChemical\\tMESH:C582670\\n26938871\\t1077\\t1086\\talectinib\\tChemical\\tMESH:C582670\\n26938871\\t1215\\t1224\\talectinib\\tChemical\\tMESH:C582670\\n26938871\\t1228\\t1231\\tALK\\tGene\\t238\\n26938871\\t1247\\t1255\\tpatients\\tSpecies\\t9606\\n26938871\\t1283\\t1300\\torgan dysfunction\\tDisease\\tMESH:D009102\\n26938871\\t1310\\t1348\\tdisseminated intravascular coagulation\\tDisease\\tMESH:D004211\\n26938871\\t1402\\t1405\\tALK\\tGene\\t238\\n26938871\\t1414\\t1417\\tALK\\tGene\\t238\\n26938871\\t1433\\t1441\\tpatients\\tSpecies\\t9606\\n26938871\\t1450\\t1464\\tcritically ill\\tDisease\\tMESH:D016638\\n\\n27022118|t|Intracranial Efficacy of Crizotinib Versus Chemotherapy in Patients With Advanced ALK-Positive Non-Small-Cell Lung Cancer: Results From PROFILE 1014.\\n27022118|a|PURPOSE: Intracranial efficacy of first-line crizotinib versus chemotherapy was compared prospectively in the phase III PROFILE 1014 study in ALK-positive non-small-cell lung cancer. PATIENTS AND METHODS: Patients were randomly assigned to receive crizotinib (250 mg twice daily; n = 172) or chemotherapy (pemetrexed 500 mg/m(2) plus cisplatin 75 mg/m(2) or carboplatin at area under the curve 5 to 6, every 3 weeks for <= six cycles; n = 171). Patients with stable treated brain metastases (tBM) were eligible. Intracranial efficacy was assessed at baseline and every 6 or 12 weeks in patients with or without known brain metastases (BM), respectively; intracranial time to tumor progression (IC-TTP; per protocol) and intracranial disease control rate (IC-DCR; post hoc) were measured. The intent-to-treat population was also assessed. RESULTS: Of 343 patients in the intent-to-treat population, 23% had tBM at baseline. A nonsignificant IC-TTP improvement was observed with crizotinib in the intent-to-treat population (hazard ratio [HR], 0.60; P = .069), patients with tBM (HR, 0.45; P = .063), and patients without BM (HR, 0.69; P = .323). Among patients with tBM, IC-DCR was significantly higher with crizotinib versus chemotherapy at 12 weeks (85% v 45%, respectively; P < .001) and 24 weeks (56% v 25%, respectively; P = .006). Progression-free survival was significantly longer with crizotinib versus chemotherapy in both subgroups (tBM present: HR, 0.40; P < .001; median, 9.0 v 4.0 months, respectively; BM absent: HR, 0.51; P < .001; median, 11.1 v 7.2 months, respectively) and in the intent-to-treat population (HR, 0.45; P < .001; median, 10.9 v 7.0 months, respectively). CONCLUSION: Compared with chemotherapy, crizotinib demonstrated a significantly higher IC-DCR in patients with tBM. Improvements in IC-TTP were not statistically significant in patients with or without tBM, although sensitivity to detect treatment differences in or between the two subgroups was low.\\n27022118\\t25\\t35\\tCrizotinib\\tChemical\\tMESH:D000077547\\n27022118\\t59\\t67\\tPatients\\tSpecies\\t9606\\n27022118\\t82\\t85\\tALK\\tGene\\t238\\n27022118\\t195\\t205\\tcrizotinib\\tChemical\\tMESH:D000077547\\n27022118\\t292\\t295\\tALK\\tGene\\t238\\n27022118\\t320\\t331\\tlung cancer\\tDisease\\tMESH:D008175\\n27022118\\t333\\t341\\tPATIENTS\\tSpecies\\t9606\\n27022118\\t355\\t363\\tPatients\\tSpecies\\t9606\\n27022118\\t398\\t408\\tcrizotinib\\tChemical\\tMESH:D000077547\\n27022118\\t456\\t466\\tpemetrexed\\tChemical\\tMESH:D000068437\\n27022118\\t484\\t493\\tcisplatin\\tChemical\\tMESH:D002945\\n27022118\\t508\\t519\\tcarboplatin\\tChemical\\tMESH:D016190\\n27022118\\t595\\t603\\tPatients\\tSpecies\\t9606\\n27022118\\t630\\t640\\tmetastases\\tDisease\\tMESH:D009362\\n27022118\\t736\\t744\\tpatients\\tSpecies\\t9606\\n27022118\\t773\\t783\\tmetastases\\tDisease\\tMESH:D009362\\n27022118\\t825\\t830\\ttumor\\tDisease\\tMESH:D009369\\n27022118\\t870\\t890\\tintracranial disease\\tDisease\\tMESH:D020765\\n27022118\\t1004\\t1012\\tpatients\\tSpecies\\t9606\\n27022118\\t1127\\t1137\\tcrizotinib\\tChemical\\tMESH:D000077547\\n27022118\\t1209\\t1217\\tpatients\\tSpecies\\t9606\\n27022118\\t1253\\t1261\\tpatients\\tSpecies\\t9606\\n27022118\\t1301\\t1309\\tpatients\\tSpecies\\t9606\\n27022118\\t1357\\t1367\\tcrizotinib\\tChemical\\tMESH:D000077547\\n27022118\\t1542\\t1552\\tcrizotinib\\tChemical\\tMESH:D000077547\\n27022118\\t1878\\t1888\\tcrizotinib\\tChemical\\tMESH:D000077547\\n27022118\\t1935\\t1943\\tpatients\\tSpecies\\t9606\\n27022118\\t2015\\t2023\\tpatients\\tSpecies\\t9606\\n27022118\\t2040\\t2043\\ttBM\\tChemical\\t-\\n\\n27315356|t|Efficacy and safety of dovitinib in pretreated patients with advanced squamous non-small cell lung cancer with FGFR1 amplification: A single-arm, phase 2 study.\\n27315356|a|BACKGROUND: Fibroblast growth factor receptor 1 (FGFR1) amplification is a potential driving oncogene in squamous cell cancer (SCC) of the lung. The current phase 2 study evaluated the efficacy and tolerability of dovitinib, an FGFR inhibitor, in patients with advanced SCC of the lung. METHODS: Patients with pretreated advanced SCC of the lung whose tumors demonstrated FGFR1 amplification of > 5 copies by fluorescence in situ hybridization were enrolled. Dovitinib at a dose of 500 mg was administered orally, once daily, on days 1 to 5 of every week, followed by 2 days off. The primary endpoint was overall response. Secondary endpoints were progression-free survival, overall survival, and toxicity. RESULTS: All 26 patients were men with a median age of 68 years (range, 52-80 years). The majority of patients were ever-smokers. The median duration of dovitinib administration (28 days per cycle) was 2.5 months (range, 0.7-8.6 months). The overall response rate was 11.5% (95% confidence interval [95% CI], 0.8%-23.8%) and the disease control rate was 50% (95% CI, 30.8%-69.2%), with 3 patients achieving partial responses. Response durations for the patients with partial responses were >=4.5 months, >= 5.1 months, and 6.1 months, respectively. After a median follow-up of 15.7 months (range, 1.2-25.6 months), the median overall survival was 5.0 months (95% CI, 3.6-6.4 months) and the median progression-free survival was 2.9 months (95% CI, 1.5-4.3 months). The most common grade 3 or higher adverse events were fatigue (19.2%), anorexia (11.5%), and hyponatremia (11.5%) (event severity was graded based on National Cancer Institute Common Terminology Criteria for Adverse Events [version 4.0]). CONCLUSIONS: Treatment with dovitinib demonstrated modest efficacy in patients with advanced SCC with FGFR1 amplification. Further studies to evaluate other biomarkers correlated with the efficacy of dovitinib in patients with SCC are warranted. Cancer 2016;122:3024-3031.   2016 American Cancer Society.\\n27315356\\t23\\t32\\tdovitinib\\tChemical\\tMESH:C500007\\n27315356\\t47\\t55\\tpatients\\tSpecies\\t9606\\n27315356\\t70\\t105\\tsquamous non-small cell lung cancer\\tDisease\\tMESH:D002289\\n27315356\\t111\\t116\\tFGFR1\\tGene\\t2260\\n27315356\\t173\\t208\\tFibroblast growth factor receptor 1\\tGene\\t2260\\n27315356\\t210\\t215\\tFGFR1\\tGene\\t2260\\n27315356\\t266\\t286\\tsquamous cell cancer\\tDisease\\tMESH:D002294\\n27315356\\t288\\t291\\tSCC\\tDisease\\tMESH:D002294\\n27315356\\t375\\t384\\tdovitinib\\tChemical\\tMESH:C500007\\n27315356\\t389\\t393\\tFGFR\\tGene\\t2260\\n27315356\\t408\\t416\\tpatients\\tSpecies\\t9606\\n27315356\\t431\\t434\\tSCC\\tDisease\\tMESH:D002294\\n27315356\\t457\\t465\\tPatients\\tSpecies\\t9606\\n27315356\\t491\\t494\\tSCC\\tDisease\\tMESH:D002294\\n27315356\\t513\\t519\\ttumors\\tDisease\\tMESH:D009369\\n27315356\\t533\\t538\\tFGFR1\\tGene\\t2260\\n27315356\\t620\\t629\\tDovitinib\\tChemical\\tMESH:C500007\\n27315356\\t858\\t866\\ttoxicity\\tDisease\\tMESH:D064420\\n27315356\\t884\\t892\\tpatients\\tSpecies\\t9606\\n27315356\\t898\\t901\\tmen\\tSpecies\\t9606\\n27315356\\t970\\t978\\tpatients\\tSpecies\\t9606\\n27315356\\t1021\\t1030\\tdovitinib\\tChemical\\tMESH:C500007\\n27315356\\t1256\\t1264\\tpatients\\tSpecies\\t9606\\n27315356\\t1321\\t1329\\tpatients\\tSpecies\\t9606\\n27315356\\t1687\\t1694\\tfatigue\\tDisease\\tMESH:D005221\\n27315356\\t1704\\t1712\\tanorexia\\tDisease\\tMESH:D000855\\n27315356\\t1726\\t1738\\thyponatremia\\tDisease\\tMESH:D007010\\n27315356\\t1792\\t1798\\tCancer\\tDisease\\tMESH:D009369\\n27315356\\t1900\\t1909\\tdovitinib\\tChemical\\tMESH:C500007\\n27315356\\t1942\\t1950\\tpatients\\tSpecies\\t9606\\n27315356\\t1965\\t1968\\tSCC\\tDisease\\tMESH:D002294\\n27315356\\t1974\\t1979\\tFGFR1\\tGene\\t2260\\n27315356\\t2072\\t2081\\tdovitinib\\tChemical\\tMESH:C500007\\n27315356\\t2085\\t2093\\tpatients\\tSpecies\\t9606\\n27315356\\t2099\\t2102\\tSCC\\tDisease\\tMESH:D002294\\n27315356\\t2118\\t2124\\tCancer\\tDisease\\tMESH:D009369\\n27315356\\t2161\\t2167\\tCancer\\tDisease\\tMESH:D009369\\n\\n27544060|t|Clinical Activity of Alectinib in Advanced RET-Rearranged Non-Small Cell Lung Cancer.\\n27544060|a|INTRODUCTION: Chromosomal rearrangements involving rearranged during transfection gene (RET) occur in 1% to 2% of NSCLCs and may confer sensitivity to rearranged during transfection (RET) inhibitors. Alectinib is an anaplastic lymphoma kinase tyrosine kinase inhibitor (TKI) that also has anti-RET activity in vitro. The clinical activity of alectinib in patients with RET-rearranged NSCLC has not yet been reported. METHODS: We have described four patients with advanced RET-rearranged NSCLC who were treated with alectinib (600 mg twice daily [n = 3] or 900 mg twice daily [n = 1]) as part of single-patient compassionate use protocols or off-label use of the commercially available drug. RESULTS: Four patients with metastatic RET-rearranged NSCLC were identified. Three of the four had received prior RET TKIs, including cabozantinib and experimental RET inhibitors. In total, we observed two (50%) objective radiographic responses after treatment with alectinib (one confirmed and one unconfirmed), with durations of therapy of 6 months and more than 5 months (treatment ongoing), respectively. Notably, one of these two patients had his dose of alectinib escalated to 900 mg twice daily and had clinical improvement in central nervous system metastases. In addition, one patient (25%) experienced a best response of stable disease lasting approximately 6 weeks (the drug discontinued for toxicity). A fourth patient who was RET TKI-naive had primary progression while receiving alectinib. CONCLUSIONS: Alectinib demonstrated preliminary antitumor activity in patients with advanced RET-rearranged NSCLC, most of whom had received prior RET inhibitors. Larger prospective studies with longer follow-up are needed to assess the efficacy of alectinib in RET-rearranged NSCLC and other RET-driven malignancies. In parallel, development of more selective, potent RET TKIs is warranted.\\n27544060\\t21\\t30\\tAlectinib\\tChemical\\tMESH:C582670\\n27544060\\t43\\t46\\tRET\\tGene\\t5979\\n27544060\\t58\\t84\\tNon-Small Cell Lung Cancer\\tDisease\\tMESH:D002289\\n27544060\\t174\\t177\\tRET\\tGene\\t5979\\n27544060\\t200\\t206\\tNSCLCs\\tDisease\\tMESH:D002289\\n27544060\\t269\\t272\\tRET\\tGene\\t5979\\n27544060\\t286\\t295\\tAlectinib\\tChemical\\tMESH:C582670\\n27544060\\t302\\t321\\tanaplastic lymphoma\\tDisease\\tMESH:D017728\\n27544060\\t380\\t383\\tRET\\tGene\\t5979\\n27544060\\t428\\t437\\talectinib\\tChemical\\tMESH:C582670\\n27544060\\t441\\t449\\tpatients\\tSpecies\\t9606\\n27544060\\t455\\t458\\tRET\\tGene\\t5979\\n27544060\\t470\\t475\\tNSCLC\\tDisease\\tMESH:D002289\\n27544060\\t535\\t543\\tpatients\\tSpecies\\t9606\\n27544060\\t558\\t561\\tRET\\tGene\\t5979\\n27544060\\t573\\t578\\tNSCLC\\tDisease\\tMESH:D002289\\n27544060\\t601\\t610\\talectinib\\tChemical\\tMESH:C582670\\n27544060\\t688\\t695\\tpatient\\tSpecies\\t9606\\n27544060\\t791\\t799\\tpatients\\tSpecies\\t9606\\n27544060\\t816\\t819\\tRET\\tGene\\t5979\\n27544060\\t831\\t836\\tNSCLC\\tDisease\\tMESH:D002289\\n27544060\\t891\\t894\\tRET\\tGene\\t5979\\n27544060\\t911\\t923\\tcabozantinib\\tChemical\\tMESH:C558660\\n27544060\\t941\\t944\\tRET\\tGene\\t5979\\n27544060\\t1043\\t1052\\talectinib\\tChemical\\tMESH:C582670\\n27544060\\t1212\\t1220\\tpatients\\tSpecies\\t9606\\n27544060\\t1237\\t1246\\talectinib\\tChemical\\tMESH:C582670\\n27544060\\t1334\\t1344\\tmetastases\\tDisease\\tMESH:D009362\\n27544060\\t1363\\t1370\\tpatient\\tSpecies\\t9606\\n27544060\\t1480\\t1488\\ttoxicity\\tDisease\\tMESH:D064420\\n27544060\\t1500\\t1507\\tpatient\\tSpecies\\t9606\\n27544060\\t1516\\t1519\\tRET\\tGene\\t5979\\n27544060\\t1570\\t1579\\talectinib\\tChemical\\tMESH:C582670\\n27544060\\t1594\\t1603\\tAlectinib\\tChemical\\tMESH:C582670\\n27544060\\t1651\\t1659\\tpatients\\tSpecies\\t9606\\n27544060\\t1674\\t1677\\tRET\\tGene\\t5979\\n27544060\\t1689\\t1694\\tNSCLC\\tDisease\\tMESH:D002289\\n27544060\\t1728\\t1731\\tRET\\tGene\\t5979\\n27544060\\t1830\\t1839\\talectinib\\tChemical\\tMESH:C582670\\n27544060\\t1843\\t1846\\tRET\\tGene\\t5979\\n27544060\\t1858\\t1863\\tNSCLC\\tDisease\\tMESH:D002289\\n27544060\\t1874\\t1877\\tRET\\tGene\\t5979\\n27544060\\t1885\\t1897\\tmalignancies\\tDisease\\tMESH:D009369\\n27544060\\t1950\\t1953\\tRET\\tGene\\t5979\\n\\n27694386|t|Acquired METD1228V Mutation and Resistance to MET Inhibition in Lung Cancer.\\n27694386|a|Amplified and/or mutated MET can act as both a primary oncogenic driver and as a promoter of tyrosine kinase inhibitor (TKI) resistance in non-small cell lung cancer (NSCLC). However, the landscape of MET-specific targeting agents remains underdeveloped, and understanding of mechanisms of resistance to MET TKIs is limited. Here, we present a case of a patient with lung adenocarcinoma harboring both a mutation in EGFR and an amplification of MET, who after progression on erlotinib responded dramatically to combined MET and EGFR inhibition with savolitinib and osimertinib. When resistance developed to this combination, a new MET kinase domain mutation, D1228V, was detected. Our in vitro findings demonstrate that METD1228V induces resistance to type I MET TKIs through impaired drug binding, while sensitivity to type II MET TKIs is maintained. Based on these findings, the patient was treated with erlotinib combined with cabozantinib, a type II MET inhibitor, and exhibited a response. SIGNIFICANCE: With several structurally distinct MET inhibitors undergoing development for treatment of NSCLC, it is critical to identify mechanism-based therapies for drug resistance. We demonstrate that an acquired METD1228V mutation mediates resistance to type I, but not type II, MET inhibitors, having therapeutic implications for the clinical use of sequential MET inhibitors. Cancer Discov; 6(12); 1334-41.  2016 AACR.See related commentary by Trusolino, p. 1306This article is highlighted in the In This Issue feature, p. 1293.\\n27694386\\t9\\t12\\tMET\\tGene\\t79811\\n27694386\\t46\\t49\\tMET\\tGene\\t79811\\n27694386\\t64\\t75\\tLung Cancer\\tDisease\\tMESH:D008175\\n27694386\\t102\\t105\\tMET\\tGene\\t79811\\n27694386\\t231\\t242\\tlung cancer\\tDisease\\tMESH:D008175\\n27694386\\t278\\t281\\tMET\\tGene\\t79811\\n27694386\\t381\\t384\\tMET\\tGene\\t79811\\n27694386\\t431\\t438\\tpatient\\tSpecies\\t9606\\n27694386\\t444\\t463\\tlung adenocarcinoma\\tDisease\\tMESH:D000077192\\n27694386\\t493\\t497\\tEGFR\\tGene\\t1956\\n27694386\\t522\\t525\\tMET\\tGene\\t79811\\n27694386\\t552\\t561\\terlotinib\\tChemical\\tMESH:D000069347\\n27694386\\t597\\t600\\tMET\\tGene\\t79811\\n27694386\\t605\\t609\\tEGFR\\tGene\\t1956\\n27694386\\t626\\t637\\tsavolitinib\\tChemical\\tMESH:C000593259\\n27694386\\t642\\t653\\tosimertinib\\tChemical\\t-\\n27694386\\t708\\t711\\tMET\\tGene\\t79811\\n27694386\\t797\\t806\\tMETD1228V\\tChemical\\t-\\n27694386\\t836\\t839\\tMET\\tGene\\t79811\\n27694386\\t905\\t908\\tMET\\tGene\\t79811\\n27694386\\t958\\t965\\tpatient\\tSpecies\\t9606\\n27694386\\t983\\t992\\terlotinib\\tChemical\\tMESH:D000069347\\n27694386\\t1007\\t1019\\tcabozantinib\\tChemical\\tMESH:C558660\\n27694386\\t1031\\t1034\\tMET\\tGene\\t79811\\n27694386\\t1121\\t1124\\tMET\\tGene\\t79811\\n27694386\\t1289\\t1298\\tMETD1228V\\tChemical\\t-\\n27694386\\t1356\\t1359\\tMET\\tGene\\t79811\\n27694386\\t1439\\t1442\\tMET\\tGene\\t79811\\n27694386\\t1455\\t1461\\tCancer\\tDisease\\tMESH:D009369\\n\\n27876675|t|A Phase 1/1b Study Evaluating Trametinib Plus Docetaxel or Pemetrexed in Patients With Advanced Non-Small Cell Lung Cancer.\\n27876675|a|OBJECTIVES: This two-part study evaluated trametinib, a MEK1/2 inhibitor, in combination with anticancer agents. Inhibition of MEK, a downstream effector of KRAS, demonstrated preclinical synergy with chemotherapy in KRAS-mutant NSCLC cell lines. Part 1 of this study identified recommended phase 2 doses of trametinib combinations. Part 2, reported herein, evaluated the safety, tolerability, pharmacokinetics, and efficacy of trametinib combinations in patients with NSCLC with and without KRAS mutations. METHODS: Phase 1b evaluated trametinib plus docetaxel with growth factor support (trametinib, 2.0 mg once daily, and docetaxel, 75 mg/m2 every 3 weeks) or pemetrexed (trametinib, 1.5 mg once daily, and pemetrexed, 500 mg/m2 every 3 weeks). Eligibility criteria for the expansion cohorts included metastatic NSCLC with measurable disease, known KRAS mutation status, Eastern Cooperative Oncology Group performance status of 1 or lower, and no more than two prior regimens. RESULTS: The primary end point of overall response rate (ORR) was met for both combinations. A confirmed partial response (PR) was observed in 10 of the 47 patients with NSCLC who received trametinib plus docetaxel (21%). The ORR was 18% (four PRs in 22 patients) in those with KRAS wild-type NSCLC versus 24% (six PRs in 25 patients) in those with KRAS-mutant NSCLC. Of the 42 patients with NSCLC treated with trametinib plus pemetrexed, six (14%) had a PR; the ORR was 17% (four of 23) in patients with KRAS-mutated NSCLC versus 11% (two of 19) in KRAS wild-type NSCLC. Adverse events-most commonly diarrhea, nausea, and fatigue-were manageable. CONCLUSIONS: Trametinib-plus-chemotherapy combinations were tolerable. Clinical activity exceeding the ORRs previously reported with docetaxel or pemetrexed alone in KRAS-mutated NSCLC and meeting prespecified criteria was observed.\\n27876675\\t30\\t40\\tTrametinib\\tChemical\\tMESH:C560077\\n27876675\\t46\\t55\\tDocetaxel\\tChemical\\tMESH:D000077143\\n27876675\\t59\\t69\\tPemetrexed\\tChemical\\tMESH:D000068437\\n27876675\\t73\\t81\\tPatients\\tSpecies\\t9606\\n27876675\\t96\\t122\\tNon-Small Cell Lung Cancer\\tDisease\\tMESH:D002289\\n27876675\\t166\\t176\\ttrametinib\\tChemical\\tMESH:C560077\\n27876675\\t180\\t186\\tMEK1/2\\tGene\\t5604;5605\\n27876675\\t251\\t254\\tMEK\\tGene\\t5609\\n27876675\\t281\\t285\\tKRAS\\tGene\\t3845\\n27876675\\t341\\t345\\tKRAS\\tGene\\t3845\\n27876675\\t353\\t358\\tNSCLC\\tDisease\\tMESH:D002289\\n27876675\\t432\\t442\\ttrametinib\\tChemical\\tMESH:C560077\\n27876675\\t552\\t562\\ttrametinib\\tChemical\\tMESH:C560077\\n27876675\\t579\\t587\\tpatients\\tSpecies\\t9606\\n27876675\\t593\\t598\\tNSCLC\\tDisease\\tMESH:D002289\\n27876675\\t616\\t620\\tKRAS\\tGene\\t3845\\n27876675\\t660\\t670\\ttrametinib\\tChemical\\tMESH:C560077\\n27876675\\t676\\t685\\tdocetaxel\\tChemical\\tMESH:D000077143\\n27876675\\t714\\t724\\ttrametinib\\tChemical\\tMESH:C560077\\n27876675\\t749\\t758\\tdocetaxel\\tChemical\\tMESH:D000077143\\n27876675\\t787\\t797\\tpemetrexed\\tChemical\\tMESH:D000068437\\n27876675\\t799\\t809\\ttrametinib\\tChemical\\tMESH:C560077\\n27876675\\t834\\t844\\tpemetrexed\\tChemical\\tMESH:D000068437\\n27876675\\t939\\t944\\tNSCLC\\tDisease\\tMESH:D002289\\n27876675\\t976\\t980\\tKRAS\\tGene\\t3845\\n27876675\\t1260\\t1268\\tpatients\\tSpecies\\t9606\\n27876675\\t1274\\t1279\\tNSCLC\\tDisease\\tMESH:D002289\\n27876675\\t1293\\t1303\\ttrametinib\\tChemical\\tMESH:C560077\\n27876675\\t1309\\t1318\\tdocetaxel\\tChemical\\tMESH:D000077143\\n27876675\\t1358\\t1366\\tpatients\\tSpecies\\t9606\\n27876675\\t1382\\t1386\\tKRAS\\tGene\\t3845\\n27876675\\t1397\\t1402\\tNSCLC\\tDisease\\tMESH:D002289\\n27876675\\t1429\\t1437\\tpatients\\tSpecies\\t9606\\n27876675\\t1453\\t1457\\tKRAS\\tGene\\t3845\\n27876675\\t1465\\t1470\\tNSCLC\\tDisease\\tMESH:D002289\\n27876675\\t1482\\t1490\\tpatients\\tSpecies\\t9606\\n27876675\\t1496\\t1501\\tNSCLC\\tDisease\\tMESH:D002289\\n27876675\\t1515\\t1525\\ttrametinib\\tChemical\\tMESH:C560077\\n27876675\\t1531\\t1541\\tpemetrexed\\tChemical\\tMESH:D000068437\\n27876675\\t1595\\t1603\\tpatients\\tSpecies\\t9606\\n27876675\\t1609\\t1613\\tKRAS\\tGene\\t3845\\n27876675\\t1622\\t1627\\tNSCLC\\tDisease\\tMESH:D002289\\n27876675\\t1654\\t1658\\tKRAS\\tGene\\t3845\\n27876675\\t1669\\t1674\\tNSCLC\\tDisease\\tMESH:D002289\\n27876675\\t1705\\t1713\\tdiarrhea\\tDisease\\tMESH:D003967\\n27876675\\t1715\\t1721\\tnausea\\tDisease\\tMESH:D009325\\n27876675\\t1727\\t1734\\tfatigue\\tDisease\\tMESH:D005221\\n27876675\\t1765\\t1775\\tTrametinib\\tChemical\\tMESH:C560077\\n27876675\\t1885\\t1894\\tdocetaxel\\tChemical\\tMESH:D000077143\\n27876675\\t1898\\t1908\\tpemetrexed\\tChemical\\tMESH:D000068437\\n27876675\\t1918\\t1922\\tKRAS\\tGene\\t3845\\n27876675\\t1931\\t1936\\tNSCLC\\tDisease\\tMESH:D002289\\n\\n28259530|t|Association between PD-L1 expression and driven gene status in NSCLC: A meta-analysis.\\n28259530|a|PURPOSE: We explored the potential clinical association between programmed death-ligand 1 (PD-L1) expression and driven gene status in non-small cell lung cancer (NSCLC). METHODS: We systemically searched through October 2015. Odd ratios (ORs) with 95% CIs were calculated to examine the association of PD-L1 expression with driven gene status. A random- or fixed-effects model was used. RESULTS: Nine studies were identified. KRAS-mutant tumors were more likely to be PD-L1 positive than KRAS-wild type tumors (51% vs 36%; OR 1.69; 95% CI 1.01-2.84; p = 0.045). In contrast, PD-L1 expression did not differ by EGFR (OR 0.86; 95% CI 0.43-1.73; p = 0.675) or ALK (OR 1.02; 95% CI 0.44-2.37; p = 0.954) status. In subgroup analysis, there was also no significant association between PD-L1 expression and EGFR status in term of the cut-offs or ethnicity. CONCLUSION: In conclusion, NSCLC with KRAS mutations showed a trend for higher frequency of positive PD-L1 expression.\\n28259530\\t20\\t25\\tPD-L1\\tGene\\t29126\\n28259530\\t63\\t68\\tNSCLC\\tDisease\\tMESH:D002289\\n28259530\\t151\\t176\\tprogrammed death-ligand 1\\tGene\\t29126\\n28259530\\t178\\t183\\tPD-L1\\tGene\\t29126\\n28259530\\t222\\t248\\tnon-small cell lung cancer\\tDisease\\tMESH:D002289\\n28259530\\t250\\t255\\tNSCLC\\tDisease\\tMESH:D002289\\n28259530\\t314\\t317\\tOdd\\tGene\\t130497\\n28259530\\t390\\t395\\tPD-L1\\tGene\\t29126\\n28259530\\t514\\t518\\tKRAS\\tGene\\t3845\\n28259530\\t526\\t532\\ttumors\\tDisease\\tMESH:D009369\\n28259530\\t556\\t561\\tPD-L1\\tGene\\t29126\\n28259530\\t576\\t580\\tKRAS\\tGene\\t3845\\n28259530\\t591\\t597\\ttumors\\tDisease\\tMESH:D009369\\n28259530\\t663\\t668\\tPD-L1\\tGene\\t29126\\n28259530\\t698\\t702\\tEGFR\\tGene\\t1956\\n28259530\\t745\\t748\\tALK\\tGene\\t238\\n28259530\\t868\\t873\\tPD-L1\\tGene\\t29126\\n28259530\\t889\\t893\\tEGFR\\tGene\\t1956\\n28259530\\t966\\t971\\tNSCLC\\tDisease\\tMESH:D002289\\n28259530\\t977\\t981\\tKRAS\\tGene\\t3845\\n28259530\\t1040\\t1045\\tPD-L1\\tGene\\t29126\\n\\n28525386|t|Prognostic value of KRAS mutation in advanced non-small-cell lung cancer treated with immune checkpoint inhibitors: A meta-analysis and review.\\n28525386|a|Immune checkpoint inhibitors (ICIs) have emerged as a promising treatment option in the fight against advanced non-small-cell lung cancer (NSCLC). KRAS is the most frequently mutated oncogene in NSCLC. We performed this meta-analysis to investigate if KRAS mutation status affects survival benefits of ICIs in patients with advanced NSCLC. Electronic databases were searched for eligible studies. We included randomized trials with the data of overall survival stratified by KRAS mutation status. From 3 eligible studies, 138 patients with KRAS mutant NSCLC and 371 with KRAS wild-type tumor were included in the meta-analysis. Compared to chemotherapy with docetaxel, ICIs improved overall survival in patients with previously treated KRAS mutant NSCLC (hazard ratio = 0.64 [95% confidence interval, 0.43-0.96], P = 0.03). For patients with KRAS wild-type NSCLC, however, ICIs did not prolong overall survival over that with chemotherapy (hazard ratio = 0.88 [95% confidence interval, 0.68-1.13], P = 0.30). In conclusion, ICIs as a salvage therapy improved overall survival over that with docetaxel in advanced NSCLC patients with KRAS mutation, but not in those with KRAS wild-type tumor. These results suggest that KRAS mutation status may be a potential biomarker for survival benefits to ICIs.\\n28525386\\t20\\t24\\tKRAS\\tGene\\t3845\\n28525386\\t61\\t72\\tlung cancer\\tDisease\\tMESH:D008175\\n28525386\\t270\\t281\\tlung cancer\\tDisease\\tMESH:D008175\\n28525386\\t283\\t288\\tNSCLC\\tDisease\\tMESH:D002289\\n28525386\\t291\\t295\\tKRAS\\tGene\\t3845\\n28525386\\t339\\t344\\tNSCLC\\tDisease\\tMESH:D002289\\n28525386\\t396\\t400\\tKRAS\\tGene\\t3845\\n28525386\\t454\\t462\\tpatients\\tSpecies\\t9606\\n28525386\\t477\\t482\\tNSCLC\\tDisease\\tMESH:D002289\\n28525386\\t619\\t623\\tKRAS\\tGene\\t3845\\n28525386\\t670\\t678\\tpatients\\tSpecies\\t9606\\n28525386\\t684\\t688\\tKRAS\\tGene\\t3845\\n28525386\\t696\\t701\\tNSCLC\\tDisease\\tMESH:D002289\\n28525386\\t715\\t719\\tKRAS\\tGene\\t3845\\n28525386\\t730\\t735\\ttumor\\tDisease\\tMESH:D009369\\n28525386\\t802\\t811\\tdocetaxel\\tChemical\\tMESH:D000077143\\n28525386\\t847\\t855\\tpatients\\tSpecies\\t9606\\n28525386\\t880\\t884\\tKRAS\\tGene\\t3845\\n28525386\\t892\\t897\\tNSCLC\\tDisease\\tMESH:D002289\\n28525386\\t972\\t980\\tpatients\\tSpecies\\t9606\\n28525386\\t986\\t990\\tKRAS\\tGene\\t3845\\n28525386\\t1001\\t1006\\tNSCLC\\tDisease\\tMESH:D002289\\n28525386\\t1235\\t1244\\tdocetaxel\\tChemical\\tMESH:D000077143\\n28525386\\t1257\\t1262\\tNSCLC\\tDisease\\tMESH:D002289\\n28525386\\t1263\\t1271\\tpatients\\tSpecies\\t9606\\n28525386\\t1277\\t1281\\tKRAS\\tGene\\t3845\\n28525386\\t1314\\t1318\\tKRAS\\tGene\\t3845\\n28525386\\t1329\\t1334\\ttumor\\tDisease\\tMESH:D009369\\n28525386\\t1363\\t1367\\tKRAS\\tGene\\t3845\\n\\n28602779|t|Ceritinib versus chemotherapy in patients with ALK-rearranged non-small-cell lung cancer previously given chemotherapy and crizotinib (ASCEND-5): a randomised, controlled, open-label, phase 3 trial.\\n28602779|a|BACKGROUND: Ceritinib is a next-generation anaplastic lymphoma kinase (ALK) inhibitor, which has shown robust anti-tumour efficacy, along with intracranial activity, in patients with ALK-rearranged non-small-cell lung cancer. In phase 1 and 2 studies, ceritinib has been shown to be highly active in both ALK inhibitor-naive and ALK inhibitor-pretreated patients who had progressed after chemotherapy (mostly multiple lines). In this study, we compared the efficacy and safety of ceritinib versus single-agent chemotherapy in patients with advanced ALK-rearranged non-small-cell lung cancer who had previously progressed following crizotinib and platinum-based doublet chemotherapy. METHODS: In this randomised, controlled, open-label, phase 3 trial, we recruited patients aged at least 18 years with ALK-rearranged stage IIIB or IV non-small-cell lung cancer (with at least one measurable lesion) who had received previous chemotherapy (one or two lines, including a platinum doublet) and crizotinib and had subsequent disease progression, from 99 centres across 20 countries. Other inclusion criteria were a WHO performance status of 0-2, adequate organ function and laboratory test results, a life expectancy of at least 12 weeks, and having recovered from previous anticancer treatment-related toxicities. We randomly allocated patients (1:1; with blocking [block size of four]; stratified by WHO performance status [0 vs 1-2] and presence or absence of brain metastases) to oral ceritinib 750 mg per day fasted (in 21 day treatment cycles) or chemotherapy (intravenous pemetrexed 500 mg/m2 or docetaxel 75 mg/m2 [investigator choice], every 21 days). Patients who discontinued chemotherapy because of progressive disease could cross over to the ceritinib group. The primary endpoint was progression-free survival, assessed by a masked independent review committee using Response Evaluation Criteria in Solid Tumors 1.1 in the intention-to-treat population, assessed every 6 weeks until month 18 and every 9 weeks thereafter. This trial is registered with ClinicalTrials.gov, number NCT01828112, and is ongoing but no longer recruiting patients. FINDINGS: Between June 28, 2013, and Nov 2, 2015, we randomly allocated 231 patients; 115 (50%) to ceritinib and 116 (50%) to chemotherapy (40 [34%] to pemetrexed, 73 [63%] to docetaxel, and three [3%] discontinued before receiving treatment). Median follow-up was 16 5 months (IQR 11 5-21 4). Ceritinib showed a significant improvement in median progression-free survival compared with chemotherapy (5 4 months [95% CI 4 1-6 9] for ceritinib vs 1 6 months [1 4-2 8] for chemotherapy; hazard ratio 0 49 [0 36-0 67]; p<0 0001). Serious adverse events were reported in 49 (43%) of 115 patients in the ceritinib group and 36 (32%) of 113 in the chemotherapy group. Treatment-related serious adverse events were similar between groups (13 [11%] in the ceritinib group vs 12 [11%] in the chemotherapy group). The most frequent grade 3-4 adverse events in the ceritinib group were increased alanine aminotransferase concentration (24 [21%] of 115 vs two [2%] of 113 in the chemotherapy group), increased gamma glutamyltransferase concentration (24 [21%] vs one [1%]), and increased aspartate aminotransferase concentration (16 [14%] vs one [1%] in the chemotherapy group). Six (5%) of 115 patients in the ceritinib group discontinued because of adverse events compared with eight (7%) of 116 in the chemotherapy group. 15 (13%) of 115 patients in the ceritinib group and five (4%) of 113 in the chemotherapy group died during the treatment period (from the day of the first dose of study treatment to 30 days after the final dose). 13 (87%) of the 15 patients who died in the ceritinib group died because of disease progression and two (13%) died because of an adverse event (one [7%] cerebrovascular accident and one [7%] respiratory failure); neither of these deaths were considered by the investigator to be treatment related. The five (4%) deaths in the chemotherapy group were all due to disease progression. INTERPRETATION: These findings show that patients derive significant clinical benefit from a more potent ALK inhibitor after failure of crizotinib, and establish ceritinib as a more efficacious treatment option compared with chemotherapy in this patient population. FUNDING: Novartis Pharmaceuticals Corporation.\\n28602779\\t0\\t9\\tCeritinib\\tChemical\\tMESH:C586847\\n28602779\\t33\\t41\\tpatients\\tSpecies\\t9606\\n28602779\\t47\\t50\\tALK\\tGene\\t238\\n28602779\\t77\\t88\\tlung cancer\\tDisease\\tMESH:D008175\\n28602779\\t123\\t133\\tcrizotinib\\tChemical\\tMESH:D000077547\\n28602779\\t211\\t220\\tCeritinib\\tChemical\\tMESH:C586847\\n28602779\\t242\\t268\\tanaplastic lymphoma kinase\\tGene\\t238\\n28602779\\t270\\t273\\tALK\\tGene\\t238\\n28602779\\t314\\t320\\ttumour\\tDisease\\tMESH:D009369\\n28602779\\t368\\t376\\tpatients\\tSpecies\\t9606\\n28602779\\t382\\t385\\tALK\\tGene\\t238\\n28602779\\t412\\t423\\tlung cancer\\tDisease\\tMESH:D008175\\n28602779\\t451\\t460\\tceritinib\\tChemical\\tMESH:C586847\\n28602779\\t504\\t507\\tALK\\tGene\\t238\\n28602779\\t528\\t531\\tALK\\tGene\\t238\\n28602779\\t553\\t561\\tpatients\\tSpecies\\t9606\\n28602779\\t679\\t688\\tceritinib\\tChemical\\tMESH:C586847\\n28602779\\t725\\t733\\tpatients\\tSpecies\\t9606\\n28602779\\t748\\t751\\tALK\\tGene\\t238\\n28602779\\t778\\t789\\tlung cancer\\tDisease\\tMESH:D008175\\n28602779\\t830\\t840\\tcrizotinib\\tChemical\\tMESH:D000077547\\n28602779\\t845\\t853\\tplatinum\\tChemical\\tMESH:D010984\\n28602779\\t963\\t971\\tpatients\\tSpecies\\t9606\\n28602779\\t1000\\t1003\\tALK\\tGene\\t238\\n28602779\\t1032\\t1058\\tnon-small-cell lung cancer\\tDisease\\tMESH:D002289\\n28602779\\t1167\\t1175\\tplatinum\\tChemical\\tMESH:D010984\\n28602779\\t1189\\t1199\\tcrizotinib\\tChemical\\tMESH:D000077547\\n28602779\\t1497\\t1507\\ttoxicities\\tDisease\\tMESH:D064420\\n28602779\\t1531\\t1539\\tpatients\\tSpecies\\t9606\\n28602779\\t1663\\t1673\\tmetastases\\tDisease\\tMESH:D009362\\n28602779\\t1683\\t1692\\tceritinib\\tChemical\\tMESH:C586847\\n28602779\\t1773\\t1783\\tpemetrexed\\tChemical\\tMESH:D000068437\\n28602779\\t1797\\t1806\\tdocetaxel\\tChemical\\tMESH:D000077143\\n28602779\\t1855\\t1863\\tPatients\\tSpecies\\t9606\\n28602779\\t1905\\t1924\\tprogressive disease\\tDisease\\tMESH:D018450\\n28602779\\t1949\\t1958\\tceritinib\\tChemical\\tMESH:C586847\\n28602779\\t2339\\t2347\\tpatients\\tSpecies\\t9606\\n28602779\\t2386\\t2391\\tNov 2\\tGene\\t1490\\n28602779\\t2425\\t2433\\tpatients\\tSpecies\\t9606\\n28602779\\t2448\\t2457\\tceritinib\\tChemical\\tMESH:C586847\\n28602779\\t2501\\t2511\\tpemetrexed\\tChemical\\tMESH:D000068437\\n28602779\\t2525\\t2534\\tdocetaxel\\tChemical\\tMESH:D000077143\\n28602779\\t2643\\t2652\\tCeritinib\\tChemical\\tMESH:C586847\\n28602779\\t2782\\t2791\\tceritinib\\tChemical\\tMESH:C586847\\n28602779\\t2932\\t2940\\tpatients\\tSpecies\\t9606\\n28602779\\t2948\\t2957\\tceritinib\\tChemical\\tMESH:C586847\\n28602779\\t3097\\t3106\\tceritinib\\tChemical\\tMESH:C586847\\n28602779\\t3203\\t3212\\tceritinib\\tChemical\\tMESH:C586847\\n28602779\\t3532\\t3540\\tpatients\\tSpecies\\t9606\\n28602779\\t3548\\t3557\\tceritinib\\tChemical\\tMESH:C586847\\n28602779\\t3678\\t3686\\tpatients\\tSpecies\\t9606\\n28602779\\t3694\\t3703\\tceritinib\\tChemical\\tMESH:C586847\\n28602779\\t3757\\t3761\\tdied\\tDisease\\tMESH:D003643\\n28602779\\t3894\\t3902\\tpatients\\tSpecies\\t9606\\n28602779\\t3907\\t3911\\tdied\\tDisease\\tMESH:D003643\\n28602779\\t3919\\t3928\\tceritinib\\tChemical\\tMESH:C586847\\n28602779\\t3935\\t3939\\tdied\\tDisease\\tMESH:D003643\\n28602779\\t3985\\t3989\\tdied\\tDisease\\tMESH:D003643\\n28602779\\t4028\\t4052\\tcerebrovascular accident\\tDisease\\tMESH:D020521\\n28602779\\t4066\\t4085\\trespiratory failure\\tDisease\\tMESH:D012131\\n28602779\\t4105\\t4111\\tdeaths\\tDisease\\tMESH:D003643\\n28602779\\t4187\\t4193\\tdeaths\\tDisease\\tMESH:D003643\\n28602779\\t4298\\t4306\\tpatients\\tSpecies\\t9606\\n28602779\\t4362\\t4365\\tALK\\tGene\\t238\\n28602779\\t4393\\t4403\\tcrizotinib\\tChemical\\tMESH:D000077547\\n28602779\\t4419\\t4428\\tceritinib\\tChemical\\tMESH:C586847\\n28602779\\t4503\\t4510\\tpatient\\tSpecies\\t9606\\n\\n30413378|t|Lorlatinib in patients with ALK-positive non-small-cell lung cancer: results from a global phase 2 study.\\n30413378|a|BACKGROUND: Lorlatinib is a potent, brain-penetrant, third-generation inhibitor of ALK and ROS1 tyrosine kinases with broad coverage of ALK mutations. In a phase 1 study, activity was seen in patients with ALK-positive non-small-cell lung cancer, most of whom had CNS metastases and progression after ALK-directed therapy. We aimed to analyse the overall and intracranial antitumour activity of lorlatinib in patients with ALK-positive, advanced non-small-cell lung cancer. METHODS: In this phase 2 study, patients with histologically or cytologically ALK-positive or ROS1-positive, advanced, non-small-cell lung cancer, with or without CNS metastases, with an Eastern Cooperative Oncology Group performance status of 0, 1, or 2, and adequate end-organ function were eligible. Patients were enrolled into six different expansion cohorts (EXP1-6) on the basis of ALK and ROS1 status and previous therapy, and were given lorlatinib 100 mg orally once daily continuously in 21-day cycles. The primary endpoint was overall and intracranial tumour response by independent central review, assessed in pooled subgroups of ALK-positive patients. Analyses of activity and safety were based on the safety analysis set (ie, all patients who received at least one dose of lorlatinib) as assessed by independent central review. Patients with measurable CNS metastases at baseline by independent central review were included in the intracranial activity analyses. In this report, we present lorlatinib activity data for the ALK-positive patients (EXP1-5 only), and safety data for all treated patients (EXP1-6). This study is ongoing and is registered with ClinicalTrials.gov, number NCT01970865. FINDINGS: Between Sept 15, 2015, and Oct 3, 2016, 276 patients were enrolled: 30 who were ALK positive and treatment naive (EXP1); 59 who were ALK positive and received previous crizotinib without (n=27; EXP2) or with (n=32; EXP3A) previous chemotherapy; 28 who were ALK positive and received one previous non-crizotinib ALK tyrosine kinase inhibitor, with or without chemotherapy (EXP3B); 112 who were ALK positive with two (n=66; EXP4) or three (n=46; EXP5) previous ALK tyrosine kinase inhibitors with or without chemotherapy; and 47 who were ROS1 positive with any previous treatment (EXP6). One patient in EXP4 died before receiving lorlatinib and was excluded from the safety analysis set. In treatment-naive patients (EXP1), an objective response was achieved in 27 (90 0%; 95% CI 73 5-97 9) of 30 patients. Three patients in EXP1 had measurable baseline CNS lesions per independent central review, and objective intracranial responses were observed in two (66 7%; 95% CI 9 4-99 2). In ALK-positive patients with at least one previous ALK tyrosine kinase inhibitor (EXP2-5), objective responses were achieved in 93 (47 0%; 39 9-54 2) of 198 patients and objective intracranial response in those with measurable baseline CNS lesions in 51 (63 0%; 51 5-73 4) of 81 patients. Objective response was achieved in 41 (69 5%; 95% CI 56 1-80 8) of 59 patients who had only received previous crizotinib (EXP2-3A), nine (32 1%; 15 9-52 4) of 28 patients with one previous non-crizotinib ALK tyrosine kinase inhibitor (EXP3B), and 43 (38 7%; 29 6-48 5) of 111 patients with two or more previous ALK tyrosine kinase inhibitors (EXP4-5). Objective intracranial response was achieved in 20 (87 0%; 95% CI 66 4-97 2) of 23 patients with measurable baseline CNS lesions in EXP2-3A, five (55 6%; 21 2-86 3) of nine patients in EXP3B, and 26 (53 1%; 38 3-67 5) of 49 patients in EXP4-5. The most common treatment-related adverse events across all patients were hypercholesterolaemia (224 [81%] of 275 patients overall and 43 [16%] grade 3-4) and hypertriglyceridaemia (166 [60%] overall and 43 [16%] grade 3-4). Serious treatment-related adverse events occurred in 19 (7%) of 275 patients and seven patients (3%) permanently discontinued treatment because of treatment-related adverse events. No treatment-related deaths were reported. INTERPRETATION: Consistent with its broad ALK mutational coverage and CNS penetration, lorlatinib showed substantial overall and intracranial activity both in treatment-naive patients with ALK-positive non-small-cell lung cancer, and in those who had progressed on crizotinib, second-generation ALK tyrosine kinase inhibitors, or after up to three previous ALK tyrosine kinase inhibitors. Thus, lorlatinib could represent an effective treatment option for patients with ALK-positive non-small-cell lung cancer in first-line or subsequent therapy. FUNDING: Pfizer.\\n30413378\\t0\\t10\\tLorlatinib\\tChemical\\tMESH:C000590786\\n30413378\\t14\\t22\\tpatients\\tSpecies\\t9606\\n30413378\\t28\\t31\\tALK\\tGene\\t238\\n30413378\\t56\\t67\\tlung cancer\\tDisease\\tMESH:D008175\\n30413378\\t118\\t128\\tLorlatinib\\tChemical\\tMESH:C000590786\\n30413378\\t189\\t192\\tALK\\tGene\\t238\\n30413378\\t197\\t201\\tROS1\\tGene\\t6098\\n30413378\\t242\\t245\\tALK\\tGene\\t238\\n30413378\\t298\\t306\\tpatients\\tSpecies\\t9606\\n30413378\\t312\\t315\\tALK\\tGene\\t238\\n30413378\\t340\\t351\\tlung cancer\\tDisease\\tMESH:D008175\\n30413378\\t374\\t384\\tmetastases\\tDisease\\tMESH:D009362\\n30413378\\t407\\t410\\tALK\\tGene\\t238\\n30413378\\t501\\t511\\tlorlatinib\\tChemical\\tMESH:C000590786\\n30413378\\t515\\t523\\tpatients\\tSpecies\\t9606\\n30413378\\t529\\t532\\tALK\\tGene\\t238\\n30413378\\t567\\t578\\tlung cancer\\tDisease\\tMESH:D008175\\n30413378\\t612\\t620\\tpatients\\tSpecies\\t9606\\n30413378\\t658\\t661\\tALK\\tGene\\t238\\n30413378\\t674\\t678\\tROS1\\tGene\\t6098\\n30413378\\t714\\t725\\tlung cancer\\tDisease\\tMESH:D008175\\n30413378\\t747\\t757\\tmetastases\\tDisease\\tMESH:D009362\\n30413378\\t883\\t891\\tPatients\\tSpecies\\t9606\\n30413378\\t944\\t950\\tEXP1-6\\tGene\\t7514;1434;64328;57510;23214\\n30413378\\t968\\t971\\tALK\\tGene\\t238\\n30413378\\t976\\t980\\tROS1\\tGene\\t6098\\n30413378\\t1025\\t1035\\tlorlatinib\\tChemical\\tMESH:C000590786\\n30413378\\t1129\\t1148\\tintracranial tumour\\tDisease\\tMESH:D001932\\n30413378\\t1221\\t1224\\tALK\\tGene\\t238\\n30413378\\t1234\\t1242\\tpatients\\tSpecies\\t9606\\n30413378\\t1323\\t1331\\tpatients\\tSpecies\\t9606\\n30413378\\t1366\\t1376\\tlorlatinib\\tChemical\\tMESH:C000590786\\n30413378\\t1421\\t1429\\tPatients\\tSpecies\\t9606\\n30413378\\t1450\\t1460\\tmetastases\\tDisease\\tMESH:D009362\\n30413378\\t1583\\t1593\\tlorlatinib\\tChemical\\tMESH:C000590786\\n30413378\\t1616\\t1619\\tALK\\tGene\\t238\\n30413378\\t1629\\t1637\\tpatients\\tSpecies\\t9606\\n30413378\\t1639\\t1645\\tEXP1-5\\tGene\\t7514;1434;64328;57510\\n30413378\\t1685\\t1693\\tpatients\\tSpecies\\t9606\\n30413378\\t1695\\t1701\\tEXP1-6\\tGene\\t7514;1434;64328;57510;23214\\n30413378\\t1843\\t1851\\tpatients\\tSpecies\\t9606\\n30413378\\t1879\\t1882\\tALK\\tGene\\t238\\n30413378\\t1913\\t1917\\tEXP1\\tGene\\t7514\\n30413378\\t1932\\t1935\\tALK\\tGene\\t238\\n30413378\\t1993\\t1997\\tEXP2\\tGene\\t1434\\n30413378\\t2056\\t2059\\tALK\\tGene\\t238\\n30413378\\t2192\\t2195\\tALK\\tGene\\t238\\n30413378\\t2221\\t2225\\tEXP4\\tGene\\t64328\\n30413378\\t2243\\t2247\\tEXP5\\tGene\\t57510\\n30413378\\t2258\\t2261\\tALK\\tGene\\t238\\n30413378\\t2335\\t2339\\tROS1\\tGene\\t6098\\n30413378\\t2378\\t2382\\tEXP6\\tGene\\t23214\\n30413378\\t2389\\t2396\\tpatient\\tSpecies\\t9606\\n30413378\\t2400\\t2404\\tEXP4\\tGene\\t64328\\n30413378\\t2427\\t2437\\tlorlatinib\\tChemical\\tMESH:C000590786\\n30413378\\t2504\\t2512\\tpatients\\tSpecies\\t9606\\n30413378\\t2514\\t2518\\tEXP1\\tGene\\t7514\\n30413378\\t2594\\t2602\\tpatients\\tSpecies\\t9606\\n30413378\\t2610\\t2618\\tpatients\\tSpecies\\t9606\\n30413378\\t2622\\t2626\\tEXP1\\tGene\\t7514\\n30413378\\t2782\\t2785\\tALK\\tGene\\t238\\n30413378\\t2795\\t2803\\tpatients\\tSpecies\\t9606\\n30413378\\t2862\\t2868\\tEXP2-5\\tGene\\t1434;64328;57510\\n30413378\\t2937\\t2945\\tpatients\\tSpecies\\t9606\\n30413378\\t3059\\t3067\\tpatients\\tSpecies\\t9606\\n30413378\\t3139\\t3147\\tpatients\\tSpecies\\t9606\\n30413378\\t3179\\t3189\\tcrizotinib\\tChemical\\tMESH:D000077547\\n30413378\\t3231\\t3239\\tpatients\\tSpecies\\t9606\\n30413378\\t3262\\t3272\\tcrizotinib\\tChemical\\tMESH:D000077547\\n30413378\\t3345\\t3353\\tpatients\\tSpecies\\t9606\\n30413378\\t3412\\t3418\\tEXP4-5\\tGene\\t64328;57510\\n30413378\\t3504\\t3512\\tpatients\\tSpecies\\t9606\\n30413378\\t3553\\t3557\\tEXP2\\tGene\\t1434\\n30413378\\t3594\\t3602\\tpatients\\tSpecies\\t9606\\n30413378\\t3645\\t3653\\tpatients\\tSpecies\\t9606\\n30413378\\t3657\\t3663\\tEXP4-5\\tGene\\t64328;57510\\n30413378\\t3725\\t3733\\tpatients\\tSpecies\\t9606\\n30413378\\t3739\\t3760\\thypercholesterolaemia\\tDisease\\t\\n30413378\\t3779\\t3787\\tpatients\\tSpecies\\t9606\\n30413378\\t3824\\t3845\\thypertriglyceridaemia\\tDisease\\t\\n30413378\\t3958\\t3966\\tpatients\\tSpecies\\t9606\\n30413378\\t3977\\t3985\\tpatients\\tSpecies\\t9606\\n30413378\\t4092\\t4098\\tdeaths\\tDisease\\tMESH:D003643\\n30413378\\t4156\\t4159\\tALK\\tGene\\t238\\n30413378\\t4201\\t4211\\tlorlatinib\\tChemical\\tMESH:C000590786\\n30413378\\t4289\\t4297\\tpatients\\tSpecies\\t9606\\n30413378\\t4303\\t4306\\tALK\\tGene\\t238\\n30413378\\t4331\\t4342\\tlung cancer\\tDisease\\tMESH:D008175\\n30413378\\t4379\\t4389\\tcrizotinib\\tChemical\\tMESH:D000077547\\n30413378\\t4471\\t4474\\tALK\\tGene\\t238\\n30413378\\t4509\\t4519\\tlorlatinib\\tChemical\\tMESH:C000590786\\n30413378\\t4570\\t4578\\tpatients\\tSpecies\\t9606\\n30413378\\t4584\\t4587\\tALK\\tGene\\t238\\n30413378\\t4612\\t4623\\tlung cancer\\tDisease\\tMESH:D008175\\n\\n34534430|t|Trastuzumab Deruxtecan in HER2-Mutant Non-Small-Cell Lung Cancer.\\n34534430|a|BACKGROUND: Human epidermal growth factor receptor 2 (HER2)-targeted therapies have not been approved for patients with non-small-cell lung cancer (NSCLC). The efficacy and safety of trastuzumab deruxtecan (formerly DS-8201), a HER2 antibody-drug conjugate, in patients with HER2-mutant NSCLC have not been investigated extensively. METHODS: We conducted a multicenter, international, phase 2 study in which trastuzumab deruxtecan (6.4 mg per kilogram of body weight) was administered to patients who had metastatic HER2-mutant NSCLC that was refractory to standard treatment. The primary outcome was objective response as assessed by independent central review. Secondary outcomes included the duration of response, progression-free survival, overall survival, and safety. Biomarkers of HER2 alterations were assessed. RESULTS: A total of 91 patients were enrolled. The median duration of follow-up was 13.1 months (range, 0.7 to 29.1). Centrally confirmed objective response occurred in 55% of the patients (95% confidence interval [CI], 44 to 65). The median duration of response was 9.3 months (95% CI, 5.7 to 14.7). Median progression-free survival was 8.2 months (95% CI, 6.0 to 11.9), and median overall survival was 17.8 months (95% CI, 13.8 to 22.1). The safety profile was generally consistent with those from previous studies; grade 3 or higher drug-related adverse events occurred in 46% of patients, the most common event being neutropenia (in 19%). Adjudicated drug-related interstitial lung disease occurred in 26% of patients and resulted in death in 2 patients. Responses were observed across different HER2 mutation subtypes, as well as in patients with no detectable HER2 expression or HER2 amplification. CONCLUSIONS: Trastuzumab deruxtecan showed durable anticancer activity in patients with previously treated HER2-mutant NSCLC. The safety profile included interstitial lung disease that was fatal in two cases. Observed toxic effects were generally consistent with those in previously reported studies. (Funded by Daiichi Sankyo and AstraZeneca; DESTINY-Lung01 ClinicalTrials.gov number, NCT03505710.).\\n34534430\\t0\\t11\\tTrastuzumab\\tChemical\\tMESH:D000068878\\n34534430\\t12\\t22\\tDeruxtecan\\tChemical\\t-\\n34534430\\t26\\t30\\tHER2\\tGene\\t2064\\n34534430\\t38\\t64\\tNon-Small-Cell Lung Cancer\\tDisease\\tMESH:D002289\\n34534430\\t78\\t118\\tHuman epidermal growth factor receptor 2\\tGene\\t2064\\n34534430\\t120\\t124\\tHER2\\tGene\\t2064\\n34534430\\t172\\t180\\tpatients\\tSpecies\\t9606\\n34534430\\t186\\t212\\tnon-small-cell lung cancer\\tDisease\\tMESH:D002289\\n34534430\\t214\\t219\\tNSCLC\\tDisease\\tMESH:D002289\\n34534430\\t249\\t260\\ttrastuzumab\\tChemical\\tMESH:D000068878\\n34534430\\t261\\t271\\tderuxtecan\\tChemical\\t-\\n34534430\\t282\\t289\\tDS-8201\\tChemical\\tMESH:C000614160\\n34534430\\t294\\t298\\tHER2\\tGene\\t2064\\n34534430\\t327\\t335\\tpatients\\tSpecies\\t9606\\n34534430\\t341\\t345\\tHER2\\tGene\\t2064\\n34534430\\t353\\t358\\tNSCLC\\tDisease\\tMESH:D002289\\n34534430\\t474\\t485\\ttrastuzumab\\tChemical\\tMESH:D000068878\\n34534430\\t486\\t496\\tderuxtecan\\tChemical\\t-\\n34534430\\t554\\t562\\tpatients\\tSpecies\\t9606\\n34534430\\t582\\t586\\tHER2\\tGene\\t2064\\n34534430\\t594\\t599\\tNSCLC\\tDisease\\tMESH:D002289\\n34534430\\t854\\t858\\tHER2\\tGene\\t2064\\n34534430\\t909\\t917\\tpatients\\tSpecies\\t9606\\n34534430\\t1066\\t1074\\tpatients\\tSpecies\\t9606\\n34534430\\t1469\\t1477\\tpatients\\tSpecies\\t9606\\n34534430\\t1507\\t1518\\tneutropenia\\tDisease\\tMESH:D009503\\n34534430\\t1567\\t1579\\tlung disease\\tDisease\\tMESH:D008171\\n34534430\\t1599\\t1607\\tpatients\\tSpecies\\t9606\\n34534430\\t1624\\t1629\\tdeath\\tDisease\\tMESH:D003643\\n34534430\\t1635\\t1643\\tpatients\\tSpecies\\t9606\\n34534430\\t1686\\t1690\\tHER2\\tGene\\t2064\\n34534430\\t1724\\t1732\\tpatients\\tSpecies\\t9606\\n34534430\\t1752\\t1756\\tHER2\\tGene\\t2064\\n34534430\\t1771\\t1775\\tHER2\\tGene\\t2064\\n34534430\\t1804\\t1815\\tTrastuzumab\\tChemical\\tMESH:D000068878\\n34534430\\t1816\\t1826\\tderuxtecan\\tChemical\\t-\\n34534430\\t1865\\t1873\\tpatients\\tSpecies\\t9606\\n34534430\\t1898\\t1902\\tHER2\\tGene\\t2064\\n34534430\\t1910\\t1915\\tNSCLC\\tDisease\\tMESH:D002289\\n34534430\\t1958\\t1970\\tlung disease\\tDisease\\tMESH:D008171\\n\\n'"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "string"
            }
          },
          "metadata": {},
          "execution_count": 29
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "print(result)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "oLz-OU6FFRYy",
        "outputId": "0f5f8358-b7dd-4d1a-edff-35a2f205b740"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "25765070|t|Cancer immunology. Mutational landscape determines sensitivity to PD-1 blockade in non-small cell lung cancer.\n",
            "25765070|a|Immune checkpoint inhibitors, which unleash a patient's own T cells to kill tumors, are revolutionizing cancer treatment. To unravel the genomic determinants of response to this therapy, we used whole-exome sequencing of non-small cell lung cancers treated with pembrolizumab, an antibody targeting programmed cell death-1 (PD-1). In two independent cohorts, higher nonsynonymous mutation burden in tumors was associated with improved objective response, durable clinical benefit, and progression-free survival. Efficacy also correlated with the molecular smoking signature, higher neoantigen burden, and DNA repair pathway mutations; each factor was also associated with mutation burden. In one responder, neoantigen-specific CD8+ T cell responses paralleled tumor regression, suggesting that anti-PD-1 therapy enhances neoantigen-specific T cell reactivity. Our results suggest that the genomic landscape of lung cancers shapes response to anti-PD-1 therapy. \n",
            "25765070\t0\t6\tCancer\tDisease\tMESH:D009369\n",
            "25765070\t157\t164\tpatient\tSpecies\t9606\n",
            "25765070\t187\t193\ttumors\tDisease\tMESH:D009369\n",
            "25765070\t347\t359\tlung cancers\tDisease\tMESH:D008175\n",
            "25765070\t373\t386\tpembrolizumab\tChemical\tMESH:C582435\n",
            "25765070\t510\t516\ttumors\tDisease\tMESH:D009369\n",
            "25765070\t843\t849\tT cell\tCellLine\tT cell\n",
            "25765070\t871\t876\ttumor\tDisease\tMESH:D009369\n",
            "25765070\t952\t958\tT cell\tCellLine\tT cell\n",
            "25765070\t1021\t1033\tlung cancers\tDisease\tMESH:D008175\n",
            "\n",
            "25891174|t|Pembrolizumab for the treatment of non-small-cell lung cancer.\n",
            "25891174|a|BACKGROUND: We assessed the efficacy and safety of programmed cell death 1 (PD-1) inhibition with pembrolizumab in patients with advanced non-small-cell lung cancer enrolled in a phase 1 study. We also sought to define and validate an expression level of the PD-1 ligand 1 (PD-L1) that is associated with the likelihood of clinical benefit. METHODS: We assigned 495 patients receiving pembrolizumab (at a dose of either 2 mg or 10 mg per kilogram of body weight every 3 weeks or 10 mg per kilogram every 2 weeks) to either a training group (182 patients) or a validation group (313 patients). We assessed PD-L1 expression in tumor samples using immunohistochemical analysis, with results reported as the percentage of neoplastic cells with staining for membranous PD-L1 (proportion score). Response was assessed every 9 weeks by central review. RESULTS: Common side effects that were attributed to pembrolizumab were fatigue, pruritus, and decreased appetite, with no clear difference according to dose or schedule. Among all the patients, the objective response rate was 19.4%, and the median duration of response was 12.5 months. The median duration of progression-free survival was 3.7 months, and the median duration of overall survival was 12.0 months. PD-L1 expression in at least 50% of tumor cells was selected as the cutoff from the training group. Among patients with a proportion score of at least 50% in the validation group, the response rate was 45.2%. Among all the patients with a proportion score of at least 50%, median progression-free survival was 6.3 months; median overall survival was not reached. CONCLUSIONS: Pembrolizumab had an acceptable side-effect profile and showed antitumor activity in patients with advanced non-small-cell lung cancer. PD-L1 expression in at least 50% of tumor cells correlated with improved efficacy of pembrolizumab. (Funded by Merck; KEYNOTE-001 ClinicalTrials.gov number, NCT01295827.).\n",
            "25891174\t0\t13\tPembrolizumab\tChemical\tMESH:C582435\n",
            "25891174\t50\t61\tlung cancer\tDisease\tMESH:D008175\n",
            "25891174\t114\t137\tprogrammed cell death 1\tGene\t5133\n",
            "25891174\t139\t143\tPD-1\tGene\t5133\n",
            "25891174\t161\t174\tpembrolizumab\tChemical\tMESH:C582435\n",
            "25891174\t178\t186\tpatients\tSpecies\t9606\n",
            "25891174\t216\t227\tlung cancer\tDisease\tMESH:D008175\n",
            "25891174\t322\t335\tPD-1 ligand 1\tGene\t29126\n",
            "25891174\t337\t342\tPD-L1\tGene\t29126\n",
            "25891174\t429\t437\tpatients\tSpecies\t9606\n",
            "25891174\t448\t461\tpembrolizumab\tChemical\tMESH:C582435\n",
            "25891174\t608\t616\tpatients\tSpecies\t9606\n",
            "25891174\t645\t653\tpatients\tSpecies\t9606\n",
            "25891174\t668\t673\tPD-L1\tGene\t29126\n",
            "25891174\t688\t693\ttumor\tDisease\tMESH:D009369\n",
            "25891174\t827\t832\tPD-L1\tGene\t29126\n",
            "25891174\t961\t974\tpembrolizumab\tChemical\tMESH:C582435\n",
            "25891174\t980\t987\tfatigue\tDisease\tMESH:D005221\n",
            "25891174\t989\t997\tpruritus\tDisease\tMESH:D011537\n",
            "25891174\t1093\t1101\tpatients\tSpecies\t9606\n",
            "25891174\t1321\t1326\tPD-L1\tGene\t29126\n",
            "25891174\t1357\t1362\ttumor\tDisease\tMESH:D009369\n",
            "25891174\t1427\t1435\tpatients\tSpecies\t9606\n",
            "25891174\t1544\t1552\tpatients\tSpecies\t9606\n",
            "25891174\t1697\t1710\tPembrolizumab\tChemical\tMESH:C582435\n",
            "25891174\t1782\t1790\tpatients\tSpecies\t9606\n",
            "25891174\t1820\t1831\tlung cancer\tDisease\tMESH:D008175\n",
            "25891174\t1833\t1838\tPD-L1\tGene\t29126\n",
            "25891174\t1869\t1874\ttumor\tDisease\tMESH:D009369\n",
            "25891174\t1918\t1931\tpembrolizumab\tChemical\tMESH:C582435\n",
            "\n",
            "26027660|t|LKB1 gene inactivation does not sensitize non-small cell lung cancer cells to mTOR inhibitors in vitro.\n",
            "26027660|a|AIM: Previous study has shown that endometrial cancers with LKB1 inactivation are highly responsive to mTOR inhibitors. In this study we examined the effect of LKB1 gene status on mTOR inhibitor responses in non-small cell lung cancer (NSCLC) cells. METHODS: Lung cancer cell lines Calu-1, H460, H1299, H1792, and A549 were treated with the mTOR inhibitors rapamycin or everolimus (RAD001). The mTOR activity was evaluated by measuring the phosphorylation of 4EBP1 and S6K, the two primary mTOR substrates. Cells proliferation was measured by MTS or sulforhodamine B assays. RESULTS: The basal level of mTOR activity in LKB1 mutant A549 and H460 cells was significantly higher than that in LKB1 wild-type Calu-1 and H1792 cells. However, the LKB1 mutant A549 and H460 cells were not more sensitive to the mTOR inhibitors than the LKB1 wild-type Calu-1 and H1792 cells. Moreover, knockdown of LKB1 gene in H1299 cells did not increase the sensitivity to the mTOR inhibitors. Treatment with rapamycin or RAD001 significantly increased the phosphorylation of AKT in both LKB1 wild-type and LKB1 mutant NSCLC cells, which was attenuated by the PI3K inhibitor LY294002. Furthermore, RAD001 combined with LY294002 markedly enhanced the growth inhibition on LKB1 wild-type H1792 cells and LKB1 mutant A549 cells. CONCLUSION: LKB1 gene inactivation in NSCLC cells does not increase the sensitivity to the mTOR inhibitors. The negative feedback activation of AKT by mTOR inhibition may contribute to the resistance of NSCLC cells to mTOR inhibitors.\n",
            "26027660\t0\t4\tLKB1\tGene\t6794\n",
            "26027660\t57\t68\tlung cancer\tDisease\tMESH:D008175\n",
            "26027660\t78\t82\tmTOR\tGene\t2475\n",
            "26027660\t139\t158\tendometrial cancers\tDisease\tMESH:D016889\n",
            "26027660\t164\t168\tLKB1\tGene\t6794\n",
            "26027660\t207\t211\tmTOR\tGene\t2475\n",
            "26027660\t264\t268\tLKB1\tGene\t6794\n",
            "26027660\t284\t288\tmTOR\tGene\t2475\n",
            "26027660\t327\t338\tlung cancer\tDisease\tMESH:D008175\n",
            "26027660\t340\t345\tNSCLC\tDisease\tMESH:D002289\n",
            "26027660\t363\t374\tLung cancer\tDisease\tMESH:D008175\n",
            "26027660\t394\t398\tH460\tCellLine\tCVCL:0459\n",
            "26027660\t400\t405\tH1299\tCellLine\tCVCL:0060\n",
            "26027660\t407\t412\tH1792\tCellLine\tCVCL:1495\n",
            "26027660\t418\t422\tA549\tCellLine\tCVCL:0023\n",
            "26027660\t445\t449\tmTOR\tGene\t2475\n",
            "26027660\t461\t470\trapamycin\tChemical\tMESH:D020123\n",
            "26027660\t474\t484\teverolimus\tChemical\tMESH:D000068338\n",
            "26027660\t499\t503\tmTOR\tGene\t2475\n",
            "26027660\t563\t568\t4EBP1\tGene\t1978\n",
            "26027660\t573\t576\tS6K\tGene\t6198\n",
            "26027660\t594\t598\tmTOR\tGene\t2475\n",
            "26027660\t654\t670\tsulforhodamine B\tChemical\tMESH:C022027\n",
            "26027660\t707\t711\tmTOR\tGene\t2475\n",
            "26027660\t724\t728\tLKB1\tGene\t6794\n",
            "26027660\t736\t740\tA549\tCellLine\tCVCL:0023\n",
            "26027660\t745\t749\tH460\tCellLine\tCVCL:0459\n",
            "26027660\t794\t798\tLKB1\tGene\t6794\n",
            "26027660\t820\t825\tH1792\tCellLine\tCVCL:1495\n",
            "26027660\t846\t850\tLKB1\tGene\t6794\n",
            "26027660\t858\t862\tA549\tCellLine\tCVCL:0023\n",
            "26027660\t867\t871\tH460\tCellLine\tCVCL:0459\n",
            "26027660\t909\t913\tmTOR\tGene\t2475\n",
            "26027660\t934\t938\tLKB1\tGene\t6794\n",
            "26027660\t960\t965\tH1792\tCellLine\tCVCL:1495\n",
            "26027660\t996\t1000\tLKB1\tGene\t6794\n",
            "26027660\t1009\t1014\tH1299\tCellLine\tCVCL:0060\n",
            "26027660\t1061\t1065\tmTOR\tGene\t2475\n",
            "26027660\t1093\t1102\trapamycin\tChemical\tMESH:D020123\n",
            "26027660\t1106\t1112\tRAD001\tChemical\tMESH:D000068338\n",
            "26027660\t1160\t1163\tAKT\tGene\t207\n",
            "26027660\t1172\t1176\tLKB1\tGene\t6794\n",
            "26027660\t1191\t1195\tLKB1\tGene\t6794\n",
            "26027660\t1203\t1208\tNSCLC\tDisease\tMESH:D002289\n",
            "26027660\t1259\t1267\tLY294002\tChemical\tMESH:C085911\n",
            "26027660\t1303\t1311\tLY294002\tChemical\tMESH:C085911\n",
            "26027660\t1355\t1359\tLKB1\tGene\t6794\n",
            "26027660\t1370\t1375\tH1792\tCellLine\tCVCL:1495\n",
            "26027660\t1386\t1390\tLKB1\tGene\t6794\n",
            "26027660\t1398\t1402\tA549\tCellLine\tCVCL:0023\n",
            "26027660\t1422\t1426\tLKB1\tGene\t6794\n",
            "26027660\t1448\t1453\tNSCLC\tDisease\tMESH:D002289\n",
            "26027660\t1501\t1505\tmTOR\tGene\t2475\n",
            "26027660\t1554\t1557\tAKT\tGene\t207\n",
            "26027660\t1561\t1565\tmTOR\tGene\t2475\n",
            "26027660\t1613\t1618\tNSCLC\tDisease\tMESH:D002289\n",
            "26027660\t1628\t1632\tmTOR\tGene\t2475\n",
            "\n",
            "26124334|t|Effectiveness of Tyrosine Kinase Inhibitors in Japanese Patients with Non-small Cell Lung Cancer Harboring Minor Epidermal Growth Factor Receptor Mutations: Results from a Multicenter Retrospective Study (HANSHIN Oncology Group 0212).\n",
            "26124334|a|AIM: Non-small cell lung cancer (NSCLC) with minor mutations in the epidermal growth factor receptor (EGFR) gene, except for the common 15 base-pair deletions in exon 19 and the L858R mutation in exon 21, is rare, and only few data exist on this patient population. The aim of the present study was to describe the clinical characteristics and to clarify the efficacy of EGFR-tyrosine kinase inhibitors (TKIs) in patients with NSCLC harboring minor mutations of the EGFR gene. PATIENTS AND METHODS: This was a multicenter, retrospective study that analyzed specimens from patients with NSCLC who had minor EGFR gene mutations and were treated with EGFR-TKIs between June 2002 and March 2012. RESULTS: Out of 56 patients with minor mutations of the EGFR gene, 44 were treated with either gefitinib or erlotinib. Mutation sites were G719X in exon 18 (n=35), L861Q in exon 21 (n=11), and G874S in exon 21 (n=1). Three patients had both the G719S and the L861Q mutation. The response rate to TKI treatment was 29.5%, and the disease control rate was 63.6%. The median progression-free survival (PFS) was 6.7 months [95% confidence interval (CI)=2.06-8.66 months]. The median PFS was 7.2 months (95% CI=4.23-12.3 months) in 32 patients who received first- or second-line treatment with EGFR-TKIs, whereas the median PFS was 1.57 months (95% CI=0.73-3.8 months) in 12 patients treated with EGFR-TKIs as a third-line or later treatment. In multivariate Cox analysis, erlotinib therapy was associated with a longer PFS than gefitinib (p=0.025). CONCLUSION: Patients with NSCLC harboring minor mutations of the EGFR gene exhibited a modest response to EGFR-TKI treatment. Treatment with first-generation EGFR-TKIs, in particular erlotinib, may be considered a first- or second-line option for patients with NSCLC with minor EGFR mutations.\n",
            "26124334\t56\t64\tPatients\tSpecies\t9606\n",
            "26124334\t70\t96\tNon-small Cell Lung Cancer\tDisease\tMESH:D002289\n",
            "26124334\t113\t145\tEpidermal Growth Factor Receptor\tGene\t1956\n",
            "26124334\t240\t266\tNon-small cell lung cancer\tDisease\tMESH:D002289\n",
            "26124334\t268\t273\tNSCLC\tDisease\tMESH:D002289\n",
            "26124334\t303\t335\tepidermal growth factor receptor\tGene\t1956\n",
            "26124334\t337\t341\tEGFR\tGene\t1956\n",
            "26124334\t413\t418\tL858R\tProteinMutation\ttmVar:p|SUB|L|858|R;HGVS:p.L858R;VariantGroup:0;CorrespondingGene:1956;RS#:121434568;CA#:126713\n",
            "26124334\t481\t488\tpatient\tSpecies\t9606\n",
            "26124334\t606\t610\tEGFR\tGene\t1956\n",
            "26124334\t648\t656\tpatients\tSpecies\t9606\n",
            "26124334\t662\t667\tNSCLC\tDisease\tMESH:D002289\n",
            "26124334\t701\t705\tEGFR\tGene\t1956\n",
            "26124334\t712\t720\tPATIENTS\tSpecies\t9606\n",
            "26124334\t807\t815\tpatients\tSpecies\t9606\n",
            "26124334\t821\t826\tNSCLC\tDisease\tMESH:D002289\n",
            "26124334\t841\t845\tEGFR\tGene\t1956\n",
            "26124334\t883\t887\tEGFR\tGene\t1956\n",
            "26124334\t946\t954\tpatients\tSpecies\t9606\n",
            "26124334\t983\t987\tEGFR\tGene\t1956\n",
            "26124334\t1022\t1031\tgefitinib\tChemical\tMESH:D000077156\n",
            "26124334\t1035\t1044\terlotinib\tChemical\tMESH:D000069347\n",
            "26124334\t1066\t1071\tG719X\tProteinMutation\ttmVar:p|SUB|G|719|X;HGVS:p.G719X;VariantGroup:1;CorrespondingGene:1956;RS#:28929495\n",
            "26124334\t1091\t1096\tL861Q\tProteinMutation\ttmVar:p|SUB|L|861|Q;HGVS:p.L861Q;VariantGroup:2;CorrespondingGene:1956;RS#:121913444;CA#:176021\n",
            "26124334\t1120\t1125\tG874S\tProteinMutation\ttmVar:p|SUB|G|874|S;HGVS:p.G874S;VariantGroup:3;CorrespondingGene:1956\n",
            "26124334\t1150\t1158\tpatients\tSpecies\t9606\n",
            "26124334\t1172\t1177\tG719S\tProteinMutation\ttmVar:p|SUB|G|719|S;HGVS:p.G719S;VariantGroup:1;CorrespondingGene:1956;RS#:28929495;CA#:126722\n",
            "26124334\t1186\t1191\tL861Q\tProteinMutation\ttmVar:p|SUB|L|861|Q;HGVS:p.L861Q;VariantGroup:2;CorrespondingGene:1956;RS#:121913444;CA#:176021\n",
            "26124334\t1457\t1465\tpatients\tSpecies\t9606\n",
            "26124334\t1516\t1520\tEGFR\tGene\t1956\n",
            "26124334\t1597\t1605\tpatients\tSpecies\t9606\n",
            "26124334\t1619\t1623\tEGFR\tGene\t1956\n",
            "26124334\t1695\t1704\terlotinib\tChemical\tMESH:D000069347\n",
            "26124334\t1751\t1760\tgefitinib\tChemical\tMESH:D000077156\n",
            "26124334\t1784\t1792\tPatients\tSpecies\t9606\n",
            "26124334\t1798\t1803\tNSCLC\tDisease\tMESH:D002289\n",
            "26124334\t1837\t1841\tEGFR\tGene\t1956\n",
            "26124334\t1878\t1882\tEGFR\tGene\t1956\n",
            "26124334\t1930\t1934\tEGFR\tGene\t1956\n",
            "26124334\t1955\t1964\terlotinib\tChemical\tMESH:D000069347\n",
            "26124334\t2019\t2027\tpatients\tSpecies\t9606\n",
            "26124334\t2033\t2038\tNSCLC\tDisease\tMESH:D002289\n",
            "26124334\t2050\t2054\tEGFR\tGene\t1956\n",
            "\n",
            "26209642|t|Value of KRAS as prognostic or predictive marker in NSCLC: results from the TAILOR trial.\n",
            "26209642|a|BACKGROUND: The prognostic and predictive role of KRAS mutations in advanced nonsmall-cell lung cancer (NSCLC) is still unclear. TAILOR prospectively assessed the prognostic and predictive value of KRAS mutations in NSCLC patients treated with erlotinib or docetaxel in second line. PATIENTS AND METHODS: NSCLC patients from 52 Italian hospitals were genotyped for KRAS and EGFR mutational status in two independent laboratories. Wild-type EGFR patients (N = 218) received first-line platinum-based chemotherapy and were randomly allocated at progression to erlotinib or docetaxel. Overall survival (OS) according to KRAS mutational status was the primary end point. RESULTS: KRAS mutations were present in 23% of TAILOR randomized cases. The presence of a KRAS mutation did not adversely affect progression-free (PFS) or overall (OS) survival [hazard ratio (HR) PFS = 1.01, 95% confidence interval (CI) 0.71-1.41, P = 0.977; OS = 1.24, 95% CI 0.87-1.77, P = 0.233], nor influenced treatment outcome (test for interaction: OS P = 0.965; PFS P = 0.417). Patients randomized to docetaxel treatment experienced longer survival independently from the KRAS mutational status of their tumors (HR: mutated KRAS 0.81, 95% CI 0.45-1.47; wild-type KRAS 0.79, 95% CI 0.57-1.10). CONCLUSION: In TAILOR, KRAS was neither prognostic nor predictive of benefit for either docetaxel or erlotinib. Docetaxel remains superior independently from KRAS status for second-line treatment in EGFR wild-type advanced NSCLC patients. CLINICAL TRIAL REGISTRATION: NCT00637910.\n",
            "26209642\t9\t13\tKRAS\tGene\t3845\n",
            "26209642\t52\t57\tNSCLC\tDisease\tMESH:D002289\n",
            "26209642\t140\t144\tKRAS\tGene\t3845\n",
            "26209642\t167\t192\tnonsmall-cell lung cancer\tDisease\tMESH:D002289\n",
            "26209642\t194\t199\tNSCLC\tDisease\tMESH:D002289\n",
            "26209642\t288\t292\tKRAS\tGene\t3845\n",
            "26209642\t306\t311\tNSCLC\tDisease\tMESH:D002289\n",
            "26209642\t312\t320\tpatients\tSpecies\t9606\n",
            "26209642\t334\t343\terlotinib\tChemical\tMESH:D000069347\n",
            "26209642\t347\t356\tdocetaxel\tChemical\tMESH:D000077143\n",
            "26209642\t373\t381\tPATIENTS\tSpecies\t9606\n",
            "26209642\t395\t400\tNSCLC\tDisease\tMESH:D002289\n",
            "26209642\t401\t409\tpatients\tSpecies\t9606\n",
            "26209642\t455\t459\tKRAS\tGene\t3845\n",
            "26209642\t464\t468\tEGFR\tGene\t1956\n",
            "26209642\t530\t534\tEGFR\tGene\t1956\n",
            "26209642\t535\t543\tpatients\tSpecies\t9606\n",
            "26209642\t574\t582\tplatinum\tChemical\tMESH:D010984\n",
            "26209642\t648\t657\terlotinib\tChemical\tMESH:D000069347\n",
            "26209642\t661\t670\tdocetaxel\tChemical\tMESH:D000077143\n",
            "26209642\t707\t711\tKRAS\tGene\t3845\n",
            "26209642\t766\t770\tKRAS\tGene\t3845\n",
            "26209642\t847\t851\tKRAS\tGene\t3845\n",
            "26209642\t1143\t1151\tPatients\tSpecies\t9606\n",
            "26209642\t1166\t1175\tdocetaxel\tChemical\tMESH:D000077143\n",
            "26209642\t1237\t1241\tKRAS\tGene\t3845\n",
            "26209642\t1269\t1275\ttumors\tDisease\tMESH:D009369\n",
            "26209642\t1289\t1293\tKRAS\tGene\t3845\n",
            "26209642\t1328\t1332\tKRAS\tGene\t3845\n",
            "26209642\t1381\t1385\tKRAS\tGene\t3845\n",
            "26209642\t1446\t1455\tdocetaxel\tChemical\tMESH:D000077143\n",
            "26209642\t1459\t1468\terlotinib\tChemical\tMESH:D000069347\n",
            "26209642\t1470\t1479\tDocetaxel\tChemical\tMESH:D000077143\n",
            "26209642\t1516\t1520\tKRAS\tGene\t3845\n",
            "26209642\t1557\t1561\tEGFR\tGene\t1956\n",
            "26209642\t1581\t1586\tNSCLC\tDisease\tMESH:D002289\n",
            "26209642\t1587\t1595\tpatients\tSpecies\t9606\n",
            "\n",
            "26287849|t|Vemurafenib in Multiple Nonmelanoma Cancers with BRAF V600 Mutations.\n",
            "26287849|a|BACKGROUND: BRAF V600 mutations occur in various nonmelanoma cancers. We undertook a histology-independent phase 2 \"basket\" study of vemurafenib in BRAF V600 mutation-positive nonmelanoma cancers. METHODS: We enrolled patients in six prespecified cancer cohorts; patients with all other tumor types were enrolled in a seventh cohort. A total of 122 patients with BRAF V600 mutation-positive cancer were treated, including 27 patients with colorectal cancer who received vemurafenib and cetuximab. The primary end point was the response rate; secondary end points included progression-free and overall survival. RESULTS: In the cohort with non-small-cell lung cancer, the response rate was 42% (95% confidence interval [CI], 20 to 67) and median progression-free survival was 7.3 months (95% CI, 3.5 to 10.8). In the cohort with Erdheim-Chester disease or Langerhans'-cell histiocytosis, the response rate was 43% (95% CI, 18 to 71); the median treatment duration was 5.9 months (range, 0.6 to 18.6), and no patients had disease progression during therapy. There were anecdotal responses among patients with pleomorphic xanthoastrocytoma, anaplastic thyroid cancer, cholangiocarcinoma, salivary-duct cancer, ovarian cancer, and clear-cell sarcoma and among patients with colorectal cancer who received vemurafenib and cetuximab. Safety was similar to that in prior studies of vemurafenib for melanoma. CONCLUSIONS: BRAF V600 appears to be a targetable oncogene in some, but not all, nonmelanoma cancers. Preliminary vemurafenib activity was observed in non-small-cell lung cancer and in Erdheim-Chester disease and Langerhans'-cell histiocytosis. The histologic context is an important determinant of response in BRAF V600-mutated cancers. (Funded by F. Hoffmann-La Roche/Genentech; ClinicalTrials.gov number, NCT01524978.).\n",
            "26287849\t0\t11\tVemurafenib\tChemical\tMESH:D000077484\n",
            "26287849\t15\t43\tMultiple Nonmelanoma Cancers\tDisease\tMESH:D009369\n",
            "26287849\t49\t53\tBRAF\tGene\t673\n",
            "26287849\t82\t86\tBRAF\tGene\t673\n",
            "26287849\t119\t138\tnonmelanoma cancers\tDisease\tMESH:D009369\n",
            "26287849\t203\t214\tvemurafenib\tChemical\tMESH:D000077484\n",
            "26287849\t218\t222\tBRAF\tGene\t673\n",
            "26287849\t246\t265\tnonmelanoma cancers\tDisease\tMESH:D009369\n",
            "26287849\t288\t296\tpatients\tSpecies\t9606\n",
            "26287849\t317\t323\tcancer\tDisease\tMESH:D009369\n",
            "26287849\t333\t341\tpatients\tSpecies\t9606\n",
            "26287849\t357\t362\ttumor\tDisease\tMESH:D009369\n",
            "26287849\t419\t427\tpatients\tSpecies\t9606\n",
            "26287849\t433\t437\tBRAF\tGene\t673\n",
            "26287849\t461\t467\tcancer\tDisease\tMESH:D009369\n",
            "26287849\t495\t503\tpatients\tSpecies\t9606\n",
            "26287849\t509\t526\tcolorectal cancer\tDisease\tMESH:D015179\n",
            "26287849\t540\t551\tvemurafenib\tChemical\tMESH:D000077484\n",
            "26287849\t556\t565\tcetuximab\tChemical\tMESH:D000068818\n",
            "26287849\t724\t735\tlung cancer\tDisease\tMESH:D008175\n",
            "26287849\t898\t921\tErdheim-Chester disease\tDisease\tMESH:D031249\n",
            "26287849\t1077\t1085\tpatients\tSpecies\t9606\n",
            "26287849\t1163\t1171\tpatients\tSpecies\t9606\n",
            "26287849\t1177\t1233\tpleomorphic xanthoastrocytoma, anaplastic thyroid cancer\tDisease\tMESH:D065646\n",
            "26287849\t1235\t1253\tcholangiocarcinoma\tDisease\tMESH:D018281\n",
            "26287849\t1255\t1275\tsalivary-duct cancer\tDisease\tMESH:D009369\n",
            "26287849\t1277\t1291\tovarian cancer\tDisease\tMESH:D010051\n",
            "26287849\t1297\t1315\tclear-cell sarcoma\tDisease\tMESH:D018227\n",
            "26287849\t1326\t1334\tpatients\tSpecies\t9606\n",
            "26287849\t1340\t1357\tcolorectal cancer\tDisease\tMESH:D015179\n",
            "26287849\t1371\t1382\tvemurafenib\tChemical\tMESH:D000077484\n",
            "26287849\t1387\t1396\tcetuximab\tChemical\tMESH:D000068818\n",
            "26287849\t1445\t1456\tvemurafenib\tChemical\tMESH:D000077484\n",
            "26287849\t1461\t1469\tmelanoma\tDisease\tMESH:D008545\n",
            "26287849\t1484\t1488\tBRAF\tGene\t673\n",
            "26287849\t1552\t1571\tnonmelanoma cancers\tDisease\tMESH:D009369\n",
            "26287849\t1637\t1648\tlung cancer\tDisease\tMESH:D008175\n",
            "26287849\t1656\t1679\tErdheim-Chester disease\tDisease\tMESH:D031249\n",
            "26287849\t1782\t1786\tBRAF\tGene\t673\n",
            "26287849\t1800\t1807\tcancers\tDisease\tMESH:D009369\n",
            "\n",
            "26439803|t|Low EGFR/MET ratio is associated with resistance to EGFR inhibitors in non-small cell lung cancer.\n",
            "26439803|a|PURPOSE: Although activating mutations in the epidermal growth factor receptor (EGFR) gene are predictive markers for response to EGFR inhibitors, 30-40% of EGFR-mutant non-small cell lung cancer (NSCLC) patients are de novo non-responders. Hence, we sought to explore additional biomarkers of response. METHODS: We conducted a prospective pilot study to characterize the expression and/or activation of key receptor tyrosine kinases (RTKs) in stage IIIB-IV NSCLC tumors. A total of 37 patients were enrolled and 34 underwent EGFR inhibitor treatment. RESULTS: As expected, patients bearing activating EGFR mutations showed increased progression free survival (PFS) compared to patients with wild-type EGFR status (9.3 vs 1.4 months, p = 0.0629). Analysis of baseline tumor RTK profiles revealed that, regardless of EGFR mutation status, higher levels of EGFR relative to MET correlated with longer PFS. At multiple EGFR/MET ratio cut-offs, including 1, 2 and 3, median PFS according to below vs. above cut-offs were 0.4 vs. 6.1 (p = 0.0001), 0.5 vs. 9.3 (p = 0.0006) and 1.0 vs. 11.2 months (p = 0.0008), respectively. CONCLUSION: The EGFR/MET ratio measured in tumors at baseline may help identify NSCLC patients most likely to benefit from prolonged PFS when treated with EGFR inhibitors.\n",
            "26439803\t4\t8\tEGFR\tGene\t1956\n",
            "26439803\t9\t12\tMET\tGene\t79811\n",
            "26439803\t52\t56\tEGFR\tGene\t1956\n",
            "26439803\t86\t97\tlung cancer\tDisease\tMESH:D008175\n",
            "26439803\t145\t177\tepidermal growth factor receptor\tGene\t1956\n",
            "26439803\t179\t183\tEGFR\tGene\t1956\n",
            "26439803\t229\t233\tEGFR\tGene\t1956\n",
            "26439803\t256\t260\tEGFR\tGene\t1956\n",
            "26439803\t283\t294\tlung cancer\tDisease\tMESH:D008175\n",
            "26439803\t303\t311\tpatients\tSpecies\t9606\n",
            "26439803\t563\t569\ttumors\tDisease\tMESH:D009369\n",
            "26439803\t585\t593\tpatients\tSpecies\t9606\n",
            "26439803\t625\t629\tEGFR\tGene\t1956\n",
            "26439803\t673\t681\tpatients\tSpecies\t9606\n",
            "26439803\t701\t705\tEGFR\tGene\t1956\n",
            "26439803\t777\t785\tpatients\tSpecies\t9606\n",
            "26439803\t801\t805\tEGFR\tGene\t1956\n",
            "26439803\t867\t872\ttumor\tDisease\tMESH:D009369\n",
            "26439803\t915\t919\tEGFR\tGene\t1956\n",
            "26439803\t954\t958\tEGFR\tGene\t1956\n",
            "26439803\t971\t974\tMET\tGene\t79811\n",
            "26439803\t1015\t1019\tEGFR\tGene\t1956\n",
            "26439803\t1020\t1023\tMET\tGene\t79811\n",
            "26439803\t1235\t1239\tEGFR\tGene\t1956\n",
            "26439803\t1240\t1243\tMET\tGene\t79811\n",
            "26439803\t1262\t1268\ttumors\tDisease\tMESH:D009369\n",
            "26439803\t1305\t1313\tpatients\tSpecies\t9606\n",
            "26439803\t1374\t1378\tEGFR\tGene\t1956\n",
            "\n",
            "26515464|t|In vitro modeling to determine mutation specificity of EGFR tyrosine kinase inhibitors against clinically relevant EGFR mutants in non-small-cell lung cancer.\n",
            "26515464|a|EGFR mutated lung cancer accounts for a significant subgroup of non-small-cell lung cancer (NSCLC). Over the last decade, multiple EGFR tyrosine kinase inhibitors (EGFR-TKIs) have been developed to target mutated EGFR. However, there is little information regarding mutation specific potency of EGFR-TKIs against various types of EGFR mutations. The purpose of this study is to establish an in vitro model to determine the \"therapeutic window\" of EGFR-TKIs against various types of EGFR mutations, including EGFR exon 20 insertion mutations. The potency of 1st (erlotinib), 2nd (afatinib) and 3rd (osimertinib and rociletinib) generation EGFR-TKIs was compared in vitro for human lung cancer cell lines and Ba/F3 cells, which exogenously express mutated or wild type EGFR. An in vitro model of mutation specificity was created by calculating the ratio of IC50 values between mutated and wild type EGFR. The in vitro model identified a wide therapeutic window of afatinib for exon 19 deletions and L858R and of osimertinib and rociletinib for T790M positive mutations. The results obtained with our models matched well with previously reported preclinical and clinical data. Interestingly, for EGFR exon 20 insertion mutations, most of which are known to be resistant to 1st and 2nd generation EGFR-TKIS, osimertinib was potent and presented a wide therapeutic window. To our knowledge, this is the first report that has identified the therapeutic window of osimertinib for EGFR exon 20 insertion mutations. In conclusion, this model will provide a preclinical rationale for proper selection of EGFR-TKIs against clinically-relevant EGFR mutations. \n",
            "26515464\t115\t119\tEGFR\tGene\t1956\n",
            "26515464\t146\t157\tlung cancer\tDisease\tMESH:D008175\n",
            "26515464\t159\t163\tEGFR\tGene\t1956\n",
            "26515464\t172\t183\tlung cancer\tDisease\tMESH:D008175\n",
            "26515464\t238\t249\tlung cancer\tDisease\tMESH:D008175\n",
            "26515464\t323\t327\tEGFR\tGene\t1956\n",
            "26515464\t372\t376\tEGFR\tGene\t1956\n",
            "26515464\t454\t458\tEGFR\tGene\t1956\n",
            "26515464\t489\t493\tEGFR\tGene\t1956\n",
            "26515464\t606\t610\tEGFR\tGene\t1956\n",
            "26515464\t641\t645\tEGFR\tGene\t1956\n",
            "26515464\t667\t671\tEGFR\tGene\t1956\n",
            "26515464\t721\t730\terlotinib\tChemical\tMESH:D000069347\n",
            "26515464\t738\t746\tafatinib\tChemical\tMESH:D000077716\n",
            "26515464\t757\t768\tosimertinib\tChemical\t-\n",
            "26515464\t773\t784\trociletinib\tChemical\tMESH:C000589977\n",
            "26515464\t797\t801\tEGFR\tGene\t1956\n",
            "26515464\t833\t838\thuman\tSpecies\t9606\n",
            "26515464\t839\t850\tlung cancer\tDisease\tMESH:D008175\n",
            "26515464\t866\t871\tBa/F3\tCellLine\tCVCL_0161;NCBITaxID:10090\n",
            "26515464\t926\t930\tEGFR\tGene\t13649\n",
            "26515464\t1056\t1060\tEGFR\tGene\t13649\n",
            "26515464\t1121\t1129\tafatinib\tChemical\tMESH:D000077716\n",
            "26515464\t1156\t1161\tL858R\tProteinMutation\ttmVar:p|SUB|L|858|R;HGVS:p.L858R;VariantGroup:1;CorrespondingGene:1956;RS#:121434568;CA#:126713\n",
            "26515464\t1169\t1180\tosimertinib\tChemical\t-\n",
            "26515464\t1185\t1196\trociletinib\tChemical\tMESH:C000589977\n",
            "26515464\t1201\t1206\tT790M\tProteinMutation\ttmVar:p|SUB|T|790|M;HGVS:p.T790M;VariantGroup:0;CorrespondingGene:1956;RS#:121434569;CA#:90928\n",
            "26515464\t1352\t1356\tEGFR\tGene\t13649\n",
            "26515464\t1452\t1456\tEGFR\tGene\t13649\n",
            "26515464\t1463\t1474\tosimertinib\tChemical\t-\n",
            "26515464\t1616\t1627\tosimertinib\tChemical\t-\n",
            "26515464\t1632\t1636\tEGFR\tGene\t13649\n",
            "26515464\t1753\t1757\tEGFR\tGene\t13649\n",
            "26515464\t1791\t1795\tEGFR\tGene\t13649\n",
            "\n",
            "26554404|t|The ALK/ROS1 Inhibitor PF-06463922 Overcomes Primary Resistance to Crizotinib in ALK-Driven Neuroblastoma.\n",
            "26554404|a|UNLABELLED: Neuroblastomas harboring activating point mutations in anaplastic lymphoma kinase (ALK) are differentially sensitive to the ALK inhibitor crizotinib, with certain mutations conferring intrinsic crizotinib resistance. To overcome this clinical obstacle, our goal was to identify inhibitors with improved potency that can target intractable ALK variants such as F1174L. We find that PF-06463922 has high potency across ALK variants and inhibits ALK more effectively than crizotinib in vitro. Most importantly, PF-06463922 induces complete tumor regression in both crizotinib-resistant and crizotinib-sensitive xenograft mouse models of neuroblastoma, as well as in patient-derived xenografts harboring the crizotinib-resistant F1174L or F1245C mutations. These studies demonstrate that PF-06463922 has the potential to overcome crizotinib resistance and exerts unprecedented activity as a single targeted agent against F1174L and F1245C ALK-mutated xenograft tumors, while also inducing responses in an R1275Q xenograft model. Taken together, these results provide the rationale to move PF-06463922 into clinical trials for treatment of patients with ALK-mutated neuroblastoma. SIGNIFICANCE: The next-generation ALK/ROS1 inhibitor PF-06463922 exerts unparalleled activity in ALK-driven neuroblastoma models with primary crizotinib resistance. Our biochemical and in vivo data provide the preclinical rationale for fast-tracking the development of this agent in children with relapsed/refractory ALK-mutant neuroblastoma.\n",
            "26554404\t4\t7\tALK\tGene\t238\n",
            "26554404\t8\t12\tROS1\tGene\t6098\n",
            "26554404\t23\t34\tPF-06463922\tChemical\tMESH:C000590786\n",
            "26554404\t67\t77\tCrizotinib\tChemical\tMESH:D000077547\n",
            "26554404\t81\t84\tALK\tGene\t238\n",
            "26554404\t92\t105\tNeuroblastoma\tDisease\tMESH:D009447\n",
            "26554404\t119\t133\tNeuroblastomas\tDisease\tMESH:D009447\n",
            "26554404\t174\t200\tanaplastic lymphoma kinase\tGene\t238\n",
            "26554404\t202\t205\tALK\tGene\t238\n",
            "26554404\t243\t246\tALK\tGene\t238\n",
            "26554404\t257\t267\tcrizotinib\tChemical\tMESH:D000077547\n",
            "26554404\t313\t323\tcrizotinib\tChemical\tMESH:D000077547\n",
            "26554404\t458\t461\tALK\tGene\t238\n",
            "26554404\t479\t485\tF1174L\tProteinMutation\ttmVar:p|SUB|F|1174|L;HGVS:p.F1174L;VariantGroup:0;CorrespondingGene:238;RS#:863225281;CA#:279616\n",
            "26554404\t500\t511\tPF-06463922\tChemical\tMESH:C000590786\n",
            "26554404\t536\t539\tALK\tGene\t238\n",
            "26554404\t562\t565\tALK\tGene\t238\n",
            "26554404\t588\t598\tcrizotinib\tChemical\tMESH:D000077547\n",
            "26554404\t627\t638\tPF-06463922\tChemical\tMESH:C000590786\n",
            "26554404\t656\t661\ttumor\tDisease\tMESH:D009369\n",
            "26554404\t681\t691\tcrizotinib\tChemical\tMESH:D000077547\n",
            "26554404\t706\t716\tcrizotinib\tChemical\tMESH:D000077547\n",
            "26554404\t737\t742\tmouse\tSpecies\t10090\n",
            "26554404\t753\t766\tneuroblastoma\tDisease\tMESH:D009447\n",
            "26554404\t782\t789\tpatient\tSpecies\t9606\n",
            "26554404\t823\t833\tcrizotinib\tChemical\tMESH:D000077547\n",
            "26554404\t844\t850\tF1174L\tProteinMutation\ttmVar:p|SUB|F|1174|L;HGVS:p.F1174L;VariantGroup:0;CorrespondingGene:238;RS#:863225281;CA#:279616\n",
            "26554404\t854\t860\tF1245C\tProteinMutation\ttmVar:p|SUB|F|1245|C;HGVS:p.F1245C;VariantGroup:1;CorrespondingGene:238;RS#:863225283;CA#:279597\n",
            "26554404\t903\t914\tPF-06463922\tChemical\tMESH:C000590786\n",
            "26554404\t945\t955\tcrizotinib\tChemical\tMESH:D000077547\n",
            "26554404\t1036\t1042\tF1174L\tProteinMutation\ttmVar:p|SUB|F|1174|L;HGVS:p.F1174L;VariantGroup:0;CorrespondingGene:238;RS#:863225281;CA#:279616\n",
            "26554404\t1047\t1053\tF1245C\tProteinMutation\ttmVar:p|SUB|F|1245|C;HGVS:p.F1245C;VariantGroup:1;CorrespondingGene:238;RS#:863225283;CA#:279597\n",
            "26554404\t1054\t1057\tALK\tGene\t238\n",
            "26554404\t1076\t1082\ttumors\tDisease\tMESH:D009369\n",
            "26554404\t1120\t1126\tR1275Q\tProteinMutation\ttmVar:p|SUB|R|1275|Q;HGVS:p.R1275Q;VariantGroup:2;CorrespondingGene:238;RS#:113994087;CA#:341482\n",
            "26554404\t1204\t1215\tPF-06463922\tChemical\tMESH:C000590786\n",
            "26554404\t1254\t1262\tpatients\tSpecies\t9606\n",
            "26554404\t1268\t1271\tALK\tGene\t238\n",
            "26554404\t1280\t1293\tneuroblastoma\tDisease\tMESH:D009447\n",
            "26554404\t1329\t1332\tALK\tGene\t238\n",
            "26554404\t1333\t1337\tROS1\tGene\t6098\n",
            "26554404\t1348\t1359\tPF-06463922\tChemical\tMESH:C000590786\n",
            "26554404\t1392\t1395\tALK\tGene\t238\n",
            "26554404\t1403\t1416\tneuroblastoma\tDisease\tMESH:D009447\n",
            "26554404\t1437\t1447\tcrizotinib\tChemical\tMESH:D000077547\n",
            "26554404\t1578\t1586\tchildren\tSpecies\t9606\n",
            "26554404\t1612\t1615\tALK\tGene\t238\n",
            "26554404\t1623\t1636\tneuroblastoma\tDisease\tMESH:D009447\n",
            "\n",
            "26758680|t|Somatically mutated ABL1 is an actionable and essential NSCLC survival gene.\n",
            "26758680|a|The lack of actionable mutations in patients with non-small cell lung cancer (NSCLC) presents a significant hurdle in the design of targeted therapies for this disease. Here, we identify somatically mutated ABL1 as a genetic dependency that is required to maintain NSCLC cell survival. We demonstrate that NSCLC cells with ABL1 mutations are sensitive to ABL inhibitors and we verify that the drug-induced effects on cell viability are specific to pharmacological inhibition of the ABL1 kinase. Furthermore, we confirm that imatinib suppresses lung tumor growth in vivo, specifically in lung cancer cells harboring a gain-of-function (GOF) mutation in ABL1. Consistent with structural modeling, we demonstrate that mutations in ABL1 identified in primary NSCLC tumors and a lung cancer cell line increase downstream pathway activation compared to wild-type ABL1. Finally, we observe that the ABL1 cancer mutants display an increased cytosolic localization, which is associated with the oncogenic properties of the ABL1 kinase. In summary, our results suggest that NSCLC patients with ABL1 mutations could be stratified for treatment with imatinib in combination with other therapies. \n",
            "26758680\t20\t24\tABL1\tGene\t25\n",
            "26758680\t56\t61\tNSCLC\tDisease\tMESH:D002289\n",
            "26758680\t113\t121\tpatients\tSpecies\t9606\n",
            "26758680\t127\t153\tnon-small cell lung cancer\tDisease\tMESH:D002289\n",
            "26758680\t155\t160\tNSCLC\tDisease\tMESH:D002289\n",
            "26758680\t284\t288\tABL1\tGene\t25\n",
            "26758680\t342\t347\tNSCLC\tDisease\tMESH:D002289\n",
            "26758680\t383\t388\tNSCLC\tDisease\tMESH:D002289\n",
            "26758680\t400\t404\tABL1\tGene\t25\n",
            "26758680\t432\t435\tABL\tGene\t25\n",
            "26758680\t559\t563\tABL1\tGene\t25\n",
            "26758680\t601\t609\timatinib\tChemical\tMESH:D000068877\n",
            "26758680\t621\t631\tlung tumor\tDisease\tMESH:D008175\n",
            "26758680\t664\t675\tlung cancer\tDisease\tMESH:D008175\n",
            "26758680\t729\t733\tABL1\tGene\t25\n",
            "26758680\t805\t809\tABL1\tGene\t25\n",
            "26758680\t832\t837\tNSCLC\tDisease\tMESH:D002289\n",
            "26758680\t851\t862\tlung cancer\tDisease\tMESH:D008175\n",
            "26758680\t934\t938\tABL1\tGene\t25\n",
            "26758680\t969\t973\tABL1\tGene\t25\n",
            "26758680\t974\t980\tcancer\tDisease\tMESH:D009369\n",
            "26758680\t1091\t1095\tABL1\tGene\t25\n",
            "26758680\t1141\t1146\tNSCLC\tDisease\tMESH:D002289\n",
            "26758680\t1147\t1155\tpatients\tSpecies\t9606\n",
            "26758680\t1161\t1165\tABL1\tGene\t25\n",
            "26758680\t1215\t1223\timatinib\tChemical\tMESH:D000068877\n",
            "\n",
            "26771872|t|Predictive relevance of PD-L1 expression combined with CD8+ TIL density in stage III non-small cell lung cancer patients receiving concurrent chemoradiotherapy.\n",
            "26771872|a|BACKGROUND: Expression of programmed cell death-ligand 1 (PD-L1) is known to be a mechanism whereby cancer can escape immune surveillance, but little is known about factors predictive of efficacy in patients with locally advanced non-small cell lung cancer (NSCLC). We investigated the predictive relevance of PD-L1 expression and CD8+ tumour-infiltrating lymphocytes (TILs) density in patients with locally advanced NSCLC receiving concurrent chemoradiotherapy (CCRT). METHODS: We retrospectively reviewed 74 consecutive patients with stage III NSCLC who had received CCRT. PD-L1 expression and CD8+ TIL density were evaluated by immunohistochemical analysis. RESULTS: Univariate and multivariate analyses demonstrated that CD8+ TIL density was an independent and significant predictive factor for progression-free survival (PFS) and OS, whereas PD-L1 expression was not correlated with PFS and OS. Sub-analysis revealed that the PD-L1+/CD8 low group had the shortest PFS (8.6 months, p = 0.02) and OS (13.9 months, p = 0.11), and that the PD-L1-/CD8 high group had the longest prognosis (median PFS and OS were not reached) by Kaplan-Meier curves of the four sub-groups. CONCLUSIONS: Among stage III NSCLC patients who received CCRT, there was a trend for poor survival in those who expressed PD-L1. Our analysis indicated that a combination of lack of PD-L1 expression and CD8+ TIL density was significantly associated with favourable survival in these patients. It is proposed that PD-L1 expression in combination with CD8+ TIL density could be a useful predictive biomarker in patients with stage III NSCLC.\n",
            "26771872\t24\t29\tPD-L1\tGene\t29126\n",
            "26771872\t55\t58\tCD8\tGene\t925\n",
            "26771872\t105\t111\tcancer\tDisease\tMESH:D009369\n",
            "26771872\t112\t120\tpatients\tSpecies\t9606\n",
            "26771872\t187\t217\tprogrammed cell death-ligand 1\tGene\t29126\n",
            "26771872\t219\t224\tPD-L1\tGene\t29126\n",
            "26771872\t261\t267\tcancer\tDisease\tMESH:D009369\n",
            "26771872\t360\t368\tpatients\tSpecies\t9606\n",
            "26771872\t391\t417\tnon-small cell lung cancer\tDisease\tMESH:D002289\n",
            "26771872\t419\t424\tNSCLC\tDisease\tMESH:D002289\n",
            "26771872\t471\t476\tPD-L1\tGene\t29126\n",
            "26771872\t492\t495\tCD8\tGene\t925\n",
            "26771872\t547\t555\tpatients\tSpecies\t9606\n",
            "26771872\t578\t583\tNSCLC\tDisease\tMESH:D002289\n",
            "26771872\t683\t691\tpatients\tSpecies\t9606\n",
            "26771872\t707\t712\tNSCLC\tDisease\tMESH:D002289\n",
            "26771872\t736\t741\tPD-L1\tGene\t29126\n",
            "26771872\t757\t760\tCD8\tGene\t925\n",
            "26771872\t886\t889\tCD8\tGene\t925\n",
            "26771872\t1008\t1013\tPD-L1\tGene\t29126\n",
            "26771872\t1092\t1097\tPD-L1\tGene\t29126\n",
            "26771872\t1099\t1102\tCD8\tGene\t925\n",
            "26771872\t1202\t1207\tPD-L1\tGene\t29126\n",
            "26771872\t1209\t1212\tCD8\tGene\t925\n",
            "26771872\t1363\t1368\tNSCLC\tDisease\tMESH:D002289\n",
            "26771872\t1369\t1377\tpatients\tSpecies\t9606\n",
            "26771872\t1456\t1461\tPD-L1\tGene\t29126\n",
            "26771872\t1516\t1521\tPD-L1\tGene\t29126\n",
            "26771872\t1537\t1540\tCD8\tGene\t925\n",
            "26771872\t1617\t1625\tpatients\tSpecies\t9606\n",
            "26771872\t1647\t1652\tPD-L1\tGene\t29126\n",
            "26771872\t1684\t1687\tCD8\tGene\t925\n",
            "26771872\t1743\t1751\tpatients\tSpecies\t9606\n",
            "26771872\t1767\t1772\tNSCLC\tDisease\tMESH:D002289\n",
            "\n",
            "26899019|t|Prognostic and Predictive Effect of TP53 Mutations in Patients with Non-Small Cell Lung Cancer from Adjuvant Cisplatin-Based Therapy Randomized Trials: A LACE-Bio Pooled Analysis.\n",
            "26899019|a|INTRODUCTION: Tumor protein p53 gene (TP53) mutations are common in stage I through III non-small cell lung cancer, but clinical trials have shown inconsistent results regarding their relationship to the effects of adjuvant therapy. The objective is to clarify their putative prognostic and predictive effects. METHODS: A pooled analysis of TP53 mutations (exons 5-8) was conducted in four randomized trials (the International Adjuvant Lung Cancer Trial, J BRonchus 10, Cancer and Leukemia Group B-9633, and Adjuvant Navelbine International Trialist Association trial) of platinum-based adjuvant chemotherapy (ACT) versus observation (OBS). Hazard ratios (HRs) and 95% confidence intervals (CIs) of mutant versus wild-type (WT) TP53 for overall survival (OS) and disease-free survival (DFS) were estimated using a multivariable Cox model stratified on trial and adjusted on sex, age, and clinicopathological variables. Predictive value was evaluated with an interaction between treatment and TP53. RESULTS: A total of 1209 patients (median follow-up 5.5 years) were included. There were 573 deaths (47%) and 653 DFS events (54%). Mutations (434 [36%]) had no prognostic effect (OBS HROS = 0.99, 95% CI: 0.77-1.28, p = 0.95; HRDFS = 0.99, 95% CI: 0.78-1.25, p = 0.92) but were marginally predictive of benefit from ACT for OS (test for interaction: OS, p = 0.06; DFS, p = 0.11). Patients with WT TP53 had a tendency toward better outcomes with ACT than did those in the OBS group (HROS = 0.77, 95% CI: 0.62-0.95, p = 0.02; HRDFS = 0.75, 95% CI: 0.62-0.92, p = 0.005). In the ACT arm, a deleterious effect of mutant versus WT TP53 was observed (HROS = 1.40, 95% CI: 1.10-1.78, p = 0.006; HRDFS = 1.31, 95% CI: 1.04-1.64, p = 0.02). CONCLUSIONS: TP53 mutation had no prognostic effect but was marginally predictive for survival from ACT. In patients who received ACT, TP53 mutation tended to be associated with shorter survival than wild-type TP53.\n",
            "26899019\t36\t40\tTP53\tGene\t7157\n",
            "26899019\t54\t62\tPatients\tSpecies\t9606\n",
            "26899019\t68\t94\tNon-Small Cell Lung Cancer\tDisease\tMESH:D002289\n",
            "26899019\t109\t118\tCisplatin\tChemical\tMESH:D002945\n",
            "26899019\t218\t222\tTP53\tGene\t7157\n",
            "26899019\t283\t294\tlung cancer\tDisease\tMESH:D008175\n",
            "26899019\t521\t525\tTP53\tGene\t7157\n",
            "26899019\t616\t627\tLung Cancer\tDisease\tMESH:D008175\n",
            "26899019\t650\t656\tCancer\tDisease\tMESH:D009369\n",
            "26899019\t661\t669\tLeukemia\tDisease\tMESH:D007938\n",
            "26899019\t697\t706\tNavelbine\tChemical\tMESH:D000077235\n",
            "26899019\t752\t760\tplatinum\tChemical\tMESH:D010984\n",
            "26899019\t908\t912\tTP53\tGene\t7157\n",
            "26899019\t1172\t1176\tTP53\tGene\t7157\n",
            "26899019\t1203\t1211\tpatients\tSpecies\t9606\n",
            "26899019\t1271\t1277\tdeaths\tDisease\tMESH:D003643\n",
            "26899019\t1558\t1566\tPatients\tSpecies\t9606\n",
            "26899019\t1575\t1579\tTP53\tGene\t7157\n",
            "26899019\t1804\t1808\tTP53\tGene\t7157\n",
            "26899019\t1923\t1927\tTP53\tGene\t7157\n",
            "26899019\t2018\t2026\tpatients\tSpecies\t9606\n",
            "26899019\t2045\t2049\tTP53\tGene\t7157\n",
            "26899019\t2120\t2124\tTP53\tGene\t7157\n",
            "\n",
            "26918046|t|Association of Nuclear PIM1 Expression with Lymph Node Metastasis and Poor Prognosis in Patients with Lung Adenocarcinoma and Squamous Cell Carcinoma.\n",
            "26918046|a|Increasing evidence indicates that aberrant expression of PIM1, p-STAT3 and c-MYC is involved in the pathogenesis of various solid tumors, but its prognostic value is still unclear in non-small cell lung cancer (NSCLC). Here, we sought to evaluate the expression and prognostic role of these markers in patients with lung adenocarcinoma (AD) and squamous cell carcinoma (SCC). Real time RT-PCR and Western blotting was used to analyze the mRNA and protein expression of PIM1 in NSCLC cell lines, respectively. The expression of PIM1, p-STAT3, and c-MYC was immunohistochemically tested in archival tumor samples from 194 lung AD and SCC patients. High nuclear PIM1 expression was detected in 43.3% of ADs and SCCs, and was significantly correlated with lymph node (LN) metastasis (P = 0.028) and histology (P = 0.003). High nuclear PIM1 expression (P = 0.034), locally advanced stage (P < 0.001), AD (P = 0.007) and poor pathologic differentiation (P = 0.002) were correlated with worse disease-free survival (DFS). High nuclear PIM1 expression (P = 0.009), advanced clinical stage (P < 0.001) and poor pathologic differentiation (P = 0.004) were independent unfavorable prognostic factors for overall survival (OS). High p-STAT3 expression was not associated with OS but significantly correlated with LN metastasis, while c-MYC was not significantly correlated with any clinicopathological parameter or survival. Therefore, in AD and SCC patients, nuclear PIM1 expression level is an independent factor for DFS and OS and it might serve as a predictive biomarker for outcome. \n",
            "26918046\t23\t27\tPIM1\tGene\t5292\n",
            "26918046\t88\t96\tPatients\tSpecies\t9606\n",
            "26918046\t102\t149\tLung Adenocarcinoma and Squamous Cell Carcinoma\tDisease\tMESH:D002294\n",
            "26918046\t209\t213\tPIM1\tGene\t5292\n",
            "26918046\t217\t222\tSTAT3\tGene\t6774\n",
            "26918046\t227\t232\tc-MYC\tGene\t4609\n",
            "26918046\t282\t288\ttumors\tDisease\tMESH:D009369\n",
            "26918046\t350\t361\tlung cancer\tDisease\tMESH:D008175\n",
            "26918046\t454\t462\tpatients\tSpecies\t9606\n",
            "26918046\t468\t487\tlung adenocarcinoma\tDisease\tMESH:D000077192\n",
            "26918046\t497\t520\tsquamous cell carcinoma\tDisease\tMESH:D002294\n",
            "26918046\t522\t525\tSCC\tDisease\tMESH:D002294\n",
            "26918046\t621\t625\tPIM1\tGene\t5292\n",
            "26918046\t679\t683\tPIM1\tGene\t5292\n",
            "26918046\t687\t692\tSTAT3\tGene\t6774\n",
            "26918046\t698\t703\tc-MYC\tGene\t4609\n",
            "26918046\t740\t754\tarchival tumor\tDisease\tMESH:D009369\n",
            "26918046\t784\t787\tSCC\tDisease\tMESH:D002294\n",
            "26918046\t788\t796\tpatients\tSpecies\t9606\n",
            "26918046\t811\t815\tPIM1\tGene\t5292\n",
            "26918046\t983\t987\tPIM1\tGene\t5292\n",
            "26918046\t1180\t1184\tPIM1\tGene\t5292\n",
            "26918046\t1375\t1380\tSTAT3\tGene\t6774\n",
            "26918046\t1474\t1479\tc-MYC\tGene\t4609\n",
            "26918046\t1586\t1589\tSCC\tDisease\tMESH:D002294\n",
            "26918046\t1590\t1598\tpatients\tSpecies\t9606\n",
            "26918046\t1608\t1612\tPIM1\tGene\t5292\n",
            "\n",
            "26938871|t|Rapid and dramatic response to alectinib in an anaplastic lymphoma kinase rearranged non-small-cell lung cancer patient who is critically ill.\n",
            "26938871|a|Anaplastic lymphoma kinase (ALK) tyrosine kinase inhibitors (TKIs) have shown promising clinical activity in the treatment of non-small-cell lung cancer (NSCLC) that harbors ALK rearrangement. The next-generation ALK-TKI, alectinib, has been reported to have potent efficacy in ALK-positive NSCLC patients including on mutations that confer resistance to crizotinib, which was the first ALK-TKI approved for ALK-positive NSCLC. The efficacy and safety of ALK-TKIs, including crizotinib and alectinib, as the first-line treatment in critically ill patients is unclear. We report one ALK-positive NSCLC patient with poor performance status (PS) and disseminated intravascular coagulation because of respiratory failure and multiple metastases, and experienced the rapid and dramatic response to alectinib without adverse events that can lead to discontinuation and dose reduction of the drug. After a couple of months of treatment with alectinib, radiological review indicated a complete response. The present case is the first reported case of rapid and marked response to alectinib in ALK-positive NSCLC patients who had poor PS and severe organ dysfunction, such as disseminated intravascular coagulation. Further investigation of the safety and efficacy of ALK-TKI for ALK-positive NSCLC patients who are critically ill is warranted.\n",
            "26938871\t31\t40\talectinib\tChemical\tMESH:C582670\n",
            "26938871\t47\t73\tanaplastic lymphoma kinase\tGene\t238\n",
            "26938871\t105\t111\tcancer\tDisease\tMESH:D009369\n",
            "26938871\t112\t119\tpatient\tSpecies\t9606\n",
            "26938871\t127\t141\tcritically ill\tDisease\tMESH:D016638\n",
            "26938871\t143\t169\tAnaplastic lymphoma kinase\tGene\t238\n",
            "26938871\t171\t174\tALK\tGene\t238\n",
            "26938871\t284\t295\tlung cancer\tDisease\tMESH:D008175\n",
            "26938871\t317\t320\tALK\tGene\t238\n",
            "26938871\t356\t359\tALK\tGene\t238\n",
            "26938871\t365\t374\talectinib\tChemical\tMESH:C582670\n",
            "26938871\t421\t424\tALK\tGene\t238\n",
            "26938871\t440\t448\tpatients\tSpecies\t9606\n",
            "26938871\t498\t508\tcrizotinib\tChemical\tMESH:D000077547\n",
            "26938871\t530\t533\tALK\tGene\t238\n",
            "26938871\t551\t554\tALK\tGene\t238\n",
            "26938871\t598\t601\tALK\tGene\t238\n",
            "26938871\t618\t628\tcrizotinib\tChemical\tMESH:D000077547\n",
            "26938871\t633\t642\talectinib\tChemical\tMESH:C582670\n",
            "26938871\t675\t689\tcritically ill\tDisease\tMESH:D016638\n",
            "26938871\t690\t698\tpatients\tSpecies\t9606\n",
            "26938871\t725\t728\tALK\tGene\t238\n",
            "26938871\t744\t751\tpatient\tSpecies\t9606\n",
            "26938871\t790\t828\tdisseminated intravascular coagulation\tDisease\tMESH:D004211\n",
            "26938871\t840\t859\trespiratory failure\tDisease\tMESH:D012131\n",
            "26938871\t873\t883\tmetastases\tDisease\tMESH:D009362\n",
            "26938871\t936\t945\talectinib\tChemical\tMESH:C582670\n",
            "26938871\t1077\t1086\talectinib\tChemical\tMESH:C582670\n",
            "26938871\t1215\t1224\talectinib\tChemical\tMESH:C582670\n",
            "26938871\t1228\t1231\tALK\tGene\t238\n",
            "26938871\t1247\t1255\tpatients\tSpecies\t9606\n",
            "26938871\t1283\t1300\torgan dysfunction\tDisease\tMESH:D009102\n",
            "26938871\t1310\t1348\tdisseminated intravascular coagulation\tDisease\tMESH:D004211\n",
            "26938871\t1402\t1405\tALK\tGene\t238\n",
            "26938871\t1414\t1417\tALK\tGene\t238\n",
            "26938871\t1433\t1441\tpatients\tSpecies\t9606\n",
            "26938871\t1450\t1464\tcritically ill\tDisease\tMESH:D016638\n",
            "\n",
            "27022118|t|Intracranial Efficacy of Crizotinib Versus Chemotherapy in Patients With Advanced ALK-Positive Non-Small-Cell Lung Cancer: Results From PROFILE 1014.\n",
            "27022118|a|PURPOSE: Intracranial efficacy of first-line crizotinib versus chemotherapy was compared prospectively in the phase III PROFILE 1014 study in ALK-positive non-small-cell lung cancer. PATIENTS AND METHODS: Patients were randomly assigned to receive crizotinib (250 mg twice daily; n = 172) or chemotherapy (pemetrexed 500 mg/m(2) plus cisplatin 75 mg/m(2) or carboplatin at area under the curve 5 to 6, every 3 weeks for <= six cycles; n = 171). Patients with stable treated brain metastases (tBM) were eligible. Intracranial efficacy was assessed at baseline and every 6 or 12 weeks in patients with or without known brain metastases (BM), respectively; intracranial time to tumor progression (IC-TTP; per protocol) and intracranial disease control rate (IC-DCR; post hoc) were measured. The intent-to-treat population was also assessed. RESULTS: Of 343 patients in the intent-to-treat population, 23% had tBM at baseline. A nonsignificant IC-TTP improvement was observed with crizotinib in the intent-to-treat population (hazard ratio [HR], 0.60; P = .069), patients with tBM (HR, 0.45; P = .063), and patients without BM (HR, 0.69; P = .323). Among patients with tBM, IC-DCR was significantly higher with crizotinib versus chemotherapy at 12 weeks (85% v 45%, respectively; P < .001) and 24 weeks (56% v 25%, respectively; P = .006). Progression-free survival was significantly longer with crizotinib versus chemotherapy in both subgroups (tBM present: HR, 0.40; P < .001; median, 9.0 v 4.0 months, respectively; BM absent: HR, 0.51; P < .001; median, 11.1 v 7.2 months, respectively) and in the intent-to-treat population (HR, 0.45; P < .001; median, 10.9 v 7.0 months, respectively). CONCLUSION: Compared with chemotherapy, crizotinib demonstrated a significantly higher IC-DCR in patients with tBM. Improvements in IC-TTP were not statistically significant in patients with or without tBM, although sensitivity to detect treatment differences in or between the two subgroups was low.\n",
            "27022118\t25\t35\tCrizotinib\tChemical\tMESH:D000077547\n",
            "27022118\t59\t67\tPatients\tSpecies\t9606\n",
            "27022118\t82\t85\tALK\tGene\t238\n",
            "27022118\t195\t205\tcrizotinib\tChemical\tMESH:D000077547\n",
            "27022118\t292\t295\tALK\tGene\t238\n",
            "27022118\t320\t331\tlung cancer\tDisease\tMESH:D008175\n",
            "27022118\t333\t341\tPATIENTS\tSpecies\t9606\n",
            "27022118\t355\t363\tPatients\tSpecies\t9606\n",
            "27022118\t398\t408\tcrizotinib\tChemical\tMESH:D000077547\n",
            "27022118\t456\t466\tpemetrexed\tChemical\tMESH:D000068437\n",
            "27022118\t484\t493\tcisplatin\tChemical\tMESH:D002945\n",
            "27022118\t508\t519\tcarboplatin\tChemical\tMESH:D016190\n",
            "27022118\t595\t603\tPatients\tSpecies\t9606\n",
            "27022118\t630\t640\tmetastases\tDisease\tMESH:D009362\n",
            "27022118\t736\t744\tpatients\tSpecies\t9606\n",
            "27022118\t773\t783\tmetastases\tDisease\tMESH:D009362\n",
            "27022118\t825\t830\ttumor\tDisease\tMESH:D009369\n",
            "27022118\t870\t890\tintracranial disease\tDisease\tMESH:D020765\n",
            "27022118\t1004\t1012\tpatients\tSpecies\t9606\n",
            "27022118\t1127\t1137\tcrizotinib\tChemical\tMESH:D000077547\n",
            "27022118\t1209\t1217\tpatients\tSpecies\t9606\n",
            "27022118\t1253\t1261\tpatients\tSpecies\t9606\n",
            "27022118\t1301\t1309\tpatients\tSpecies\t9606\n",
            "27022118\t1357\t1367\tcrizotinib\tChemical\tMESH:D000077547\n",
            "27022118\t1542\t1552\tcrizotinib\tChemical\tMESH:D000077547\n",
            "27022118\t1878\t1888\tcrizotinib\tChemical\tMESH:D000077547\n",
            "27022118\t1935\t1943\tpatients\tSpecies\t9606\n",
            "27022118\t2015\t2023\tpatients\tSpecies\t9606\n",
            "27022118\t2040\t2043\ttBM\tChemical\t-\n",
            "\n",
            "27315356|t|Efficacy and safety of dovitinib in pretreated patients with advanced squamous non-small cell lung cancer with FGFR1 amplification: A single-arm, phase 2 study.\n",
            "27315356|a|BACKGROUND: Fibroblast growth factor receptor 1 (FGFR1) amplification is a potential driving oncogene in squamous cell cancer (SCC) of the lung. The current phase 2 study evaluated the efficacy and tolerability of dovitinib, an FGFR inhibitor, in patients with advanced SCC of the lung. METHODS: Patients with pretreated advanced SCC of the lung whose tumors demonstrated FGFR1 amplification of > 5 copies by fluorescence in situ hybridization were enrolled. Dovitinib at a dose of 500 mg was administered orally, once daily, on days 1 to 5 of every week, followed by 2 days off. The primary endpoint was overall response. Secondary endpoints were progression-free survival, overall survival, and toxicity. RESULTS: All 26 patients were men with a median age of 68 years (range, 52-80 years). The majority of patients were ever-smokers. The median duration of dovitinib administration (28 days per cycle) was 2.5 months (range, 0.7-8.6 months). The overall response rate was 11.5% (95% confidence interval [95% CI], 0.8%-23.8%) and the disease control rate was 50% (95% CI, 30.8%-69.2%), with 3 patients achieving partial responses. Response durations for the patients with partial responses were >=4.5 months, >= 5.1 months, and 6.1 months, respectively. After a median follow-up of 15.7 months (range, 1.2-25.6 months), the median overall survival was 5.0 months (95% CI, 3.6-6.4 months) and the median progression-free survival was 2.9 months (95% CI, 1.5-4.3 months). The most common grade 3 or higher adverse events were fatigue (19.2%), anorexia (11.5%), and hyponatremia (11.5%) (event severity was graded based on National Cancer Institute Common Terminology Criteria for Adverse Events [version 4.0]). CONCLUSIONS: Treatment with dovitinib demonstrated modest efficacy in patients with advanced SCC with FGFR1 amplification. Further studies to evaluate other biomarkers correlated with the efficacy of dovitinib in patients with SCC are warranted. Cancer 2016;122:3024-3031.   2016 American Cancer Society.\n",
            "27315356\t23\t32\tdovitinib\tChemical\tMESH:C500007\n",
            "27315356\t47\t55\tpatients\tSpecies\t9606\n",
            "27315356\t70\t105\tsquamous non-small cell lung cancer\tDisease\tMESH:D002289\n",
            "27315356\t111\t116\tFGFR1\tGene\t2260\n",
            "27315356\t173\t208\tFibroblast growth factor receptor 1\tGene\t2260\n",
            "27315356\t210\t215\tFGFR1\tGene\t2260\n",
            "27315356\t266\t286\tsquamous cell cancer\tDisease\tMESH:D002294\n",
            "27315356\t288\t291\tSCC\tDisease\tMESH:D002294\n",
            "27315356\t375\t384\tdovitinib\tChemical\tMESH:C500007\n",
            "27315356\t389\t393\tFGFR\tGene\t2260\n",
            "27315356\t408\t416\tpatients\tSpecies\t9606\n",
            "27315356\t431\t434\tSCC\tDisease\tMESH:D002294\n",
            "27315356\t457\t465\tPatients\tSpecies\t9606\n",
            "27315356\t491\t494\tSCC\tDisease\tMESH:D002294\n",
            "27315356\t513\t519\ttumors\tDisease\tMESH:D009369\n",
            "27315356\t533\t538\tFGFR1\tGene\t2260\n",
            "27315356\t620\t629\tDovitinib\tChemical\tMESH:C500007\n",
            "27315356\t858\t866\ttoxicity\tDisease\tMESH:D064420\n",
            "27315356\t884\t892\tpatients\tSpecies\t9606\n",
            "27315356\t898\t901\tmen\tSpecies\t9606\n",
            "27315356\t970\t978\tpatients\tSpecies\t9606\n",
            "27315356\t1021\t1030\tdovitinib\tChemical\tMESH:C500007\n",
            "27315356\t1256\t1264\tpatients\tSpecies\t9606\n",
            "27315356\t1321\t1329\tpatients\tSpecies\t9606\n",
            "27315356\t1687\t1694\tfatigue\tDisease\tMESH:D005221\n",
            "27315356\t1704\t1712\tanorexia\tDisease\tMESH:D000855\n",
            "27315356\t1726\t1738\thyponatremia\tDisease\tMESH:D007010\n",
            "27315356\t1792\t1798\tCancer\tDisease\tMESH:D009369\n",
            "27315356\t1900\t1909\tdovitinib\tChemical\tMESH:C500007\n",
            "27315356\t1942\t1950\tpatients\tSpecies\t9606\n",
            "27315356\t1965\t1968\tSCC\tDisease\tMESH:D002294\n",
            "27315356\t1974\t1979\tFGFR1\tGene\t2260\n",
            "27315356\t2072\t2081\tdovitinib\tChemical\tMESH:C500007\n",
            "27315356\t2085\t2093\tpatients\tSpecies\t9606\n",
            "27315356\t2099\t2102\tSCC\tDisease\tMESH:D002294\n",
            "27315356\t2118\t2124\tCancer\tDisease\tMESH:D009369\n",
            "27315356\t2161\t2167\tCancer\tDisease\tMESH:D009369\n",
            "\n",
            "27544060|t|Clinical Activity of Alectinib in Advanced RET-Rearranged Non-Small Cell Lung Cancer.\n",
            "27544060|a|INTRODUCTION: Chromosomal rearrangements involving rearranged during transfection gene (RET) occur in 1% to 2% of NSCLCs and may confer sensitivity to rearranged during transfection (RET) inhibitors. Alectinib is an anaplastic lymphoma kinase tyrosine kinase inhibitor (TKI) that also has anti-RET activity in vitro. The clinical activity of alectinib in patients with RET-rearranged NSCLC has not yet been reported. METHODS: We have described four patients with advanced RET-rearranged NSCLC who were treated with alectinib (600 mg twice daily [n = 3] or 900 mg twice daily [n = 1]) as part of single-patient compassionate use protocols or off-label use of the commercially available drug. RESULTS: Four patients with metastatic RET-rearranged NSCLC were identified. Three of the four had received prior RET TKIs, including cabozantinib and experimental RET inhibitors. In total, we observed two (50%) objective radiographic responses after treatment with alectinib (one confirmed and one unconfirmed), with durations of therapy of 6 months and more than 5 months (treatment ongoing), respectively. Notably, one of these two patients had his dose of alectinib escalated to 900 mg twice daily and had clinical improvement in central nervous system metastases. In addition, one patient (25%) experienced a best response of stable disease lasting approximately 6 weeks (the drug discontinued for toxicity). A fourth patient who was RET TKI-naive had primary progression while receiving alectinib. CONCLUSIONS: Alectinib demonstrated preliminary antitumor activity in patients with advanced RET-rearranged NSCLC, most of whom had received prior RET inhibitors. Larger prospective studies with longer follow-up are needed to assess the efficacy of alectinib in RET-rearranged NSCLC and other RET-driven malignancies. In parallel, development of more selective, potent RET TKIs is warranted.\n",
            "27544060\t21\t30\tAlectinib\tChemical\tMESH:C582670\n",
            "27544060\t43\t46\tRET\tGene\t5979\n",
            "27544060\t58\t84\tNon-Small Cell Lung Cancer\tDisease\tMESH:D002289\n",
            "27544060\t174\t177\tRET\tGene\t5979\n",
            "27544060\t200\t206\tNSCLCs\tDisease\tMESH:D002289\n",
            "27544060\t269\t272\tRET\tGene\t5979\n",
            "27544060\t286\t295\tAlectinib\tChemical\tMESH:C582670\n",
            "27544060\t302\t321\tanaplastic lymphoma\tDisease\tMESH:D017728\n",
            "27544060\t380\t383\tRET\tGene\t5979\n",
            "27544060\t428\t437\talectinib\tChemical\tMESH:C582670\n",
            "27544060\t441\t449\tpatients\tSpecies\t9606\n",
            "27544060\t455\t458\tRET\tGene\t5979\n",
            "27544060\t470\t475\tNSCLC\tDisease\tMESH:D002289\n",
            "27544060\t535\t543\tpatients\tSpecies\t9606\n",
            "27544060\t558\t561\tRET\tGene\t5979\n",
            "27544060\t573\t578\tNSCLC\tDisease\tMESH:D002289\n",
            "27544060\t601\t610\talectinib\tChemical\tMESH:C582670\n",
            "27544060\t688\t695\tpatient\tSpecies\t9606\n",
            "27544060\t791\t799\tpatients\tSpecies\t9606\n",
            "27544060\t816\t819\tRET\tGene\t5979\n",
            "27544060\t831\t836\tNSCLC\tDisease\tMESH:D002289\n",
            "27544060\t891\t894\tRET\tGene\t5979\n",
            "27544060\t911\t923\tcabozantinib\tChemical\tMESH:C558660\n",
            "27544060\t941\t944\tRET\tGene\t5979\n",
            "27544060\t1043\t1052\talectinib\tChemical\tMESH:C582670\n",
            "27544060\t1212\t1220\tpatients\tSpecies\t9606\n",
            "27544060\t1237\t1246\talectinib\tChemical\tMESH:C582670\n",
            "27544060\t1334\t1344\tmetastases\tDisease\tMESH:D009362\n",
            "27544060\t1363\t1370\tpatient\tSpecies\t9606\n",
            "27544060\t1480\t1488\ttoxicity\tDisease\tMESH:D064420\n",
            "27544060\t1500\t1507\tpatient\tSpecies\t9606\n",
            "27544060\t1516\t1519\tRET\tGene\t5979\n",
            "27544060\t1570\t1579\talectinib\tChemical\tMESH:C582670\n",
            "27544060\t1594\t1603\tAlectinib\tChemical\tMESH:C582670\n",
            "27544060\t1651\t1659\tpatients\tSpecies\t9606\n",
            "27544060\t1674\t1677\tRET\tGene\t5979\n",
            "27544060\t1689\t1694\tNSCLC\tDisease\tMESH:D002289\n",
            "27544060\t1728\t1731\tRET\tGene\t5979\n",
            "27544060\t1830\t1839\talectinib\tChemical\tMESH:C582670\n",
            "27544060\t1843\t1846\tRET\tGene\t5979\n",
            "27544060\t1858\t1863\tNSCLC\tDisease\tMESH:D002289\n",
            "27544060\t1874\t1877\tRET\tGene\t5979\n",
            "27544060\t1885\t1897\tmalignancies\tDisease\tMESH:D009369\n",
            "27544060\t1950\t1953\tRET\tGene\t5979\n",
            "\n",
            "27694386|t|Acquired METD1228V Mutation and Resistance to MET Inhibition in Lung Cancer.\n",
            "27694386|a|Amplified and/or mutated MET can act as both a primary oncogenic driver and as a promoter of tyrosine kinase inhibitor (TKI) resistance in non-small cell lung cancer (NSCLC). However, the landscape of MET-specific targeting agents remains underdeveloped, and understanding of mechanisms of resistance to MET TKIs is limited. Here, we present a case of a patient with lung adenocarcinoma harboring both a mutation in EGFR and an amplification of MET, who after progression on erlotinib responded dramatically to combined MET and EGFR inhibition with savolitinib and osimertinib. When resistance developed to this combination, a new MET kinase domain mutation, D1228V, was detected. Our in vitro findings demonstrate that METD1228V induces resistance to type I MET TKIs through impaired drug binding, while sensitivity to type II MET TKIs is maintained. Based on these findings, the patient was treated with erlotinib combined with cabozantinib, a type II MET inhibitor, and exhibited a response. SIGNIFICANCE: With several structurally distinct MET inhibitors undergoing development for treatment of NSCLC, it is critical to identify mechanism-based therapies for drug resistance. We demonstrate that an acquired METD1228V mutation mediates resistance to type I, but not type II, MET inhibitors, having therapeutic implications for the clinical use of sequential MET inhibitors. Cancer Discov; 6(12); 1334-41.  2016 AACR.See related commentary by Trusolino, p. 1306This article is highlighted in the In This Issue feature, p. 1293.\n",
            "27694386\t9\t12\tMET\tGene\t79811\n",
            "27694386\t46\t49\tMET\tGene\t79811\n",
            "27694386\t64\t75\tLung Cancer\tDisease\tMESH:D008175\n",
            "27694386\t102\t105\tMET\tGene\t79811\n",
            "27694386\t231\t242\tlung cancer\tDisease\tMESH:D008175\n",
            "27694386\t278\t281\tMET\tGene\t79811\n",
            "27694386\t381\t384\tMET\tGene\t79811\n",
            "27694386\t431\t438\tpatient\tSpecies\t9606\n",
            "27694386\t444\t463\tlung adenocarcinoma\tDisease\tMESH:D000077192\n",
            "27694386\t493\t497\tEGFR\tGene\t1956\n",
            "27694386\t522\t525\tMET\tGene\t79811\n",
            "27694386\t552\t561\terlotinib\tChemical\tMESH:D000069347\n",
            "27694386\t597\t600\tMET\tGene\t79811\n",
            "27694386\t605\t609\tEGFR\tGene\t1956\n",
            "27694386\t626\t637\tsavolitinib\tChemical\tMESH:C000593259\n",
            "27694386\t642\t653\tosimertinib\tChemical\t-\n",
            "27694386\t708\t711\tMET\tGene\t79811\n",
            "27694386\t797\t806\tMETD1228V\tChemical\t-\n",
            "27694386\t836\t839\tMET\tGene\t79811\n",
            "27694386\t905\t908\tMET\tGene\t79811\n",
            "27694386\t958\t965\tpatient\tSpecies\t9606\n",
            "27694386\t983\t992\terlotinib\tChemical\tMESH:D000069347\n",
            "27694386\t1007\t1019\tcabozantinib\tChemical\tMESH:C558660\n",
            "27694386\t1031\t1034\tMET\tGene\t79811\n",
            "27694386\t1121\t1124\tMET\tGene\t79811\n",
            "27694386\t1289\t1298\tMETD1228V\tChemical\t-\n",
            "27694386\t1356\t1359\tMET\tGene\t79811\n",
            "27694386\t1439\t1442\tMET\tGene\t79811\n",
            "27694386\t1455\t1461\tCancer\tDisease\tMESH:D009369\n",
            "\n",
            "27876675|t|A Phase 1/1b Study Evaluating Trametinib Plus Docetaxel or Pemetrexed in Patients With Advanced Non-Small Cell Lung Cancer.\n",
            "27876675|a|OBJECTIVES: This two-part study evaluated trametinib, a MEK1/2 inhibitor, in combination with anticancer agents. Inhibition of MEK, a downstream effector of KRAS, demonstrated preclinical synergy with chemotherapy in KRAS-mutant NSCLC cell lines. Part 1 of this study identified recommended phase 2 doses of trametinib combinations. Part 2, reported herein, evaluated the safety, tolerability, pharmacokinetics, and efficacy of trametinib combinations in patients with NSCLC with and without KRAS mutations. METHODS: Phase 1b evaluated trametinib plus docetaxel with growth factor support (trametinib, 2.0 mg once daily, and docetaxel, 75 mg/m2 every 3 weeks) or pemetrexed (trametinib, 1.5 mg once daily, and pemetrexed, 500 mg/m2 every 3 weeks). Eligibility criteria for the expansion cohorts included metastatic NSCLC with measurable disease, known KRAS mutation status, Eastern Cooperative Oncology Group performance status of 1 or lower, and no more than two prior regimens. RESULTS: The primary end point of overall response rate (ORR) was met for both combinations. A confirmed partial response (PR) was observed in 10 of the 47 patients with NSCLC who received trametinib plus docetaxel (21%). The ORR was 18% (four PRs in 22 patients) in those with KRAS wild-type NSCLC versus 24% (six PRs in 25 patients) in those with KRAS-mutant NSCLC. Of the 42 patients with NSCLC treated with trametinib plus pemetrexed, six (14%) had a PR; the ORR was 17% (four of 23) in patients with KRAS-mutated NSCLC versus 11% (two of 19) in KRAS wild-type NSCLC. Adverse events-most commonly diarrhea, nausea, and fatigue-were manageable. CONCLUSIONS: Trametinib-plus-chemotherapy combinations were tolerable. Clinical activity exceeding the ORRs previously reported with docetaxel or pemetrexed alone in KRAS-mutated NSCLC and meeting prespecified criteria was observed.\n",
            "27876675\t30\t40\tTrametinib\tChemical\tMESH:C560077\n",
            "27876675\t46\t55\tDocetaxel\tChemical\tMESH:D000077143\n",
            "27876675\t59\t69\tPemetrexed\tChemical\tMESH:D000068437\n",
            "27876675\t73\t81\tPatients\tSpecies\t9606\n",
            "27876675\t96\t122\tNon-Small Cell Lung Cancer\tDisease\tMESH:D002289\n",
            "27876675\t166\t176\ttrametinib\tChemical\tMESH:C560077\n",
            "27876675\t180\t186\tMEK1/2\tGene\t5604;5605\n",
            "27876675\t251\t254\tMEK\tGene\t5609\n",
            "27876675\t281\t285\tKRAS\tGene\t3845\n",
            "27876675\t341\t345\tKRAS\tGene\t3845\n",
            "27876675\t353\t358\tNSCLC\tDisease\tMESH:D002289\n",
            "27876675\t432\t442\ttrametinib\tChemical\tMESH:C560077\n",
            "27876675\t552\t562\ttrametinib\tChemical\tMESH:C560077\n",
            "27876675\t579\t587\tpatients\tSpecies\t9606\n",
            "27876675\t593\t598\tNSCLC\tDisease\tMESH:D002289\n",
            "27876675\t616\t620\tKRAS\tGene\t3845\n",
            "27876675\t660\t670\ttrametinib\tChemical\tMESH:C560077\n",
            "27876675\t676\t685\tdocetaxel\tChemical\tMESH:D000077143\n",
            "27876675\t714\t724\ttrametinib\tChemical\tMESH:C560077\n",
            "27876675\t749\t758\tdocetaxel\tChemical\tMESH:D000077143\n",
            "27876675\t787\t797\tpemetrexed\tChemical\tMESH:D000068437\n",
            "27876675\t799\t809\ttrametinib\tChemical\tMESH:C560077\n",
            "27876675\t834\t844\tpemetrexed\tChemical\tMESH:D000068437\n",
            "27876675\t939\t944\tNSCLC\tDisease\tMESH:D002289\n",
            "27876675\t976\t980\tKRAS\tGene\t3845\n",
            "27876675\t1260\t1268\tpatients\tSpecies\t9606\n",
            "27876675\t1274\t1279\tNSCLC\tDisease\tMESH:D002289\n",
            "27876675\t1293\t1303\ttrametinib\tChemical\tMESH:C560077\n",
            "27876675\t1309\t1318\tdocetaxel\tChemical\tMESH:D000077143\n",
            "27876675\t1358\t1366\tpatients\tSpecies\t9606\n",
            "27876675\t1382\t1386\tKRAS\tGene\t3845\n",
            "27876675\t1397\t1402\tNSCLC\tDisease\tMESH:D002289\n",
            "27876675\t1429\t1437\tpatients\tSpecies\t9606\n",
            "27876675\t1453\t1457\tKRAS\tGene\t3845\n",
            "27876675\t1465\t1470\tNSCLC\tDisease\tMESH:D002289\n",
            "27876675\t1482\t1490\tpatients\tSpecies\t9606\n",
            "27876675\t1496\t1501\tNSCLC\tDisease\tMESH:D002289\n",
            "27876675\t1515\t1525\ttrametinib\tChemical\tMESH:C560077\n",
            "27876675\t1531\t1541\tpemetrexed\tChemical\tMESH:D000068437\n",
            "27876675\t1595\t1603\tpatients\tSpecies\t9606\n",
            "27876675\t1609\t1613\tKRAS\tGene\t3845\n",
            "27876675\t1622\t1627\tNSCLC\tDisease\tMESH:D002289\n",
            "27876675\t1654\t1658\tKRAS\tGene\t3845\n",
            "27876675\t1669\t1674\tNSCLC\tDisease\tMESH:D002289\n",
            "27876675\t1705\t1713\tdiarrhea\tDisease\tMESH:D003967\n",
            "27876675\t1715\t1721\tnausea\tDisease\tMESH:D009325\n",
            "27876675\t1727\t1734\tfatigue\tDisease\tMESH:D005221\n",
            "27876675\t1765\t1775\tTrametinib\tChemical\tMESH:C560077\n",
            "27876675\t1885\t1894\tdocetaxel\tChemical\tMESH:D000077143\n",
            "27876675\t1898\t1908\tpemetrexed\tChemical\tMESH:D000068437\n",
            "27876675\t1918\t1922\tKRAS\tGene\t3845\n",
            "27876675\t1931\t1936\tNSCLC\tDisease\tMESH:D002289\n",
            "\n",
            "28259530|t|Association between PD-L1 expression and driven gene status in NSCLC: A meta-analysis.\n",
            "28259530|a|PURPOSE: We explored the potential clinical association between programmed death-ligand 1 (PD-L1) expression and driven gene status in non-small cell lung cancer (NSCLC). METHODS: We systemically searched through October 2015. Odd ratios (ORs) with 95% CIs were calculated to examine the association of PD-L1 expression with driven gene status. A random- or fixed-effects model was used. RESULTS: Nine studies were identified. KRAS-mutant tumors were more likely to be PD-L1 positive than KRAS-wild type tumors (51% vs 36%; OR 1.69; 95% CI 1.01-2.84; p = 0.045). In contrast, PD-L1 expression did not differ by EGFR (OR 0.86; 95% CI 0.43-1.73; p = 0.675) or ALK (OR 1.02; 95% CI 0.44-2.37; p = 0.954) status. In subgroup analysis, there was also no significant association between PD-L1 expression and EGFR status in term of the cut-offs or ethnicity. CONCLUSION: In conclusion, NSCLC with KRAS mutations showed a trend for higher frequency of positive PD-L1 expression.\n",
            "28259530\t20\t25\tPD-L1\tGene\t29126\n",
            "28259530\t63\t68\tNSCLC\tDisease\tMESH:D002289\n",
            "28259530\t151\t176\tprogrammed death-ligand 1\tGene\t29126\n",
            "28259530\t178\t183\tPD-L1\tGene\t29126\n",
            "28259530\t222\t248\tnon-small cell lung cancer\tDisease\tMESH:D002289\n",
            "28259530\t250\t255\tNSCLC\tDisease\tMESH:D002289\n",
            "28259530\t314\t317\tOdd\tGene\t130497\n",
            "28259530\t390\t395\tPD-L1\tGene\t29126\n",
            "28259530\t514\t518\tKRAS\tGene\t3845\n",
            "28259530\t526\t532\ttumors\tDisease\tMESH:D009369\n",
            "28259530\t556\t561\tPD-L1\tGene\t29126\n",
            "28259530\t576\t580\tKRAS\tGene\t3845\n",
            "28259530\t591\t597\ttumors\tDisease\tMESH:D009369\n",
            "28259530\t663\t668\tPD-L1\tGene\t29126\n",
            "28259530\t698\t702\tEGFR\tGene\t1956\n",
            "28259530\t745\t748\tALK\tGene\t238\n",
            "28259530\t868\t873\tPD-L1\tGene\t29126\n",
            "28259530\t889\t893\tEGFR\tGene\t1956\n",
            "28259530\t966\t971\tNSCLC\tDisease\tMESH:D002289\n",
            "28259530\t977\t981\tKRAS\tGene\t3845\n",
            "28259530\t1040\t1045\tPD-L1\tGene\t29126\n",
            "\n",
            "28525386|t|Prognostic value of KRAS mutation in advanced non-small-cell lung cancer treated with immune checkpoint inhibitors: A meta-analysis and review.\n",
            "28525386|a|Immune checkpoint inhibitors (ICIs) have emerged as a promising treatment option in the fight against advanced non-small-cell lung cancer (NSCLC). KRAS is the most frequently mutated oncogene in NSCLC. We performed this meta-analysis to investigate if KRAS mutation status affects survival benefits of ICIs in patients with advanced NSCLC. Electronic databases were searched for eligible studies. We included randomized trials with the data of overall survival stratified by KRAS mutation status. From 3 eligible studies, 138 patients with KRAS mutant NSCLC and 371 with KRAS wild-type tumor were included in the meta-analysis. Compared to chemotherapy with docetaxel, ICIs improved overall survival in patients with previously treated KRAS mutant NSCLC (hazard ratio = 0.64 [95% confidence interval, 0.43-0.96], P = 0.03). For patients with KRAS wild-type NSCLC, however, ICIs did not prolong overall survival over that with chemotherapy (hazard ratio = 0.88 [95% confidence interval, 0.68-1.13], P = 0.30). In conclusion, ICIs as a salvage therapy improved overall survival over that with docetaxel in advanced NSCLC patients with KRAS mutation, but not in those with KRAS wild-type tumor. These results suggest that KRAS mutation status may be a potential biomarker for survival benefits to ICIs.\n",
            "28525386\t20\t24\tKRAS\tGene\t3845\n",
            "28525386\t61\t72\tlung cancer\tDisease\tMESH:D008175\n",
            "28525386\t270\t281\tlung cancer\tDisease\tMESH:D008175\n",
            "28525386\t283\t288\tNSCLC\tDisease\tMESH:D002289\n",
            "28525386\t291\t295\tKRAS\tGene\t3845\n",
            "28525386\t339\t344\tNSCLC\tDisease\tMESH:D002289\n",
            "28525386\t396\t400\tKRAS\tGene\t3845\n",
            "28525386\t454\t462\tpatients\tSpecies\t9606\n",
            "28525386\t477\t482\tNSCLC\tDisease\tMESH:D002289\n",
            "28525386\t619\t623\tKRAS\tGene\t3845\n",
            "28525386\t670\t678\tpatients\tSpecies\t9606\n",
            "28525386\t684\t688\tKRAS\tGene\t3845\n",
            "28525386\t696\t701\tNSCLC\tDisease\tMESH:D002289\n",
            "28525386\t715\t719\tKRAS\tGene\t3845\n",
            "28525386\t730\t735\ttumor\tDisease\tMESH:D009369\n",
            "28525386\t802\t811\tdocetaxel\tChemical\tMESH:D000077143\n",
            "28525386\t847\t855\tpatients\tSpecies\t9606\n",
            "28525386\t880\t884\tKRAS\tGene\t3845\n",
            "28525386\t892\t897\tNSCLC\tDisease\tMESH:D002289\n",
            "28525386\t972\t980\tpatients\tSpecies\t9606\n",
            "28525386\t986\t990\tKRAS\tGene\t3845\n",
            "28525386\t1001\t1006\tNSCLC\tDisease\tMESH:D002289\n",
            "28525386\t1235\t1244\tdocetaxel\tChemical\tMESH:D000077143\n",
            "28525386\t1257\t1262\tNSCLC\tDisease\tMESH:D002289\n",
            "28525386\t1263\t1271\tpatients\tSpecies\t9606\n",
            "28525386\t1277\t1281\tKRAS\tGene\t3845\n",
            "28525386\t1314\t1318\tKRAS\tGene\t3845\n",
            "28525386\t1329\t1334\ttumor\tDisease\tMESH:D009369\n",
            "28525386\t1363\t1367\tKRAS\tGene\t3845\n",
            "\n",
            "28602779|t|Ceritinib versus chemotherapy in patients with ALK-rearranged non-small-cell lung cancer previously given chemotherapy and crizotinib (ASCEND-5): a randomised, controlled, open-label, phase 3 trial.\n",
            "28602779|a|BACKGROUND: Ceritinib is a next-generation anaplastic lymphoma kinase (ALK) inhibitor, which has shown robust anti-tumour efficacy, along with intracranial activity, in patients with ALK-rearranged non-small-cell lung cancer. In phase 1 and 2 studies, ceritinib has been shown to be highly active in both ALK inhibitor-naive and ALK inhibitor-pretreated patients who had progressed after chemotherapy (mostly multiple lines). In this study, we compared the efficacy and safety of ceritinib versus single-agent chemotherapy in patients with advanced ALK-rearranged non-small-cell lung cancer who had previously progressed following crizotinib and platinum-based doublet chemotherapy. METHODS: In this randomised, controlled, open-label, phase 3 trial, we recruited patients aged at least 18 years with ALK-rearranged stage IIIB or IV non-small-cell lung cancer (with at least one measurable lesion) who had received previous chemotherapy (one or two lines, including a platinum doublet) and crizotinib and had subsequent disease progression, from 99 centres across 20 countries. Other inclusion criteria were a WHO performance status of 0-2, adequate organ function and laboratory test results, a life expectancy of at least 12 weeks, and having recovered from previous anticancer treatment-related toxicities. We randomly allocated patients (1:1; with blocking [block size of four]; stratified by WHO performance status [0 vs 1-2] and presence or absence of brain metastases) to oral ceritinib 750 mg per day fasted (in 21 day treatment cycles) or chemotherapy (intravenous pemetrexed 500 mg/m2 or docetaxel 75 mg/m2 [investigator choice], every 21 days). Patients who discontinued chemotherapy because of progressive disease could cross over to the ceritinib group. The primary endpoint was progression-free survival, assessed by a masked independent review committee using Response Evaluation Criteria in Solid Tumors 1.1 in the intention-to-treat population, assessed every 6 weeks until month 18 and every 9 weeks thereafter. This trial is registered with ClinicalTrials.gov, number NCT01828112, and is ongoing but no longer recruiting patients. FINDINGS: Between June 28, 2013, and Nov 2, 2015, we randomly allocated 231 patients; 115 (50%) to ceritinib and 116 (50%) to chemotherapy (40 [34%] to pemetrexed, 73 [63%] to docetaxel, and three [3%] discontinued before receiving treatment). Median follow-up was 16 5 months (IQR 11 5-21 4). Ceritinib showed a significant improvement in median progression-free survival compared with chemotherapy (5 4 months [95% CI 4 1-6 9] for ceritinib vs 1 6 months [1 4-2 8] for chemotherapy; hazard ratio 0 49 [0 36-0 67]; p<0 0001). Serious adverse events were reported in 49 (43%) of 115 patients in the ceritinib group and 36 (32%) of 113 in the chemotherapy group. Treatment-related serious adverse events were similar between groups (13 [11%] in the ceritinib group vs 12 [11%] in the chemotherapy group). The most frequent grade 3-4 adverse events in the ceritinib group were increased alanine aminotransferase concentration (24 [21%] of 115 vs two [2%] of 113 in the chemotherapy group), increased gamma glutamyltransferase concentration (24 [21%] vs one [1%]), and increased aspartate aminotransferase concentration (16 [14%] vs one [1%] in the chemotherapy group). Six (5%) of 115 patients in the ceritinib group discontinued because of adverse events compared with eight (7%) of 116 in the chemotherapy group. 15 (13%) of 115 patients in the ceritinib group and five (4%) of 113 in the chemotherapy group died during the treatment period (from the day of the first dose of study treatment to 30 days after the final dose). 13 (87%) of the 15 patients who died in the ceritinib group died because of disease progression and two (13%) died because of an adverse event (one [7%] cerebrovascular accident and one [7%] respiratory failure); neither of these deaths were considered by the investigator to be treatment related. The five (4%) deaths in the chemotherapy group were all due to disease progression. INTERPRETATION: These findings show that patients derive significant clinical benefit from a more potent ALK inhibitor after failure of crizotinib, and establish ceritinib as a more efficacious treatment option compared with chemotherapy in this patient population. FUNDING: Novartis Pharmaceuticals Corporation.\n",
            "28602779\t0\t9\tCeritinib\tChemical\tMESH:C586847\n",
            "28602779\t33\t41\tpatients\tSpecies\t9606\n",
            "28602779\t47\t50\tALK\tGene\t238\n",
            "28602779\t77\t88\tlung cancer\tDisease\tMESH:D008175\n",
            "28602779\t123\t133\tcrizotinib\tChemical\tMESH:D000077547\n",
            "28602779\t211\t220\tCeritinib\tChemical\tMESH:C586847\n",
            "28602779\t242\t268\tanaplastic lymphoma kinase\tGene\t238\n",
            "28602779\t270\t273\tALK\tGene\t238\n",
            "28602779\t314\t320\ttumour\tDisease\tMESH:D009369\n",
            "28602779\t368\t376\tpatients\tSpecies\t9606\n",
            "28602779\t382\t385\tALK\tGene\t238\n",
            "28602779\t412\t423\tlung cancer\tDisease\tMESH:D008175\n",
            "28602779\t451\t460\tceritinib\tChemical\tMESH:C586847\n",
            "28602779\t504\t507\tALK\tGene\t238\n",
            "28602779\t528\t531\tALK\tGene\t238\n",
            "28602779\t553\t561\tpatients\tSpecies\t9606\n",
            "28602779\t679\t688\tceritinib\tChemical\tMESH:C586847\n",
            "28602779\t725\t733\tpatients\tSpecies\t9606\n",
            "28602779\t748\t751\tALK\tGene\t238\n",
            "28602779\t778\t789\tlung cancer\tDisease\tMESH:D008175\n",
            "28602779\t830\t840\tcrizotinib\tChemical\tMESH:D000077547\n",
            "28602779\t845\t853\tplatinum\tChemical\tMESH:D010984\n",
            "28602779\t963\t971\tpatients\tSpecies\t9606\n",
            "28602779\t1000\t1003\tALK\tGene\t238\n",
            "28602779\t1032\t1058\tnon-small-cell lung cancer\tDisease\tMESH:D002289\n",
            "28602779\t1167\t1175\tplatinum\tChemical\tMESH:D010984\n",
            "28602779\t1189\t1199\tcrizotinib\tChemical\tMESH:D000077547\n",
            "28602779\t1497\t1507\ttoxicities\tDisease\tMESH:D064420\n",
            "28602779\t1531\t1539\tpatients\tSpecies\t9606\n",
            "28602779\t1663\t1673\tmetastases\tDisease\tMESH:D009362\n",
            "28602779\t1683\t1692\tceritinib\tChemical\tMESH:C586847\n",
            "28602779\t1773\t1783\tpemetrexed\tChemical\tMESH:D000068437\n",
            "28602779\t1797\t1806\tdocetaxel\tChemical\tMESH:D000077143\n",
            "28602779\t1855\t1863\tPatients\tSpecies\t9606\n",
            "28602779\t1905\t1924\tprogressive disease\tDisease\tMESH:D018450\n",
            "28602779\t1949\t1958\tceritinib\tChemical\tMESH:C586847\n",
            "28602779\t2339\t2347\tpatients\tSpecies\t9606\n",
            "28602779\t2386\t2391\tNov 2\tGene\t1490\n",
            "28602779\t2425\t2433\tpatients\tSpecies\t9606\n",
            "28602779\t2448\t2457\tceritinib\tChemical\tMESH:C586847\n",
            "28602779\t2501\t2511\tpemetrexed\tChemical\tMESH:D000068437\n",
            "28602779\t2525\t2534\tdocetaxel\tChemical\tMESH:D000077143\n",
            "28602779\t2643\t2652\tCeritinib\tChemical\tMESH:C586847\n",
            "28602779\t2782\t2791\tceritinib\tChemical\tMESH:C586847\n",
            "28602779\t2932\t2940\tpatients\tSpecies\t9606\n",
            "28602779\t2948\t2957\tceritinib\tChemical\tMESH:C586847\n",
            "28602779\t3097\t3106\tceritinib\tChemical\tMESH:C586847\n",
            "28602779\t3203\t3212\tceritinib\tChemical\tMESH:C586847\n",
            "28602779\t3532\t3540\tpatients\tSpecies\t9606\n",
            "28602779\t3548\t3557\tceritinib\tChemical\tMESH:C586847\n",
            "28602779\t3678\t3686\tpatients\tSpecies\t9606\n",
            "28602779\t3694\t3703\tceritinib\tChemical\tMESH:C586847\n",
            "28602779\t3757\t3761\tdied\tDisease\tMESH:D003643\n",
            "28602779\t3894\t3902\tpatients\tSpecies\t9606\n",
            "28602779\t3907\t3911\tdied\tDisease\tMESH:D003643\n",
            "28602779\t3919\t3928\tceritinib\tChemical\tMESH:C586847\n",
            "28602779\t3935\t3939\tdied\tDisease\tMESH:D003643\n",
            "28602779\t3985\t3989\tdied\tDisease\tMESH:D003643\n",
            "28602779\t4028\t4052\tcerebrovascular accident\tDisease\tMESH:D020521\n",
            "28602779\t4066\t4085\trespiratory failure\tDisease\tMESH:D012131\n",
            "28602779\t4105\t4111\tdeaths\tDisease\tMESH:D003643\n",
            "28602779\t4187\t4193\tdeaths\tDisease\tMESH:D003643\n",
            "28602779\t4298\t4306\tpatients\tSpecies\t9606\n",
            "28602779\t4362\t4365\tALK\tGene\t238\n",
            "28602779\t4393\t4403\tcrizotinib\tChemical\tMESH:D000077547\n",
            "28602779\t4419\t4428\tceritinib\tChemical\tMESH:C586847\n",
            "28602779\t4503\t4510\tpatient\tSpecies\t9606\n",
            "\n",
            "30413378|t|Lorlatinib in patients with ALK-positive non-small-cell lung cancer: results from a global phase 2 study.\n",
            "30413378|a|BACKGROUND: Lorlatinib is a potent, brain-penetrant, third-generation inhibitor of ALK and ROS1 tyrosine kinases with broad coverage of ALK mutations. In a phase 1 study, activity was seen in patients with ALK-positive non-small-cell lung cancer, most of whom had CNS metastases and progression after ALK-directed therapy. We aimed to analyse the overall and intracranial antitumour activity of lorlatinib in patients with ALK-positive, advanced non-small-cell lung cancer. METHODS: In this phase 2 study, patients with histologically or cytologically ALK-positive or ROS1-positive, advanced, non-small-cell lung cancer, with or without CNS metastases, with an Eastern Cooperative Oncology Group performance status of 0, 1, or 2, and adequate end-organ function were eligible. Patients were enrolled into six different expansion cohorts (EXP1-6) on the basis of ALK and ROS1 status and previous therapy, and were given lorlatinib 100 mg orally once daily continuously in 21-day cycles. The primary endpoint was overall and intracranial tumour response by independent central review, assessed in pooled subgroups of ALK-positive patients. Analyses of activity and safety were based on the safety analysis set (ie, all patients who received at least one dose of lorlatinib) as assessed by independent central review. Patients with measurable CNS metastases at baseline by independent central review were included in the intracranial activity analyses. In this report, we present lorlatinib activity data for the ALK-positive patients (EXP1-5 only), and safety data for all treated patients (EXP1-6). This study is ongoing and is registered with ClinicalTrials.gov, number NCT01970865. FINDINGS: Between Sept 15, 2015, and Oct 3, 2016, 276 patients were enrolled: 30 who were ALK positive and treatment naive (EXP1); 59 who were ALK positive and received previous crizotinib without (n=27; EXP2) or with (n=32; EXP3A) previous chemotherapy; 28 who were ALK positive and received one previous non-crizotinib ALK tyrosine kinase inhibitor, with or without chemotherapy (EXP3B); 112 who were ALK positive with two (n=66; EXP4) or three (n=46; EXP5) previous ALK tyrosine kinase inhibitors with or without chemotherapy; and 47 who were ROS1 positive with any previous treatment (EXP6). One patient in EXP4 died before receiving lorlatinib and was excluded from the safety analysis set. In treatment-naive patients (EXP1), an objective response was achieved in 27 (90 0%; 95% CI 73 5-97 9) of 30 patients. Three patients in EXP1 had measurable baseline CNS lesions per independent central review, and objective intracranial responses were observed in two (66 7%; 95% CI 9 4-99 2). In ALK-positive patients with at least one previous ALK tyrosine kinase inhibitor (EXP2-5), objective responses were achieved in 93 (47 0%; 39 9-54 2) of 198 patients and objective intracranial response in those with measurable baseline CNS lesions in 51 (63 0%; 51 5-73 4) of 81 patients. Objective response was achieved in 41 (69 5%; 95% CI 56 1-80 8) of 59 patients who had only received previous crizotinib (EXP2-3A), nine (32 1%; 15 9-52 4) of 28 patients with one previous non-crizotinib ALK tyrosine kinase inhibitor (EXP3B), and 43 (38 7%; 29 6-48 5) of 111 patients with two or more previous ALK tyrosine kinase inhibitors (EXP4-5). Objective intracranial response was achieved in 20 (87 0%; 95% CI 66 4-97 2) of 23 patients with measurable baseline CNS lesions in EXP2-3A, five (55 6%; 21 2-86 3) of nine patients in EXP3B, and 26 (53 1%; 38 3-67 5) of 49 patients in EXP4-5. The most common treatment-related adverse events across all patients were hypercholesterolaemia (224 [81%] of 275 patients overall and 43 [16%] grade 3-4) and hypertriglyceridaemia (166 [60%] overall and 43 [16%] grade 3-4). Serious treatment-related adverse events occurred in 19 (7%) of 275 patients and seven patients (3%) permanently discontinued treatment because of treatment-related adverse events. No treatment-related deaths were reported. INTERPRETATION: Consistent with its broad ALK mutational coverage and CNS penetration, lorlatinib showed substantial overall and intracranial activity both in treatment-naive patients with ALK-positive non-small-cell lung cancer, and in those who had progressed on crizotinib, second-generation ALK tyrosine kinase inhibitors, or after up to three previous ALK tyrosine kinase inhibitors. Thus, lorlatinib could represent an effective treatment option for patients with ALK-positive non-small-cell lung cancer in first-line or subsequent therapy. FUNDING: Pfizer.\n",
            "30413378\t0\t10\tLorlatinib\tChemical\tMESH:C000590786\n",
            "30413378\t14\t22\tpatients\tSpecies\t9606\n",
            "30413378\t28\t31\tALK\tGene\t238\n",
            "30413378\t56\t67\tlung cancer\tDisease\tMESH:D008175\n",
            "30413378\t118\t128\tLorlatinib\tChemical\tMESH:C000590786\n",
            "30413378\t189\t192\tALK\tGene\t238\n",
            "30413378\t197\t201\tROS1\tGene\t6098\n",
            "30413378\t242\t245\tALK\tGene\t238\n",
            "30413378\t298\t306\tpatients\tSpecies\t9606\n",
            "30413378\t312\t315\tALK\tGene\t238\n",
            "30413378\t340\t351\tlung cancer\tDisease\tMESH:D008175\n",
            "30413378\t374\t384\tmetastases\tDisease\tMESH:D009362\n",
            "30413378\t407\t410\tALK\tGene\t238\n",
            "30413378\t501\t511\tlorlatinib\tChemical\tMESH:C000590786\n",
            "30413378\t515\t523\tpatients\tSpecies\t9606\n",
            "30413378\t529\t532\tALK\tGene\t238\n",
            "30413378\t567\t578\tlung cancer\tDisease\tMESH:D008175\n",
            "30413378\t612\t620\tpatients\tSpecies\t9606\n",
            "30413378\t658\t661\tALK\tGene\t238\n",
            "30413378\t674\t678\tROS1\tGene\t6098\n",
            "30413378\t714\t725\tlung cancer\tDisease\tMESH:D008175\n",
            "30413378\t747\t757\tmetastases\tDisease\tMESH:D009362\n",
            "30413378\t883\t891\tPatients\tSpecies\t9606\n",
            "30413378\t944\t950\tEXP1-6\tGene\t7514;1434;64328;57510;23214\n",
            "30413378\t968\t971\tALK\tGene\t238\n",
            "30413378\t976\t980\tROS1\tGene\t6098\n",
            "30413378\t1025\t1035\tlorlatinib\tChemical\tMESH:C000590786\n",
            "30413378\t1129\t1148\tintracranial tumour\tDisease\tMESH:D001932\n",
            "30413378\t1221\t1224\tALK\tGene\t238\n",
            "30413378\t1234\t1242\tpatients\tSpecies\t9606\n",
            "30413378\t1323\t1331\tpatients\tSpecies\t9606\n",
            "30413378\t1366\t1376\tlorlatinib\tChemical\tMESH:C000590786\n",
            "30413378\t1421\t1429\tPatients\tSpecies\t9606\n",
            "30413378\t1450\t1460\tmetastases\tDisease\tMESH:D009362\n",
            "30413378\t1583\t1593\tlorlatinib\tChemical\tMESH:C000590786\n",
            "30413378\t1616\t1619\tALK\tGene\t238\n",
            "30413378\t1629\t1637\tpatients\tSpecies\t9606\n",
            "30413378\t1639\t1645\tEXP1-5\tGene\t7514;1434;64328;57510\n",
            "30413378\t1685\t1693\tpatients\tSpecies\t9606\n",
            "30413378\t1695\t1701\tEXP1-6\tGene\t7514;1434;64328;57510;23214\n",
            "30413378\t1843\t1851\tpatients\tSpecies\t9606\n",
            "30413378\t1879\t1882\tALK\tGene\t238\n",
            "30413378\t1913\t1917\tEXP1\tGene\t7514\n",
            "30413378\t1932\t1935\tALK\tGene\t238\n",
            "30413378\t1993\t1997\tEXP2\tGene\t1434\n",
            "30413378\t2056\t2059\tALK\tGene\t238\n",
            "30413378\t2192\t2195\tALK\tGene\t238\n",
            "30413378\t2221\t2225\tEXP4\tGene\t64328\n",
            "30413378\t2243\t2247\tEXP5\tGene\t57510\n",
            "30413378\t2258\t2261\tALK\tGene\t238\n",
            "30413378\t2335\t2339\tROS1\tGene\t6098\n",
            "30413378\t2378\t2382\tEXP6\tGene\t23214\n",
            "30413378\t2389\t2396\tpatient\tSpecies\t9606\n",
            "30413378\t2400\t2404\tEXP4\tGene\t64328\n",
            "30413378\t2427\t2437\tlorlatinib\tChemical\tMESH:C000590786\n",
            "30413378\t2504\t2512\tpatients\tSpecies\t9606\n",
            "30413378\t2514\t2518\tEXP1\tGene\t7514\n",
            "30413378\t2594\t2602\tpatients\tSpecies\t9606\n",
            "30413378\t2610\t2618\tpatients\tSpecies\t9606\n",
            "30413378\t2622\t2626\tEXP1\tGene\t7514\n",
            "30413378\t2782\t2785\tALK\tGene\t238\n",
            "30413378\t2795\t2803\tpatients\tSpecies\t9606\n",
            "30413378\t2862\t2868\tEXP2-5\tGene\t1434;64328;57510\n",
            "30413378\t2937\t2945\tpatients\tSpecies\t9606\n",
            "30413378\t3059\t3067\tpatients\tSpecies\t9606\n",
            "30413378\t3139\t3147\tpatients\tSpecies\t9606\n",
            "30413378\t3179\t3189\tcrizotinib\tChemical\tMESH:D000077547\n",
            "30413378\t3231\t3239\tpatients\tSpecies\t9606\n",
            "30413378\t3262\t3272\tcrizotinib\tChemical\tMESH:D000077547\n",
            "30413378\t3345\t3353\tpatients\tSpecies\t9606\n",
            "30413378\t3412\t3418\tEXP4-5\tGene\t64328;57510\n",
            "30413378\t3504\t3512\tpatients\tSpecies\t9606\n",
            "30413378\t3553\t3557\tEXP2\tGene\t1434\n",
            "30413378\t3594\t3602\tpatients\tSpecies\t9606\n",
            "30413378\t3645\t3653\tpatients\tSpecies\t9606\n",
            "30413378\t3657\t3663\tEXP4-5\tGene\t64328;57510\n",
            "30413378\t3725\t3733\tpatients\tSpecies\t9606\n",
            "30413378\t3739\t3760\thypercholesterolaemia\tDisease\t\n",
            "30413378\t3779\t3787\tpatients\tSpecies\t9606\n",
            "30413378\t3824\t3845\thypertriglyceridaemia\tDisease\t\n",
            "30413378\t3958\t3966\tpatients\tSpecies\t9606\n",
            "30413378\t3977\t3985\tpatients\tSpecies\t9606\n",
            "30413378\t4092\t4098\tdeaths\tDisease\tMESH:D003643\n",
            "30413378\t4156\t4159\tALK\tGene\t238\n",
            "30413378\t4201\t4211\tlorlatinib\tChemical\tMESH:C000590786\n",
            "30413378\t4289\t4297\tpatients\tSpecies\t9606\n",
            "30413378\t4303\t4306\tALK\tGene\t238\n",
            "30413378\t4331\t4342\tlung cancer\tDisease\tMESH:D008175\n",
            "30413378\t4379\t4389\tcrizotinib\tChemical\tMESH:D000077547\n",
            "30413378\t4471\t4474\tALK\tGene\t238\n",
            "30413378\t4509\t4519\tlorlatinib\tChemical\tMESH:C000590786\n",
            "30413378\t4570\t4578\tpatients\tSpecies\t9606\n",
            "30413378\t4584\t4587\tALK\tGene\t238\n",
            "30413378\t4612\t4623\tlung cancer\tDisease\tMESH:D008175\n",
            "\n",
            "34534430|t|Trastuzumab Deruxtecan in HER2-Mutant Non-Small-Cell Lung Cancer.\n",
            "34534430|a|BACKGROUND: Human epidermal growth factor receptor 2 (HER2)-targeted therapies have not been approved for patients with non-small-cell lung cancer (NSCLC). The efficacy and safety of trastuzumab deruxtecan (formerly DS-8201), a HER2 antibody-drug conjugate, in patients with HER2-mutant NSCLC have not been investigated extensively. METHODS: We conducted a multicenter, international, phase 2 study in which trastuzumab deruxtecan (6.4 mg per kilogram of body weight) was administered to patients who had metastatic HER2-mutant NSCLC that was refractory to standard treatment. The primary outcome was objective response as assessed by independent central review. Secondary outcomes included the duration of response, progression-free survival, overall survival, and safety. Biomarkers of HER2 alterations were assessed. RESULTS: A total of 91 patients were enrolled. The median duration of follow-up was 13.1 months (range, 0.7 to 29.1). Centrally confirmed objective response occurred in 55% of the patients (95% confidence interval [CI], 44 to 65). The median duration of response was 9.3 months (95% CI, 5.7 to 14.7). Median progression-free survival was 8.2 months (95% CI, 6.0 to 11.9), and median overall survival was 17.8 months (95% CI, 13.8 to 22.1). The safety profile was generally consistent with those from previous studies; grade 3 or higher drug-related adverse events occurred in 46% of patients, the most common event being neutropenia (in 19%). Adjudicated drug-related interstitial lung disease occurred in 26% of patients and resulted in death in 2 patients. Responses were observed across different HER2 mutation subtypes, as well as in patients with no detectable HER2 expression or HER2 amplification. CONCLUSIONS: Trastuzumab deruxtecan showed durable anticancer activity in patients with previously treated HER2-mutant NSCLC. The safety profile included interstitial lung disease that was fatal in two cases. Observed toxic effects were generally consistent with those in previously reported studies. (Funded by Daiichi Sankyo and AstraZeneca; DESTINY-Lung01 ClinicalTrials.gov number, NCT03505710.).\n",
            "34534430\t0\t11\tTrastuzumab\tChemical\tMESH:D000068878\n",
            "34534430\t12\t22\tDeruxtecan\tChemical\t-\n",
            "34534430\t26\t30\tHER2\tGene\t2064\n",
            "34534430\t38\t64\tNon-Small-Cell Lung Cancer\tDisease\tMESH:D002289\n",
            "34534430\t78\t118\tHuman epidermal growth factor receptor 2\tGene\t2064\n",
            "34534430\t120\t124\tHER2\tGene\t2064\n",
            "34534430\t172\t180\tpatients\tSpecies\t9606\n",
            "34534430\t186\t212\tnon-small-cell lung cancer\tDisease\tMESH:D002289\n",
            "34534430\t214\t219\tNSCLC\tDisease\tMESH:D002289\n",
            "34534430\t249\t260\ttrastuzumab\tChemical\tMESH:D000068878\n",
            "34534430\t261\t271\tderuxtecan\tChemical\t-\n",
            "34534430\t282\t289\tDS-8201\tChemical\tMESH:C000614160\n",
            "34534430\t294\t298\tHER2\tGene\t2064\n",
            "34534430\t327\t335\tpatients\tSpecies\t9606\n",
            "34534430\t341\t345\tHER2\tGene\t2064\n",
            "34534430\t353\t358\tNSCLC\tDisease\tMESH:D002289\n",
            "34534430\t474\t485\ttrastuzumab\tChemical\tMESH:D000068878\n",
            "34534430\t486\t496\tderuxtecan\tChemical\t-\n",
            "34534430\t554\t562\tpatients\tSpecies\t9606\n",
            "34534430\t582\t586\tHER2\tGene\t2064\n",
            "34534430\t594\t599\tNSCLC\tDisease\tMESH:D002289\n",
            "34534430\t854\t858\tHER2\tGene\t2064\n",
            "34534430\t909\t917\tpatients\tSpecies\t9606\n",
            "34534430\t1066\t1074\tpatients\tSpecies\t9606\n",
            "34534430\t1469\t1477\tpatients\tSpecies\t9606\n",
            "34534430\t1507\t1518\tneutropenia\tDisease\tMESH:D009503\n",
            "34534430\t1567\t1579\tlung disease\tDisease\tMESH:D008171\n",
            "34534430\t1599\t1607\tpatients\tSpecies\t9606\n",
            "34534430\t1624\t1629\tdeath\tDisease\tMESH:D003643\n",
            "34534430\t1635\t1643\tpatients\tSpecies\t9606\n",
            "34534430\t1686\t1690\tHER2\tGene\t2064\n",
            "34534430\t1724\t1732\tpatients\tSpecies\t9606\n",
            "34534430\t1752\t1756\tHER2\tGene\t2064\n",
            "34534430\t1771\t1775\tHER2\tGene\t2064\n",
            "34534430\t1804\t1815\tTrastuzumab\tChemical\tMESH:D000068878\n",
            "34534430\t1816\t1826\tderuxtecan\tChemical\t-\n",
            "34534430\t1865\t1873\tpatients\tSpecies\t9606\n",
            "34534430\t1898\t1902\tHER2\tGene\t2064\n",
            "34534430\t1910\t1915\tNSCLC\tDisease\tMESH:D002289\n",
            "34534430\t1958\t1970\tlung disease\tDisease\tMESH:D008171\n",
            "\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "def get_entities_from_pubtator_format(pubtator_output):\n",
        "  entities = []\n",
        "  lines = pubtator_output.split(\"\\n\")\n",
        "  for line in lines:\n",
        "    tokens = line.split(\"\\t\")\n",
        "    if len(tokens) < 6: continue\n",
        "    pmid, start_offset, end_offset, entity_mention, entity_type, entity_id = tokens[0:6]\n",
        "    entities.append([pmid, start_offset, end_offset, entity_mention, entity_type, entity_id])\n",
        "  return entities"
      ],
      "metadata": {
        "id": "-EogxvcYLnHx"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "pubtator_output=process_pmids(selected_pmids_str, \"pubtator\", \"\")"
      ],
      "metadata": {
        "id": "IW8mUJaOPTuZ"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "pubtator_entities = get_entities_from_pubtator_format(pubtator_output)"
      ],
      "metadata": {
        "id": "Hk9IQs6mPZLj"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "pubtator_entities"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "6aFouUMNPa0i",
        "outputId": "94801069-abfe-4955-914f-c3c272058d7d"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "[['25765070', '0', '6', 'Cancer', 'Disease', 'MESH:D009369'],\n",
              " ['25765070', '157', '164', 'patient', 'Species', '9606'],\n",
              " ['25765070', '187', '193', 'tumors', 'Disease', 'MESH:D009369'],\n",
              " ['25765070', '347', '359', 'lung cancers', 'Disease', 'MESH:D008175'],\n",
              " ['25765070', '373', '386', 'pembrolizumab', 'Chemical', 'MESH:C582435'],\n",
              " ['25765070', '510', '516', 'tumors', 'Disease', 'MESH:D009369'],\n",
              " ['25765070', '843', '849', 'T cell', 'CellLine', 'T cell'],\n",
              " ['25765070', '871', '876', 'tumor', 'Disease', 'MESH:D009369'],\n",
              " ['25765070', '952', '958', 'T cell', 'CellLine', 'T cell'],\n",
              " ['25765070', '1021', '1033', 'lung cancers', 'Disease', 'MESH:D008175'],\n",
              " ['25891174', '0', '13', 'Pembrolizumab', 'Chemical', 'MESH:C582435'],\n",
              " ['25891174', '50', '61', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['25891174', '114', '137', 'programmed cell death 1', 'Gene', '5133'],\n",
              " ['25891174', '139', '143', 'PD-1', 'Gene', '5133'],\n",
              " ['25891174', '161', '174', 'pembrolizumab', 'Chemical', 'MESH:C582435'],\n",
              " ['25891174', '178', '186', 'patients', 'Species', '9606'],\n",
              " ['25891174', '216', '227', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['25891174', '322', '335', 'PD-1 ligand 1', 'Gene', '29126'],\n",
              " ['25891174', '337', '342', 'PD-L1', 'Gene', '29126'],\n",
              " ['25891174', '429', '437', 'patients', 'Species', '9606'],\n",
              " ['25891174', '448', '461', 'pembrolizumab', 'Chemical', 'MESH:C582435'],\n",
              " ['25891174', '608', '616', 'patients', 'Species', '9606'],\n",
              " ['25891174', '645', '653', 'patients', 'Species', '9606'],\n",
              " ['25891174', '668', '673', 'PD-L1', 'Gene', '29126'],\n",
              " ['25891174', '688', '693', 'tumor', 'Disease', 'MESH:D009369'],\n",
              " ['25891174', '827', '832', 'PD-L1', 'Gene', '29126'],\n",
              " ['25891174', '961', '974', 'pembrolizumab', 'Chemical', 'MESH:C582435'],\n",
              " ['25891174', '980', '987', 'fatigue', 'Disease', 'MESH:D005221'],\n",
              " ['25891174', '989', '997', 'pruritus', 'Disease', 'MESH:D011537'],\n",
              " ['25891174', '1093', '1101', 'patients', 'Species', '9606'],\n",
              " ['25891174', '1321', '1326', 'PD-L1', 'Gene', '29126'],\n",
              " ['25891174', '1357', '1362', 'tumor', 'Disease', 'MESH:D009369'],\n",
              " ['25891174', '1427', '1435', 'patients', 'Species', '9606'],\n",
              " ['25891174', '1544', '1552', 'patients', 'Species', '9606'],\n",
              " ['25891174', '1697', '1710', 'Pembrolizumab', 'Chemical', 'MESH:C582435'],\n",
              " ['25891174', '1782', '1790', 'patients', 'Species', '9606'],\n",
              " ['25891174', '1820', '1831', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['25891174', '1833', '1838', 'PD-L1', 'Gene', '29126'],\n",
              " ['25891174', '1869', '1874', 'tumor', 'Disease', 'MESH:D009369'],\n",
              " ['25891174', '1918', '1931', 'pembrolizumab', 'Chemical', 'MESH:C582435'],\n",
              " ['26027660', '0', '4', 'LKB1', 'Gene', '6794'],\n",
              " ['26027660', '57', '68', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['26027660', '78', '82', 'mTOR', 'Gene', '2475'],\n",
              " ['26027660', '139', '158', 'endometrial cancers', 'Disease', 'MESH:D016889'],\n",
              " ['26027660', '164', '168', 'LKB1', 'Gene', '6794'],\n",
              " ['26027660', '207', '211', 'mTOR', 'Gene', '2475'],\n",
              " ['26027660', '264', '268', 'LKB1', 'Gene', '6794'],\n",
              " ['26027660', '284', '288', 'mTOR', 'Gene', '2475'],\n",
              " ['26027660', '327', '338', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['26027660', '340', '345', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['26027660', '363', '374', 'Lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['26027660', '394', '398', 'H460', 'CellLine', 'CVCL:0459'],\n",
              " ['26027660', '400', '405', 'H1299', 'CellLine', 'CVCL:0060'],\n",
              " ['26027660', '407', '412', 'H1792', 'CellLine', 'CVCL:1495'],\n",
              " ['26027660', '418', '422', 'A549', 'CellLine', 'CVCL:0023'],\n",
              " ['26027660', '445', '449', 'mTOR', 'Gene', '2475'],\n",
              " ['26027660', '461', '470', 'rapamycin', 'Chemical', 'MESH:D020123'],\n",
              " ['26027660', '474', '484', 'everolimus', 'Chemical', 'MESH:D000068338'],\n",
              " ['26027660', '499', '503', 'mTOR', 'Gene', '2475'],\n",
              " ['26027660', '563', '568', '4EBP1', 'Gene', '1978'],\n",
              " ['26027660', '573', '576', 'S6K', 'Gene', '6198'],\n",
              " ['26027660', '594', '598', 'mTOR', 'Gene', '2475'],\n",
              " ['26027660', '654', '670', 'sulforhodamine B', 'Chemical', 'MESH:C022027'],\n",
              " ['26027660', '707', '711', 'mTOR', 'Gene', '2475'],\n",
              " ['26027660', '724', '728', 'LKB1', 'Gene', '6794'],\n",
              " ['26027660', '736', '740', 'A549', 'CellLine', 'CVCL:0023'],\n",
              " ['26027660', '745', '749', 'H460', 'CellLine', 'CVCL:0459'],\n",
              " ['26027660', '794', '798', 'LKB1', 'Gene', '6794'],\n",
              " ['26027660', '820', '825', 'H1792', 'CellLine', 'CVCL:1495'],\n",
              " ['26027660', '846', '850', 'LKB1', 'Gene', '6794'],\n",
              " ['26027660', '858', '862', 'A549', 'CellLine', 'CVCL:0023'],\n",
              " ['26027660', '867', '871', 'H460', 'CellLine', 'CVCL:0459'],\n",
              " ['26027660', '909', '913', 'mTOR', 'Gene', '2475'],\n",
              " ['26027660', '934', '938', 'LKB1', 'Gene', '6794'],\n",
              " ['26027660', '960', '965', 'H1792', 'CellLine', 'CVCL:1495'],\n",
              " ['26027660', '996', '1000', 'LKB1', 'Gene', '6794'],\n",
              " ['26027660', '1009', '1014', 'H1299', 'CellLine', 'CVCL:0060'],\n",
              " ['26027660', '1061', '1065', 'mTOR', 'Gene', '2475'],\n",
              " ['26027660', '1093', '1102', 'rapamycin', 'Chemical', 'MESH:D020123'],\n",
              " ['26027660', '1106', '1112', 'RAD001', 'Chemical', 'MESH:D000068338'],\n",
              " ['26027660', '1160', '1163', 'AKT', 'Gene', '207'],\n",
              " ['26027660', '1172', '1176', 'LKB1', 'Gene', '6794'],\n",
              " ['26027660', '1191', '1195', 'LKB1', 'Gene', '6794'],\n",
              " ['26027660', '1203', '1208', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['26027660', '1259', '1267', 'LY294002', 'Chemical', 'MESH:C085911'],\n",
              " ['26027660', '1303', '1311', 'LY294002', 'Chemical', 'MESH:C085911'],\n",
              " ['26027660', '1355', '1359', 'LKB1', 'Gene', '6794'],\n",
              " ['26027660', '1370', '1375', 'H1792', 'CellLine', 'CVCL:1495'],\n",
              " ['26027660', '1386', '1390', 'LKB1', 'Gene', '6794'],\n",
              " ['26027660', '1398', '1402', 'A549', 'CellLine', 'CVCL:0023'],\n",
              " ['26027660', '1422', '1426', 'LKB1', 'Gene', '6794'],\n",
              " ['26027660', '1448', '1453', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['26027660', '1501', '1505', 'mTOR', 'Gene', '2475'],\n",
              " ['26027660', '1554', '1557', 'AKT', 'Gene', '207'],\n",
              " ['26027660', '1561', '1565', 'mTOR', 'Gene', '2475'],\n",
              " ['26027660', '1613', '1618', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['26027660', '1628', '1632', 'mTOR', 'Gene', '2475'],\n",
              " ['26124334', '56', '64', 'Patients', 'Species', '9606'],\n",
              " ['26124334',\n",
              "  '70',\n",
              "  '96',\n",
              "  'Non-small Cell Lung Cancer',\n",
              "  'Disease',\n",
              "  'MESH:D002289'],\n",
              " ['26124334',\n",
              "  '113',\n",
              "  '145',\n",
              "  'Epidermal Growth Factor Receptor',\n",
              "  'Gene',\n",
              "  '1956'],\n",
              " ['26124334',\n",
              "  '240',\n",
              "  '266',\n",
              "  'Non-small cell lung cancer',\n",
              "  'Disease',\n",
              "  'MESH:D002289'],\n",
              " ['26124334', '268', '273', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['26124334',\n",
              "  '303',\n",
              "  '335',\n",
              "  'epidermal growth factor receptor',\n",
              "  'Gene',\n",
              "  '1956'],\n",
              " ['26124334', '337', '341', 'EGFR', 'Gene', '1956'],\n",
              " ['26124334',\n",
              "  '413',\n",
              "  '418',\n",
              "  'L858R',\n",
              "  'ProteinMutation',\n",
              "  'tmVar:p|SUB|L|858|R;HGVS:p.L858R;VariantGroup:0;CorrespondingGene:1956;RS#:121434568;CA#:126713'],\n",
              " ['26124334', '481', '488', 'patient', 'Species', '9606'],\n",
              " ['26124334', '606', '610', 'EGFR', 'Gene', '1956'],\n",
              " ['26124334', '648', '656', 'patients', 'Species', '9606'],\n",
              " ['26124334', '662', '667', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['26124334', '701', '705', 'EGFR', 'Gene', '1956'],\n",
              " ['26124334', '712', '720', 'PATIENTS', 'Species', '9606'],\n",
              " ['26124334', '807', '815', 'patients', 'Species', '9606'],\n",
              " ['26124334', '821', '826', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['26124334', '841', '845', 'EGFR', 'Gene', '1956'],\n",
              " ['26124334', '883', '887', 'EGFR', 'Gene', '1956'],\n",
              " ['26124334', '946', '954', 'patients', 'Species', '9606'],\n",
              " ['26124334', '983', '987', 'EGFR', 'Gene', '1956'],\n",
              " ['26124334', '1022', '1031', 'gefitinib', 'Chemical', 'MESH:D000077156'],\n",
              " ['26124334', '1035', '1044', 'erlotinib', 'Chemical', 'MESH:D000069347'],\n",
              " ['26124334',\n",
              "  '1066',\n",
              "  '1071',\n",
              "  'G719X',\n",
              "  'ProteinMutation',\n",
              "  'tmVar:p|SUB|G|719|X;HGVS:p.G719X;VariantGroup:1;CorrespondingGene:1956;RS#:28929495'],\n",
              " ['26124334',\n",
              "  '1091',\n",
              "  '1096',\n",
              "  'L861Q',\n",
              "  'ProteinMutation',\n",
              "  'tmVar:p|SUB|L|861|Q;HGVS:p.L861Q;VariantGroup:2;CorrespondingGene:1956;RS#:121913444;CA#:176021'],\n",
              " ['26124334',\n",
              "  '1120',\n",
              "  '1125',\n",
              "  'G874S',\n",
              "  'ProteinMutation',\n",
              "  'tmVar:p|SUB|G|874|S;HGVS:p.G874S;VariantGroup:3;CorrespondingGene:1956'],\n",
              " ['26124334', '1150', '1158', 'patients', 'Species', '9606'],\n",
              " ['26124334',\n",
              "  '1172',\n",
              "  '1177',\n",
              "  'G719S',\n",
              "  'ProteinMutation',\n",
              "  'tmVar:p|SUB|G|719|S;HGVS:p.G719S;VariantGroup:1;CorrespondingGene:1956;RS#:28929495;CA#:126722'],\n",
              " ['26124334',\n",
              "  '1186',\n",
              "  '1191',\n",
              "  'L861Q',\n",
              "  'ProteinMutation',\n",
              "  'tmVar:p|SUB|L|861|Q;HGVS:p.L861Q;VariantGroup:2;CorrespondingGene:1956;RS#:121913444;CA#:176021'],\n",
              " ['26124334', '1457', '1465', 'patients', 'Species', '9606'],\n",
              " ['26124334', '1516', '1520', 'EGFR', 'Gene', '1956'],\n",
              " ['26124334', '1597', '1605', 'patients', 'Species', '9606'],\n",
              " ['26124334', '1619', '1623', 'EGFR', 'Gene', '1956'],\n",
              " ['26124334', '1695', '1704', 'erlotinib', 'Chemical', 'MESH:D000069347'],\n",
              " ['26124334', '1751', '1760', 'gefitinib', 'Chemical', 'MESH:D000077156'],\n",
              " ['26124334', '1784', '1792', 'Patients', 'Species', '9606'],\n",
              " ['26124334', '1798', '1803', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['26124334', '1837', '1841', 'EGFR', 'Gene', '1956'],\n",
              " ['26124334', '1878', '1882', 'EGFR', 'Gene', '1956'],\n",
              " ['26124334', '1930', '1934', 'EGFR', 'Gene', '1956'],\n",
              " ['26124334', '1955', '1964', 'erlotinib', 'Chemical', 'MESH:D000069347'],\n",
              " ['26124334', '2019', '2027', 'patients', 'Species', '9606'],\n",
              " ['26124334', '2033', '2038', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['26124334', '2050', '2054', 'EGFR', 'Gene', '1956'],\n",
              " ['26209642', '9', '13', 'KRAS', 'Gene', '3845'],\n",
              " ['26209642', '52', '57', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['26209642', '140', '144', 'KRAS', 'Gene', '3845'],\n",
              " ['26209642',\n",
              "  '167',\n",
              "  '192',\n",
              "  'nonsmall-cell lung cancer',\n",
              "  'Disease',\n",
              "  'MESH:D002289'],\n",
              " ['26209642', '194', '199', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['26209642', '288', '292', 'KRAS', 'Gene', '3845'],\n",
              " ['26209642', '306', '311', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['26209642', '312', '320', 'patients', 'Species', '9606'],\n",
              " ['26209642', '334', '343', 'erlotinib', 'Chemical', 'MESH:D000069347'],\n",
              " ['26209642', '347', '356', 'docetaxel', 'Chemical', 'MESH:D000077143'],\n",
              " ['26209642', '373', '381', 'PATIENTS', 'Species', '9606'],\n",
              " ['26209642', '395', '400', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['26209642', '401', '409', 'patients', 'Species', '9606'],\n",
              " ['26209642', '455', '459', 'KRAS', 'Gene', '3845'],\n",
              " ['26209642', '464', '468', 'EGFR', 'Gene', '1956'],\n",
              " ['26209642', '530', '534', 'EGFR', 'Gene', '1956'],\n",
              " ['26209642', '535', '543', 'patients', 'Species', '9606'],\n",
              " ['26209642', '574', '582', 'platinum', 'Chemical', 'MESH:D010984'],\n",
              " ['26209642', '648', '657', 'erlotinib', 'Chemical', 'MESH:D000069347'],\n",
              " ['26209642', '661', '670', 'docetaxel', 'Chemical', 'MESH:D000077143'],\n",
              " ['26209642', '707', '711', 'KRAS', 'Gene', '3845'],\n",
              " ['26209642', '766', '770', 'KRAS', 'Gene', '3845'],\n",
              " ['26209642', '847', '851', 'KRAS', 'Gene', '3845'],\n",
              " ['26209642', '1143', '1151', 'Patients', 'Species', '9606'],\n",
              " ['26209642', '1166', '1175', 'docetaxel', 'Chemical', 'MESH:D000077143'],\n",
              " ['26209642', '1237', '1241', 'KRAS', 'Gene', '3845'],\n",
              " ['26209642', '1269', '1275', 'tumors', 'Disease', 'MESH:D009369'],\n",
              " ['26209642', '1289', '1293', 'KRAS', 'Gene', '3845'],\n",
              " ['26209642', '1328', '1332', 'KRAS', 'Gene', '3845'],\n",
              " ['26209642', '1381', '1385', 'KRAS', 'Gene', '3845'],\n",
              " ['26209642', '1446', '1455', 'docetaxel', 'Chemical', 'MESH:D000077143'],\n",
              " ['26209642', '1459', '1468', 'erlotinib', 'Chemical', 'MESH:D000069347'],\n",
              " ['26209642', '1470', '1479', 'Docetaxel', 'Chemical', 'MESH:D000077143'],\n",
              " ['26209642', '1516', '1520', 'KRAS', 'Gene', '3845'],\n",
              " ['26209642', '1557', '1561', 'EGFR', 'Gene', '1956'],\n",
              " ['26209642', '1581', '1586', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['26209642', '1587', '1595', 'patients', 'Species', '9606'],\n",
              " ['26287849', '0', '11', 'Vemurafenib', 'Chemical', 'MESH:D000077484'],\n",
              " ['26287849',\n",
              "  '15',\n",
              "  '43',\n",
              "  'Multiple Nonmelanoma Cancers',\n",
              "  'Disease',\n",
              "  'MESH:D009369'],\n",
              " ['26287849', '49', '53', 'BRAF', 'Gene', '673'],\n",
              " ['26287849', '82', '86', 'BRAF', 'Gene', '673'],\n",
              " ['26287849', '119', '138', 'nonmelanoma cancers', 'Disease', 'MESH:D009369'],\n",
              " ['26287849', '203', '214', 'vemurafenib', 'Chemical', 'MESH:D000077484'],\n",
              " ['26287849', '218', '222', 'BRAF', 'Gene', '673'],\n",
              " ['26287849', '246', '265', 'nonmelanoma cancers', 'Disease', 'MESH:D009369'],\n",
              " ['26287849', '288', '296', 'patients', 'Species', '9606'],\n",
              " ['26287849', '317', '323', 'cancer', 'Disease', 'MESH:D009369'],\n",
              " ['26287849', '333', '341', 'patients', 'Species', '9606'],\n",
              " ['26287849', '357', '362', 'tumor', 'Disease', 'MESH:D009369'],\n",
              " ['26287849', '419', '427', 'patients', 'Species', '9606'],\n",
              " ['26287849', '433', '437', 'BRAF', 'Gene', '673'],\n",
              " ['26287849', '461', '467', 'cancer', 'Disease', 'MESH:D009369'],\n",
              " ['26287849', '495', '503', 'patients', 'Species', '9606'],\n",
              " ['26287849', '509', '526', 'colorectal cancer', 'Disease', 'MESH:D015179'],\n",
              " ['26287849', '540', '551', 'vemurafenib', 'Chemical', 'MESH:D000077484'],\n",
              " ['26287849', '556', '565', 'cetuximab', 'Chemical', 'MESH:D000068818'],\n",
              " ['26287849', '724', '735', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['26287849',\n",
              "  '898',\n",
              "  '921',\n",
              "  'Erdheim-Chester disease',\n",
              "  'Disease',\n",
              "  'MESH:D031249'],\n",
              " ['26287849', '1077', '1085', 'patients', 'Species', '9606'],\n",
              " ['26287849', '1163', '1171', 'patients', 'Species', '9606'],\n",
              " ['26287849',\n",
              "  '1177',\n",
              "  '1233',\n",
              "  'pleomorphic xanthoastrocytoma, anaplastic thyroid cancer',\n",
              "  'Disease',\n",
              "  'MESH:D065646'],\n",
              " ['26287849', '1235', '1253', 'cholangiocarcinoma', 'Disease', 'MESH:D018281'],\n",
              " ['26287849',\n",
              "  '1255',\n",
              "  '1275',\n",
              "  'salivary-duct cancer',\n",
              "  'Disease',\n",
              "  'MESH:D009369'],\n",
              " ['26287849', '1277', '1291', 'ovarian cancer', 'Disease', 'MESH:D010051'],\n",
              " ['26287849', '1297', '1315', 'clear-cell sarcoma', 'Disease', 'MESH:D018227'],\n",
              " ['26287849', '1326', '1334', 'patients', 'Species', '9606'],\n",
              " ['26287849', '1340', '1357', 'colorectal cancer', 'Disease', 'MESH:D015179'],\n",
              " ['26287849', '1371', '1382', 'vemurafenib', 'Chemical', 'MESH:D000077484'],\n",
              " ['26287849', '1387', '1396', 'cetuximab', 'Chemical', 'MESH:D000068818'],\n",
              " ['26287849', '1445', '1456', 'vemurafenib', 'Chemical', 'MESH:D000077484'],\n",
              " ['26287849', '1461', '1469', 'melanoma', 'Disease', 'MESH:D008545'],\n",
              " ['26287849', '1484', '1488', 'BRAF', 'Gene', '673'],\n",
              " ['26287849',\n",
              "  '1552',\n",
              "  '1571',\n",
              "  'nonmelanoma cancers',\n",
              "  'Disease',\n",
              "  'MESH:D009369'],\n",
              " ['26287849', '1637', '1648', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['26287849',\n",
              "  '1656',\n",
              "  '1679',\n",
              "  'Erdheim-Chester disease',\n",
              "  'Disease',\n",
              "  'MESH:D031249'],\n",
              " ['26287849', '1782', '1786', 'BRAF', 'Gene', '673'],\n",
              " ['26287849', '1800', '1807', 'cancers', 'Disease', 'MESH:D009369'],\n",
              " ['26439803', '4', '8', 'EGFR', 'Gene', '1956'],\n",
              " ['26439803', '9', '12', 'MET', 'Gene', '79811'],\n",
              " ['26439803', '52', '56', 'EGFR', 'Gene', '1956'],\n",
              " ['26439803', '86', '97', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['26439803',\n",
              "  '145',\n",
              "  '177',\n",
              "  'epidermal growth factor receptor',\n",
              "  'Gene',\n",
              "  '1956'],\n",
              " ['26439803', '179', '183', 'EGFR', 'Gene', '1956'],\n",
              " ['26439803', '229', '233', 'EGFR', 'Gene', '1956'],\n",
              " ['26439803', '256', '260', 'EGFR', 'Gene', '1956'],\n",
              " ['26439803', '283', '294', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['26439803', '303', '311', 'patients', 'Species', '9606'],\n",
              " ['26439803', '563', '569', 'tumors', 'Disease', 'MESH:D009369'],\n",
              " ['26439803', '585', '593', 'patients', 'Species', '9606'],\n",
              " ['26439803', '625', '629', 'EGFR', 'Gene', '1956'],\n",
              " ['26439803', '673', '681', 'patients', 'Species', '9606'],\n",
              " ['26439803', '701', '705', 'EGFR', 'Gene', '1956'],\n",
              " ['26439803', '777', '785', 'patients', 'Species', '9606'],\n",
              " ['26439803', '801', '805', 'EGFR', 'Gene', '1956'],\n",
              " ['26439803', '867', '872', 'tumor', 'Disease', 'MESH:D009369'],\n",
              " ['26439803', '915', '919', 'EGFR', 'Gene', '1956'],\n",
              " ['26439803', '954', '958', 'EGFR', 'Gene', '1956'],\n",
              " ['26439803', '971', '974', 'MET', 'Gene', '79811'],\n",
              " ['26439803', '1015', '1019', 'EGFR', 'Gene', '1956'],\n",
              " ['26439803', '1020', '1023', 'MET', 'Gene', '79811'],\n",
              " ['26439803', '1235', '1239', 'EGFR', 'Gene', '1956'],\n",
              " ['26439803', '1240', '1243', 'MET', 'Gene', '79811'],\n",
              " ['26439803', '1262', '1268', 'tumors', 'Disease', 'MESH:D009369'],\n",
              " ['26439803', '1305', '1313', 'patients', 'Species', '9606'],\n",
              " ['26439803', '1374', '1378', 'EGFR', 'Gene', '1956'],\n",
              " ['26515464', '115', '119', 'EGFR', 'Gene', '1956'],\n",
              " ['26515464', '146', '157', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['26515464', '159', '163', 'EGFR', 'Gene', '1956'],\n",
              " ['26515464', '172', '183', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['26515464', '238', '249', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['26515464', '323', '327', 'EGFR', 'Gene', '1956'],\n",
              " ['26515464', '372', '376', 'EGFR', 'Gene', '1956'],\n",
              " ['26515464', '454', '458', 'EGFR', 'Gene', '1956'],\n",
              " ['26515464', '489', '493', 'EGFR', 'Gene', '1956'],\n",
              " ['26515464', '606', '610', 'EGFR', 'Gene', '1956'],\n",
              " ['26515464', '641', '645', 'EGFR', 'Gene', '1956'],\n",
              " ['26515464', '667', '671', 'EGFR', 'Gene', '1956'],\n",
              " ['26515464', '721', '730', 'erlotinib', 'Chemical', 'MESH:D000069347'],\n",
              " ['26515464', '738', '746', 'afatinib', 'Chemical', 'MESH:D000077716'],\n",
              " ['26515464', '757', '768', 'osimertinib', 'Chemical', '-'],\n",
              " ['26515464', '773', '784', 'rociletinib', 'Chemical', 'MESH:C000589977'],\n",
              " ['26515464', '797', '801', 'EGFR', 'Gene', '1956'],\n",
              " ['26515464', '833', '838', 'human', 'Species', '9606'],\n",
              " ['26515464', '839', '850', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['26515464', '866', '871', 'Ba/F3', 'CellLine', 'CVCL_0161;NCBITaxID:10090'],\n",
              " ['26515464', '926', '930', 'EGFR', 'Gene', '13649'],\n",
              " ['26515464', '1056', '1060', 'EGFR', 'Gene', '13649'],\n",
              " ['26515464', '1121', '1129', 'afatinib', 'Chemical', 'MESH:D000077716'],\n",
              " ['26515464',\n",
              "  '1156',\n",
              "  '1161',\n",
              "  'L858R',\n",
              "  'ProteinMutation',\n",
              "  'tmVar:p|SUB|L|858|R;HGVS:p.L858R;VariantGroup:1;CorrespondingGene:1956;RS#:121434568;CA#:126713'],\n",
              " ['26515464', '1169', '1180', 'osimertinib', 'Chemical', '-'],\n",
              " ['26515464', '1185', '1196', 'rociletinib', 'Chemical', 'MESH:C000589977'],\n",
              " ['26515464',\n",
              "  '1201',\n",
              "  '1206',\n",
              "  'T790M',\n",
              "  'ProteinMutation',\n",
              "  'tmVar:p|SUB|T|790|M;HGVS:p.T790M;VariantGroup:0;CorrespondingGene:1956;RS#:121434569;CA#:90928'],\n",
              " ['26515464', '1352', '1356', 'EGFR', 'Gene', '13649'],\n",
              " ['26515464', '1452', '1456', 'EGFR', 'Gene', '13649'],\n",
              " ['26515464', '1463', '1474', 'osimertinib', 'Chemical', '-'],\n",
              " ['26515464', '1616', '1627', 'osimertinib', 'Chemical', '-'],\n",
              " ['26515464', '1632', '1636', 'EGFR', 'Gene', '13649'],\n",
              " ['26515464', '1753', '1757', 'EGFR', 'Gene', '13649'],\n",
              " ['26515464', '1791', '1795', 'EGFR', 'Gene', '13649'],\n",
              " ['26554404', '4', '7', 'ALK', 'Gene', '238'],\n",
              " ['26554404', '8', '12', 'ROS1', 'Gene', '6098'],\n",
              " ['26554404', '23', '34', 'PF-06463922', 'Chemical', 'MESH:C000590786'],\n",
              " ['26554404', '67', '77', 'Crizotinib', 'Chemical', 'MESH:D000077547'],\n",
              " ['26554404', '81', '84', 'ALK', 'Gene', '238'],\n",
              " ['26554404', '92', '105', 'Neuroblastoma', 'Disease', 'MESH:D009447'],\n",
              " ['26554404', '119', '133', 'Neuroblastomas', 'Disease', 'MESH:D009447'],\n",
              " ['26554404', '174', '200', 'anaplastic lymphoma kinase', 'Gene', '238'],\n",
              " ['26554404', '202', '205', 'ALK', 'Gene', '238'],\n",
              " ['26554404', '243', '246', 'ALK', 'Gene', '238'],\n",
              " ['26554404', '257', '267', 'crizotinib', 'Chemical', 'MESH:D000077547'],\n",
              " ['26554404', '313', '323', 'crizotinib', 'Chemical', 'MESH:D000077547'],\n",
              " ['26554404', '458', '461', 'ALK', 'Gene', '238'],\n",
              " ['26554404',\n",
              "  '479',\n",
              "  '485',\n",
              "  'F1174L',\n",
              "  'ProteinMutation',\n",
              "  'tmVar:p|SUB|F|1174|L;HGVS:p.F1174L;VariantGroup:0;CorrespondingGene:238;RS#:863225281;CA#:279616'],\n",
              " ['26554404', '500', '511', 'PF-06463922', 'Chemical', 'MESH:C000590786'],\n",
              " ['26554404', '536', '539', 'ALK', 'Gene', '238'],\n",
              " ['26554404', '562', '565', 'ALK', 'Gene', '238'],\n",
              " ['26554404', '588', '598', 'crizotinib', 'Chemical', 'MESH:D000077547'],\n",
              " ['26554404', '627', '638', 'PF-06463922', 'Chemical', 'MESH:C000590786'],\n",
              " ['26554404', '656', '661', 'tumor', 'Disease', 'MESH:D009369'],\n",
              " ['26554404', '681', '691', 'crizotinib', 'Chemical', 'MESH:D000077547'],\n",
              " ['26554404', '706', '716', 'crizotinib', 'Chemical', 'MESH:D000077547'],\n",
              " ['26554404', '737', '742', 'mouse', 'Species', '10090'],\n",
              " ['26554404', '753', '766', 'neuroblastoma', 'Disease', 'MESH:D009447'],\n",
              " ['26554404', '782', '789', 'patient', 'Species', '9606'],\n",
              " ['26554404', '823', '833', 'crizotinib', 'Chemical', 'MESH:D000077547'],\n",
              " ['26554404',\n",
              "  '844',\n",
              "  '850',\n",
              "  'F1174L',\n",
              "  'ProteinMutation',\n",
              "  'tmVar:p|SUB|F|1174|L;HGVS:p.F1174L;VariantGroup:0;CorrespondingGene:238;RS#:863225281;CA#:279616'],\n",
              " ['26554404',\n",
              "  '854',\n",
              "  '860',\n",
              "  'F1245C',\n",
              "  'ProteinMutation',\n",
              "  'tmVar:p|SUB|F|1245|C;HGVS:p.F1245C;VariantGroup:1;CorrespondingGene:238;RS#:863225283;CA#:279597'],\n",
              " ['26554404', '903', '914', 'PF-06463922', 'Chemical', 'MESH:C000590786'],\n",
              " ['26554404', '945', '955', 'crizotinib', 'Chemical', 'MESH:D000077547'],\n",
              " ['26554404',\n",
              "  '1036',\n",
              "  '1042',\n",
              "  'F1174L',\n",
              "  'ProteinMutation',\n",
              "  'tmVar:p|SUB|F|1174|L;HGVS:p.F1174L;VariantGroup:0;CorrespondingGene:238;RS#:863225281;CA#:279616'],\n",
              " ['26554404',\n",
              "  '1047',\n",
              "  '1053',\n",
              "  'F1245C',\n",
              "  'ProteinMutation',\n",
              "  'tmVar:p|SUB|F|1245|C;HGVS:p.F1245C;VariantGroup:1;CorrespondingGene:238;RS#:863225283;CA#:279597'],\n",
              " ['26554404', '1054', '1057', 'ALK', 'Gene', '238'],\n",
              " ['26554404', '1076', '1082', 'tumors', 'Disease', 'MESH:D009369'],\n",
              " ['26554404',\n",
              "  '1120',\n",
              "  '1126',\n",
              "  'R1275Q',\n",
              "  'ProteinMutation',\n",
              "  'tmVar:p|SUB|R|1275|Q;HGVS:p.R1275Q;VariantGroup:2;CorrespondingGene:238;RS#:113994087;CA#:341482'],\n",
              " ['26554404', '1204', '1215', 'PF-06463922', 'Chemical', 'MESH:C000590786'],\n",
              " ['26554404', '1254', '1262', 'patients', 'Species', '9606'],\n",
              " ['26554404', '1268', '1271', 'ALK', 'Gene', '238'],\n",
              " ['26554404', '1280', '1293', 'neuroblastoma', 'Disease', 'MESH:D009447'],\n",
              " ['26554404', '1329', '1332', 'ALK', 'Gene', '238'],\n",
              " ['26554404', '1333', '1337', 'ROS1', 'Gene', '6098'],\n",
              " ['26554404', '1348', '1359', 'PF-06463922', 'Chemical', 'MESH:C000590786'],\n",
              " ['26554404', '1392', '1395', 'ALK', 'Gene', '238'],\n",
              " ['26554404', '1403', '1416', 'neuroblastoma', 'Disease', 'MESH:D009447'],\n",
              " ['26554404', '1437', '1447', 'crizotinib', 'Chemical', 'MESH:D000077547'],\n",
              " ['26554404', '1578', '1586', 'children', 'Species', '9606'],\n",
              " ['26554404', '1612', '1615', 'ALK', 'Gene', '238'],\n",
              " ['26554404', '1623', '1636', 'neuroblastoma', 'Disease', 'MESH:D009447'],\n",
              " ['26758680', '20', '24', 'ABL1', 'Gene', '25'],\n",
              " ['26758680', '56', '61', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['26758680', '113', '121', 'patients', 'Species', '9606'],\n",
              " ['26758680',\n",
              "  '127',\n",
              "  '153',\n",
              "  'non-small cell lung cancer',\n",
              "  'Disease',\n",
              "  'MESH:D002289'],\n",
              " ['26758680', '155', '160', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['26758680', '284', '288', 'ABL1', 'Gene', '25'],\n",
              " ['26758680', '342', '347', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['26758680', '383', '388', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['26758680', '400', '404', 'ABL1', 'Gene', '25'],\n",
              " ['26758680', '432', '435', 'ABL', 'Gene', '25'],\n",
              " ['26758680', '559', '563', 'ABL1', 'Gene', '25'],\n",
              " ['26758680', '601', '609', 'imatinib', 'Chemical', 'MESH:D000068877'],\n",
              " ['26758680', '621', '631', 'lung tumor', 'Disease', 'MESH:D008175'],\n",
              " ['26758680', '664', '675', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['26758680', '729', '733', 'ABL1', 'Gene', '25'],\n",
              " ['26758680', '805', '809', 'ABL1', 'Gene', '25'],\n",
              " ['26758680', '832', '837', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['26758680', '851', '862', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['26758680', '934', '938', 'ABL1', 'Gene', '25'],\n",
              " ['26758680', '969', '973', 'ABL1', 'Gene', '25'],\n",
              " ['26758680', '974', '980', 'cancer', 'Disease', 'MESH:D009369'],\n",
              " ['26758680', '1091', '1095', 'ABL1', 'Gene', '25'],\n",
              " ['26758680', '1141', '1146', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['26758680', '1147', '1155', 'patients', 'Species', '9606'],\n",
              " ['26758680', '1161', '1165', 'ABL1', 'Gene', '25'],\n",
              " ['26758680', '1215', '1223', 'imatinib', 'Chemical', 'MESH:D000068877'],\n",
              " ['26771872', '24', '29', 'PD-L1', 'Gene', '29126'],\n",
              " ['26771872', '55', '58', 'CD8', 'Gene', '925'],\n",
              " ['26771872', '105', '111', 'cancer', 'Disease', 'MESH:D009369'],\n",
              " ['26771872', '112', '120', 'patients', 'Species', '9606'],\n",
              " ['26771872', '187', '217', 'programmed cell death-ligand 1', 'Gene', '29126'],\n",
              " ['26771872', '219', '224', 'PD-L1', 'Gene', '29126'],\n",
              " ['26771872', '261', '267', 'cancer', 'Disease', 'MESH:D009369'],\n",
              " ['26771872', '360', '368', 'patients', 'Species', '9606'],\n",
              " ['26771872',\n",
              "  '391',\n",
              "  '417',\n",
              "  'non-small cell lung cancer',\n",
              "  'Disease',\n",
              "  'MESH:D002289'],\n",
              " ['26771872', '419', '424', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['26771872', '471', '476', 'PD-L1', 'Gene', '29126'],\n",
              " ['26771872', '492', '495', 'CD8', 'Gene', '925'],\n",
              " ['26771872', '547', '555', 'patients', 'Species', '9606'],\n",
              " ['26771872', '578', '583', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['26771872', '683', '691', 'patients', 'Species', '9606'],\n",
              " ['26771872', '707', '712', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['26771872', '736', '741', 'PD-L1', 'Gene', '29126'],\n",
              " ['26771872', '757', '760', 'CD8', 'Gene', '925'],\n",
              " ['26771872', '886', '889', 'CD8', 'Gene', '925'],\n",
              " ['26771872', '1008', '1013', 'PD-L1', 'Gene', '29126'],\n",
              " ['26771872', '1092', '1097', 'PD-L1', 'Gene', '29126'],\n",
              " ['26771872', '1099', '1102', 'CD8', 'Gene', '925'],\n",
              " ['26771872', '1202', '1207', 'PD-L1', 'Gene', '29126'],\n",
              " ['26771872', '1209', '1212', 'CD8', 'Gene', '925'],\n",
              " ['26771872', '1363', '1368', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['26771872', '1369', '1377', 'patients', 'Species', '9606'],\n",
              " ['26771872', '1456', '1461', 'PD-L1', 'Gene', '29126'],\n",
              " ['26771872', '1516', '1521', 'PD-L1', 'Gene', '29126'],\n",
              " ['26771872', '1537', '1540', 'CD8', 'Gene', '925'],\n",
              " ['26771872', '1617', '1625', 'patients', 'Species', '9606'],\n",
              " ['26771872', '1647', '1652', 'PD-L1', 'Gene', '29126'],\n",
              " ['26771872', '1684', '1687', 'CD8', 'Gene', '925'],\n",
              " ['26771872', '1743', '1751', 'patients', 'Species', '9606'],\n",
              " ['26771872', '1767', '1772', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['26899019', '36', '40', 'TP53', 'Gene', '7157'],\n",
              " ['26899019', '54', '62', 'Patients', 'Species', '9606'],\n",
              " ['26899019',\n",
              "  '68',\n",
              "  '94',\n",
              "  'Non-Small Cell Lung Cancer',\n",
              "  'Disease',\n",
              "  'MESH:D002289'],\n",
              " ['26899019', '109', '118', 'Cisplatin', 'Chemical', 'MESH:D002945'],\n",
              " ['26899019', '218', '222', 'TP53', 'Gene', '7157'],\n",
              " ['26899019', '283', '294', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['26899019', '521', '525', 'TP53', 'Gene', '7157'],\n",
              " ['26899019', '616', '627', 'Lung Cancer', 'Disease', 'MESH:D008175'],\n",
              " ['26899019', '650', '656', 'Cancer', 'Disease', 'MESH:D009369'],\n",
              " ['26899019', '661', '669', 'Leukemia', 'Disease', 'MESH:D007938'],\n",
              " ['26899019', '697', '706', 'Navelbine', 'Chemical', 'MESH:D000077235'],\n",
              " ['26899019', '752', '760', 'platinum', 'Chemical', 'MESH:D010984'],\n",
              " ['26899019', '908', '912', 'TP53', 'Gene', '7157'],\n",
              " ['26899019', '1172', '1176', 'TP53', 'Gene', '7157'],\n",
              " ['26899019', '1203', '1211', 'patients', 'Species', '9606'],\n",
              " ['26899019', '1271', '1277', 'deaths', 'Disease', 'MESH:D003643'],\n",
              " ['26899019', '1558', '1566', 'Patients', 'Species', '9606'],\n",
              " ['26899019', '1575', '1579', 'TP53', 'Gene', '7157'],\n",
              " ['26899019', '1804', '1808', 'TP53', 'Gene', '7157'],\n",
              " ['26899019', '1923', '1927', 'TP53', 'Gene', '7157'],\n",
              " ['26899019', '2018', '2026', 'patients', 'Species', '9606'],\n",
              " ['26899019', '2045', '2049', 'TP53', 'Gene', '7157'],\n",
              " ['26899019', '2120', '2124', 'TP53', 'Gene', '7157'],\n",
              " ['26918046', '23', '27', 'PIM1', 'Gene', '5292'],\n",
              " ['26918046', '88', '96', 'Patients', 'Species', '9606'],\n",
              " ['26918046',\n",
              "  '102',\n",
              "  '149',\n",
              "  'Lung Adenocarcinoma and Squamous Cell Carcinoma',\n",
              "  'Disease',\n",
              "  'MESH:D002294'],\n",
              " ['26918046', '209', '213', 'PIM1', 'Gene', '5292'],\n",
              " ['26918046', '217', '222', 'STAT3', 'Gene', '6774'],\n",
              " ['26918046', '227', '232', 'c-MYC', 'Gene', '4609'],\n",
              " ['26918046', '282', '288', 'tumors', 'Disease', 'MESH:D009369'],\n",
              " ['26918046', '350', '361', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['26918046', '454', '462', 'patients', 'Species', '9606'],\n",
              " ['26918046',\n",
              "  '468',\n",
              "  '487',\n",
              "  'lung adenocarcinoma',\n",
              "  'Disease',\n",
              "  'MESH:D000077192'],\n",
              " ['26918046',\n",
              "  '497',\n",
              "  '520',\n",
              "  'squamous cell carcinoma',\n",
              "  'Disease',\n",
              "  'MESH:D002294'],\n",
              " ['26918046', '522', '525', 'SCC', 'Disease', 'MESH:D002294'],\n",
              " ['26918046', '621', '625', 'PIM1', 'Gene', '5292'],\n",
              " ['26918046', '679', '683', 'PIM1', 'Gene', '5292'],\n",
              " ['26918046', '687', '692', 'STAT3', 'Gene', '6774'],\n",
              " ['26918046', '698', '703', 'c-MYC', 'Gene', '4609'],\n",
              " ['26918046', '740', '754', 'archival tumor', 'Disease', 'MESH:D009369'],\n",
              " ['26918046', '784', '787', 'SCC', 'Disease', 'MESH:D002294'],\n",
              " ['26918046', '788', '796', 'patients', 'Species', '9606'],\n",
              " ['26918046', '811', '815', 'PIM1', 'Gene', '5292'],\n",
              " ['26918046', '983', '987', 'PIM1', 'Gene', '5292'],\n",
              " ['26918046', '1180', '1184', 'PIM1', 'Gene', '5292'],\n",
              " ['26918046', '1375', '1380', 'STAT3', 'Gene', '6774'],\n",
              " ['26918046', '1474', '1479', 'c-MYC', 'Gene', '4609'],\n",
              " ['26918046', '1586', '1589', 'SCC', 'Disease', 'MESH:D002294'],\n",
              " ['26918046', '1590', '1598', 'patients', 'Species', '9606'],\n",
              " ['26918046', '1608', '1612', 'PIM1', 'Gene', '5292'],\n",
              " ['26938871', '31', '40', 'alectinib', 'Chemical', 'MESH:C582670'],\n",
              " ['26938871', '47', '73', 'anaplastic lymphoma kinase', 'Gene', '238'],\n",
              " ['26938871', '105', '111', 'cancer', 'Disease', 'MESH:D009369'],\n",
              " ['26938871', '112', '119', 'patient', 'Species', '9606'],\n",
              " ['26938871', '127', '141', 'critically ill', 'Disease', 'MESH:D016638'],\n",
              " ['26938871', '143', '169', 'Anaplastic lymphoma kinase', 'Gene', '238'],\n",
              " ['26938871', '171', '174', 'ALK', 'Gene', '238'],\n",
              " ['26938871', '284', '295', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['26938871', '317', '320', 'ALK', 'Gene', '238'],\n",
              " ['26938871', '356', '359', 'ALK', 'Gene', '238'],\n",
              " ['26938871', '365', '374', 'alectinib', 'Chemical', 'MESH:C582670'],\n",
              " ['26938871', '421', '424', 'ALK', 'Gene', '238'],\n",
              " ['26938871', '440', '448', 'patients', 'Species', '9606'],\n",
              " ['26938871', '498', '508', 'crizotinib', 'Chemical', 'MESH:D000077547'],\n",
              " ['26938871', '530', '533', 'ALK', 'Gene', '238'],\n",
              " ['26938871', '551', '554', 'ALK', 'Gene', '238'],\n",
              " ['26938871', '598', '601', 'ALK', 'Gene', '238'],\n",
              " ['26938871', '618', '628', 'crizotinib', 'Chemical', 'MESH:D000077547'],\n",
              " ['26938871', '633', '642', 'alectinib', 'Chemical', 'MESH:C582670'],\n",
              " ['26938871', '675', '689', 'critically ill', 'Disease', 'MESH:D016638'],\n",
              " ['26938871', '690', '698', 'patients', 'Species', '9606'],\n",
              " ['26938871', '725', '728', 'ALK', 'Gene', '238'],\n",
              " ['26938871', '744', '751', 'patient', 'Species', '9606'],\n",
              " ['26938871',\n",
              "  '790',\n",
              "  '828',\n",
              "  'disseminated intravascular coagulation',\n",
              "  'Disease',\n",
              "  'MESH:D004211'],\n",
              " ['26938871', '840', '859', 'respiratory failure', 'Disease', 'MESH:D012131'],\n",
              " ['26938871', '873', '883', 'metastases', 'Disease', 'MESH:D009362'],\n",
              " ['26938871', '936', '945', 'alectinib', 'Chemical', 'MESH:C582670'],\n",
              " ['26938871', '1077', '1086', 'alectinib', 'Chemical', 'MESH:C582670'],\n",
              " ['26938871', '1215', '1224', 'alectinib', 'Chemical', 'MESH:C582670'],\n",
              " ['26938871', '1228', '1231', 'ALK', 'Gene', '238'],\n",
              " ['26938871', '1247', '1255', 'patients', 'Species', '9606'],\n",
              " ['26938871', '1283', '1300', 'organ dysfunction', 'Disease', 'MESH:D009102'],\n",
              " ['26938871',\n",
              "  '1310',\n",
              "  '1348',\n",
              "  'disseminated intravascular coagulation',\n",
              "  'Disease',\n",
              "  'MESH:D004211'],\n",
              " ['26938871', '1402', '1405', 'ALK', 'Gene', '238'],\n",
              " ['26938871', '1414', '1417', 'ALK', 'Gene', '238'],\n",
              " ['26938871', '1433', '1441', 'patients', 'Species', '9606'],\n",
              " ['26938871', '1450', '1464', 'critically ill', 'Disease', 'MESH:D016638'],\n",
              " ['27022118', '25', '35', 'Crizotinib', 'Chemical', 'MESH:D000077547'],\n",
              " ['27022118', '59', '67', 'Patients', 'Species', '9606'],\n",
              " ['27022118', '82', '85', 'ALK', 'Gene', '238'],\n",
              " ['27022118', '195', '205', 'crizotinib', 'Chemical', 'MESH:D000077547'],\n",
              " ['27022118', '292', '295', 'ALK', 'Gene', '238'],\n",
              " ['27022118', '320', '331', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['27022118', '333', '341', 'PATIENTS', 'Species', '9606'],\n",
              " ['27022118', '355', '363', 'Patients', 'Species', '9606'],\n",
              " ['27022118', '398', '408', 'crizotinib', 'Chemical', 'MESH:D000077547'],\n",
              " ['27022118', '456', '466', 'pemetrexed', 'Chemical', 'MESH:D000068437'],\n",
              " ['27022118', '484', '493', 'cisplatin', 'Chemical', 'MESH:D002945'],\n",
              " ['27022118', '508', '519', 'carboplatin', 'Chemical', 'MESH:D016190'],\n",
              " ['27022118', '595', '603', 'Patients', 'Species', '9606'],\n",
              " ['27022118', '630', '640', 'metastases', 'Disease', 'MESH:D009362'],\n",
              " ['27022118', '736', '744', 'patients', 'Species', '9606'],\n",
              " ['27022118', '773', '783', 'metastases', 'Disease', 'MESH:D009362'],\n",
              " ['27022118', '825', '830', 'tumor', 'Disease', 'MESH:D009369'],\n",
              " ['27022118', '870', '890', 'intracranial disease', 'Disease', 'MESH:D020765'],\n",
              " ['27022118', '1004', '1012', 'patients', 'Species', '9606'],\n",
              " ['27022118', '1127', '1137', 'crizotinib', 'Chemical', 'MESH:D000077547'],\n",
              " ['27022118', '1209', '1217', 'patients', 'Species', '9606'],\n",
              " ['27022118', '1253', '1261', 'patients', 'Species', '9606'],\n",
              " ['27022118', '1301', '1309', 'patients', 'Species', '9606'],\n",
              " ['27022118', '1357', '1367', 'crizotinib', 'Chemical', 'MESH:D000077547'],\n",
              " ['27022118', '1542', '1552', 'crizotinib', 'Chemical', 'MESH:D000077547'],\n",
              " ['27022118', '1878', '1888', 'crizotinib', 'Chemical', 'MESH:D000077547'],\n",
              " ['27022118', '1935', '1943', 'patients', 'Species', '9606'],\n",
              " ['27022118', '2015', '2023', 'patients', 'Species', '9606'],\n",
              " ['27022118', '2040', '2043', 'tBM', 'Chemical', '-'],\n",
              " ['27315356', '23', '32', 'dovitinib', 'Chemical', 'MESH:C500007'],\n",
              " ['27315356', '47', '55', 'patients', 'Species', '9606'],\n",
              " ['27315356',\n",
              "  '70',\n",
              "  '105',\n",
              "  'squamous non-small cell lung cancer',\n",
              "  'Disease',\n",
              "  'MESH:D002289'],\n",
              " ['27315356', '111', '116', 'FGFR1', 'Gene', '2260'],\n",
              " ['27315356',\n",
              "  '173',\n",
              "  '208',\n",
              "  'Fibroblast growth factor receptor 1',\n",
              "  'Gene',\n",
              "  '2260'],\n",
              " ['27315356', '210', '215', 'FGFR1', 'Gene', '2260'],\n",
              " ['27315356', '266', '286', 'squamous cell cancer', 'Disease', 'MESH:D002294'],\n",
              " ['27315356', '288', '291', 'SCC', 'Disease', 'MESH:D002294'],\n",
              " ['27315356', '375', '384', 'dovitinib', 'Chemical', 'MESH:C500007'],\n",
              " ['27315356', '389', '393', 'FGFR', 'Gene', '2260'],\n",
              " ['27315356', '408', '416', 'patients', 'Species', '9606'],\n",
              " ['27315356', '431', '434', 'SCC', 'Disease', 'MESH:D002294'],\n",
              " ['27315356', '457', '465', 'Patients', 'Species', '9606'],\n",
              " ['27315356', '491', '494', 'SCC', 'Disease', 'MESH:D002294'],\n",
              " ['27315356', '513', '519', 'tumors', 'Disease', 'MESH:D009369'],\n",
              " ['27315356', '533', '538', 'FGFR1', 'Gene', '2260'],\n",
              " ['27315356', '620', '629', 'Dovitinib', 'Chemical', 'MESH:C500007'],\n",
              " ['27315356', '858', '866', 'toxicity', 'Disease', 'MESH:D064420'],\n",
              " ['27315356', '884', '892', 'patients', 'Species', '9606'],\n",
              " ['27315356', '898', '901', 'men', 'Species', '9606'],\n",
              " ['27315356', '970', '978', 'patients', 'Species', '9606'],\n",
              " ['27315356', '1021', '1030', 'dovitinib', 'Chemical', 'MESH:C500007'],\n",
              " ['27315356', '1256', '1264', 'patients', 'Species', '9606'],\n",
              " ['27315356', '1321', '1329', 'patients', 'Species', '9606'],\n",
              " ['27315356', '1687', '1694', 'fatigue', 'Disease', 'MESH:D005221'],\n",
              " ['27315356', '1704', '1712', 'anorexia', 'Disease', 'MESH:D000855'],\n",
              " ['27315356', '1726', '1738', 'hyponatremia', 'Disease', 'MESH:D007010'],\n",
              " ['27315356', '1792', '1798', 'Cancer', 'Disease', 'MESH:D009369'],\n",
              " ['27315356', '1900', '1909', 'dovitinib', 'Chemical', 'MESH:C500007'],\n",
              " ['27315356', '1942', '1950', 'patients', 'Species', '9606'],\n",
              " ['27315356', '1965', '1968', 'SCC', 'Disease', 'MESH:D002294'],\n",
              " ['27315356', '1974', '1979', 'FGFR1', 'Gene', '2260'],\n",
              " ['27315356', '2072', '2081', 'dovitinib', 'Chemical', 'MESH:C500007'],\n",
              " ['27315356', '2085', '2093', 'patients', 'Species', '9606'],\n",
              " ['27315356', '2099', '2102', 'SCC', 'Disease', 'MESH:D002294'],\n",
              " ['27315356', '2118', '2124', 'Cancer', 'Disease', 'MESH:D009369'],\n",
              " ['27315356', '2161', '2167', 'Cancer', 'Disease', 'MESH:D009369'],\n",
              " ['27544060', '21', '30', 'Alectinib', 'Chemical', 'MESH:C582670'],\n",
              " ['27544060', '43', '46', 'RET', 'Gene', '5979'],\n",
              " ['27544060',\n",
              "  '58',\n",
              "  '84',\n",
              "  'Non-Small Cell Lung Cancer',\n",
              "  'Disease',\n",
              "  'MESH:D002289'],\n",
              " ['27544060', '174', '177', 'RET', 'Gene', '5979'],\n",
              " ['27544060', '200', '206', 'NSCLCs', 'Disease', 'MESH:D002289'],\n",
              " ['27544060', '269', '272', 'RET', 'Gene', '5979'],\n",
              " ['27544060', '286', '295', 'Alectinib', 'Chemical', 'MESH:C582670'],\n",
              " ['27544060', '302', '321', 'anaplastic lymphoma', 'Disease', 'MESH:D017728'],\n",
              " ['27544060', '380', '383', 'RET', 'Gene', '5979'],\n",
              " ['27544060', '428', '437', 'alectinib', 'Chemical', 'MESH:C582670'],\n",
              " ['27544060', '441', '449', 'patients', 'Species', '9606'],\n",
              " ['27544060', '455', '458', 'RET', 'Gene', '5979'],\n",
              " ['27544060', '470', '475', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['27544060', '535', '543', 'patients', 'Species', '9606'],\n",
              " ['27544060', '558', '561', 'RET', 'Gene', '5979'],\n",
              " ['27544060', '573', '578', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['27544060', '601', '610', 'alectinib', 'Chemical', 'MESH:C582670'],\n",
              " ['27544060', '688', '695', 'patient', 'Species', '9606'],\n",
              " ['27544060', '791', '799', 'patients', 'Species', '9606'],\n",
              " ['27544060', '816', '819', 'RET', 'Gene', '5979'],\n",
              " ['27544060', '831', '836', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['27544060', '891', '894', 'RET', 'Gene', '5979'],\n",
              " ['27544060', '911', '923', 'cabozantinib', 'Chemical', 'MESH:C558660'],\n",
              " ['27544060', '941', '944', 'RET', 'Gene', '5979'],\n",
              " ['27544060', '1043', '1052', 'alectinib', 'Chemical', 'MESH:C582670'],\n",
              " ['27544060', '1212', '1220', 'patients', 'Species', '9606'],\n",
              " ['27544060', '1237', '1246', 'alectinib', 'Chemical', 'MESH:C582670'],\n",
              " ['27544060', '1334', '1344', 'metastases', 'Disease', 'MESH:D009362'],\n",
              " ['27544060', '1363', '1370', 'patient', 'Species', '9606'],\n",
              " ['27544060', '1480', '1488', 'toxicity', 'Disease', 'MESH:D064420'],\n",
              " ['27544060', '1500', '1507', 'patient', 'Species', '9606'],\n",
              " ['27544060', '1516', '1519', 'RET', 'Gene', '5979'],\n",
              " ['27544060', '1570', '1579', 'alectinib', 'Chemical', 'MESH:C582670'],\n",
              " ['27544060', '1594', '1603', 'Alectinib', 'Chemical', 'MESH:C582670'],\n",
              " ['27544060', '1651', '1659', 'patients', 'Species', '9606'],\n",
              " ['27544060', '1674', '1677', 'RET', 'Gene', '5979'],\n",
              " ['27544060', '1689', '1694', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['27544060', '1728', '1731', 'RET', 'Gene', '5979'],\n",
              " ['27544060', '1830', '1839', 'alectinib', 'Chemical', 'MESH:C582670'],\n",
              " ['27544060', '1843', '1846', 'RET', 'Gene', '5979'],\n",
              " ['27544060', '1858', '1863', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['27544060', '1874', '1877', 'RET', 'Gene', '5979'],\n",
              " ['27544060', '1885', '1897', 'malignancies', 'Disease', 'MESH:D009369'],\n",
              " ['27544060', '1950', '1953', 'RET', 'Gene', '5979'],\n",
              " ['27694386', '9', '12', 'MET', 'Gene', '79811'],\n",
              " ['27694386', '46', '49', 'MET', 'Gene', '79811'],\n",
              " ['27694386', '64', '75', 'Lung Cancer', 'Disease', 'MESH:D008175'],\n",
              " ['27694386', '102', '105', 'MET', 'Gene', '79811'],\n",
              " ['27694386', '231', '242', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['27694386', '278', '281', 'MET', 'Gene', '79811'],\n",
              " ['27694386', '381', '384', 'MET', 'Gene', '79811'],\n",
              " ['27694386', '431', '438', 'patient', 'Species', '9606'],\n",
              " ['27694386',\n",
              "  '444',\n",
              "  '463',\n",
              "  'lung adenocarcinoma',\n",
              "  'Disease',\n",
              "  'MESH:D000077192'],\n",
              " ['27694386', '493', '497', 'EGFR', 'Gene', '1956'],\n",
              " ['27694386', '522', '525', 'MET', 'Gene', '79811'],\n",
              " ['27694386', '552', '561', 'erlotinib', 'Chemical', 'MESH:D000069347'],\n",
              " ['27694386', '597', '600', 'MET', 'Gene', '79811'],\n",
              " ['27694386', '605', '609', 'EGFR', 'Gene', '1956'],\n",
              " ['27694386', '626', '637', 'savolitinib', 'Chemical', 'MESH:C000593259'],\n",
              " ['27694386', '642', '653', 'osimertinib', 'Chemical', '-'],\n",
              " ['27694386', '708', '711', 'MET', 'Gene', '79811'],\n",
              " ['27694386', '797', '806', 'METD1228V', 'Chemical', '-'],\n",
              " ['27694386', '836', '839', 'MET', 'Gene', '79811'],\n",
              " ['27694386', '905', '908', 'MET', 'Gene', '79811'],\n",
              " ['27694386', '958', '965', 'patient', 'Species', '9606'],\n",
              " ['27694386', '983', '992', 'erlotinib', 'Chemical', 'MESH:D000069347'],\n",
              " ['27694386', '1007', '1019', 'cabozantinib', 'Chemical', 'MESH:C558660'],\n",
              " ['27694386', '1031', '1034', 'MET', 'Gene', '79811'],\n",
              " ['27694386', '1121', '1124', 'MET', 'Gene', '79811'],\n",
              " ['27694386', '1289', '1298', 'METD1228V', 'Chemical', '-'],\n",
              " ['27694386', '1356', '1359', 'MET', 'Gene', '79811'],\n",
              " ['27694386', '1439', '1442', 'MET', 'Gene', '79811'],\n",
              " ['27694386', '1455', '1461', 'Cancer', 'Disease', 'MESH:D009369'],\n",
              " ['27876675', '30', '40', 'Trametinib', 'Chemical', 'MESH:C560077'],\n",
              " ['27876675', '46', '55', 'Docetaxel', 'Chemical', 'MESH:D000077143'],\n",
              " ['27876675', '59', '69', 'Pemetrexed', 'Chemical', 'MESH:D000068437'],\n",
              " ['27876675', '73', '81', 'Patients', 'Species', '9606'],\n",
              " ['27876675',\n",
              "  '96',\n",
              "  '122',\n",
              "  'Non-Small Cell Lung Cancer',\n",
              "  'Disease',\n",
              "  'MESH:D002289'],\n",
              " ['27876675', '166', '176', 'trametinib', 'Chemical', 'MESH:C560077'],\n",
              " ['27876675', '180', '186', 'MEK1/2', 'Gene', '5604;5605'],\n",
              " ['27876675', '251', '254', 'MEK', 'Gene', '5609'],\n",
              " ['27876675', '281', '285', 'KRAS', 'Gene', '3845'],\n",
              " ['27876675', '341', '345', 'KRAS', 'Gene', '3845'],\n",
              " ['27876675', '353', '358', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['27876675', '432', '442', 'trametinib', 'Chemical', 'MESH:C560077'],\n",
              " ['27876675', '552', '562', 'trametinib', 'Chemical', 'MESH:C560077'],\n",
              " ['27876675', '579', '587', 'patients', 'Species', '9606'],\n",
              " ['27876675', '593', '598', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['27876675', '616', '620', 'KRAS', 'Gene', '3845'],\n",
              " ['27876675', '660', '670', 'trametinib', 'Chemical', 'MESH:C560077'],\n",
              " ['27876675', '676', '685', 'docetaxel', 'Chemical', 'MESH:D000077143'],\n",
              " ['27876675', '714', '724', 'trametinib', 'Chemical', 'MESH:C560077'],\n",
              " ['27876675', '749', '758', 'docetaxel', 'Chemical', 'MESH:D000077143'],\n",
              " ['27876675', '787', '797', 'pemetrexed', 'Chemical', 'MESH:D000068437'],\n",
              " ['27876675', '799', '809', 'trametinib', 'Chemical', 'MESH:C560077'],\n",
              " ['27876675', '834', '844', 'pemetrexed', 'Chemical', 'MESH:D000068437'],\n",
              " ['27876675', '939', '944', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['27876675', '976', '980', 'KRAS', 'Gene', '3845'],\n",
              " ['27876675', '1260', '1268', 'patients', 'Species', '9606'],\n",
              " ['27876675', '1274', '1279', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['27876675', '1293', '1303', 'trametinib', 'Chemical', 'MESH:C560077'],\n",
              " ['27876675', '1309', '1318', 'docetaxel', 'Chemical', 'MESH:D000077143'],\n",
              " ['27876675', '1358', '1366', 'patients', 'Species', '9606'],\n",
              " ['27876675', '1382', '1386', 'KRAS', 'Gene', '3845'],\n",
              " ['27876675', '1397', '1402', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['27876675', '1429', '1437', 'patients', 'Species', '9606'],\n",
              " ['27876675', '1453', '1457', 'KRAS', 'Gene', '3845'],\n",
              " ['27876675', '1465', '1470', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['27876675', '1482', '1490', 'patients', 'Species', '9606'],\n",
              " ['27876675', '1496', '1501', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['27876675', '1515', '1525', 'trametinib', 'Chemical', 'MESH:C560077'],\n",
              " ['27876675', '1531', '1541', 'pemetrexed', 'Chemical', 'MESH:D000068437'],\n",
              " ['27876675', '1595', '1603', 'patients', 'Species', '9606'],\n",
              " ['27876675', '1609', '1613', 'KRAS', 'Gene', '3845'],\n",
              " ['27876675', '1622', '1627', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['27876675', '1654', '1658', 'KRAS', 'Gene', '3845'],\n",
              " ['27876675', '1669', '1674', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['27876675', '1705', '1713', 'diarrhea', 'Disease', 'MESH:D003967'],\n",
              " ['27876675', '1715', '1721', 'nausea', 'Disease', 'MESH:D009325'],\n",
              " ['27876675', '1727', '1734', 'fatigue', 'Disease', 'MESH:D005221'],\n",
              " ['27876675', '1765', '1775', 'Trametinib', 'Chemical', 'MESH:C560077'],\n",
              " ['27876675', '1885', '1894', 'docetaxel', 'Chemical', 'MESH:D000077143'],\n",
              " ['27876675', '1898', '1908', 'pemetrexed', 'Chemical', 'MESH:D000068437'],\n",
              " ['27876675', '1918', '1922', 'KRAS', 'Gene', '3845'],\n",
              " ['27876675', '1931', '1936', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['28259530', '20', '25', 'PD-L1', 'Gene', '29126'],\n",
              " ['28259530', '63', '68', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['28259530', '151', '176', 'programmed death-ligand 1', 'Gene', '29126'],\n",
              " ['28259530', '178', '183', 'PD-L1', 'Gene', '29126'],\n",
              " ['28259530',\n",
              "  '222',\n",
              "  '248',\n",
              "  'non-small cell lung cancer',\n",
              "  'Disease',\n",
              "  'MESH:D002289'],\n",
              " ['28259530', '250', '255', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['28259530', '314', '317', 'Odd', 'Gene', '130497'],\n",
              " ['28259530', '390', '395', 'PD-L1', 'Gene', '29126'],\n",
              " ['28259530', '514', '518', 'KRAS', 'Gene', '3845'],\n",
              " ['28259530', '526', '532', 'tumors', 'Disease', 'MESH:D009369'],\n",
              " ['28259530', '556', '561', 'PD-L1', 'Gene', '29126'],\n",
              " ['28259530', '576', '580', 'KRAS', 'Gene', '3845'],\n",
              " ['28259530', '591', '597', 'tumors', 'Disease', 'MESH:D009369'],\n",
              " ['28259530', '663', '668', 'PD-L1', 'Gene', '29126'],\n",
              " ['28259530', '698', '702', 'EGFR', 'Gene', '1956'],\n",
              " ['28259530', '745', '748', 'ALK', 'Gene', '238'],\n",
              " ['28259530', '868', '873', 'PD-L1', 'Gene', '29126'],\n",
              " ['28259530', '889', '893', 'EGFR', 'Gene', '1956'],\n",
              " ['28259530', '966', '971', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['28259530', '977', '981', 'KRAS', 'Gene', '3845'],\n",
              " ['28259530', '1040', '1045', 'PD-L1', 'Gene', '29126'],\n",
              " ['28525386', '20', '24', 'KRAS', 'Gene', '3845'],\n",
              " ['28525386', '61', '72', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['28525386', '270', '281', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['28525386', '283', '288', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['28525386', '291', '295', 'KRAS', 'Gene', '3845'],\n",
              " ['28525386', '339', '344', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['28525386', '396', '400', 'KRAS', 'Gene', '3845'],\n",
              " ['28525386', '454', '462', 'patients', 'Species', '9606'],\n",
              " ['28525386', '477', '482', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['28525386', '619', '623', 'KRAS', 'Gene', '3845'],\n",
              " ['28525386', '670', '678', 'patients', 'Species', '9606'],\n",
              " ['28525386', '684', '688', 'KRAS', 'Gene', '3845'],\n",
              " ['28525386', '696', '701', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['28525386', '715', '719', 'KRAS', 'Gene', '3845'],\n",
              " ['28525386', '730', '735', 'tumor', 'Disease', 'MESH:D009369'],\n",
              " ['28525386', '802', '811', 'docetaxel', 'Chemical', 'MESH:D000077143'],\n",
              " ['28525386', '847', '855', 'patients', 'Species', '9606'],\n",
              " ['28525386', '880', '884', 'KRAS', 'Gene', '3845'],\n",
              " ['28525386', '892', '897', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['28525386', '972', '980', 'patients', 'Species', '9606'],\n",
              " ['28525386', '986', '990', 'KRAS', 'Gene', '3845'],\n",
              " ['28525386', '1001', '1006', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['28525386', '1235', '1244', 'docetaxel', 'Chemical', 'MESH:D000077143'],\n",
              " ['28525386', '1257', '1262', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['28525386', '1263', '1271', 'patients', 'Species', '9606'],\n",
              " ['28525386', '1277', '1281', 'KRAS', 'Gene', '3845'],\n",
              " ['28525386', '1314', '1318', 'KRAS', 'Gene', '3845'],\n",
              " ['28525386', '1329', '1334', 'tumor', 'Disease', 'MESH:D009369'],\n",
              " ['28525386', '1363', '1367', 'KRAS', 'Gene', '3845'],\n",
              " ['28602779', '0', '9', 'Ceritinib', 'Chemical', 'MESH:C586847'],\n",
              " ['28602779', '33', '41', 'patients', 'Species', '9606'],\n",
              " ['28602779', '47', '50', 'ALK', 'Gene', '238'],\n",
              " ['28602779', '77', '88', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['28602779', '123', '133', 'crizotinib', 'Chemical', 'MESH:D000077547'],\n",
              " ['28602779', '211', '220', 'Ceritinib', 'Chemical', 'MESH:C586847'],\n",
              " ['28602779', '242', '268', 'anaplastic lymphoma kinase', 'Gene', '238'],\n",
              " ['28602779', '270', '273', 'ALK', 'Gene', '238'],\n",
              " ['28602779', '314', '320', 'tumour', 'Disease', 'MESH:D009369'],\n",
              " ['28602779', '368', '376', 'patients', 'Species', '9606'],\n",
              " ['28602779', '382', '385', 'ALK', 'Gene', '238'],\n",
              " ['28602779', '412', '423', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['28602779', '451', '460', 'ceritinib', 'Chemical', 'MESH:C586847'],\n",
              " ['28602779', '504', '507', 'ALK', 'Gene', '238'],\n",
              " ['28602779', '528', '531', 'ALK', 'Gene', '238'],\n",
              " ['28602779', '553', '561', 'patients', 'Species', '9606'],\n",
              " ['28602779', '679', '688', 'ceritinib', 'Chemical', 'MESH:C586847'],\n",
              " ['28602779', '725', '733', 'patients', 'Species', '9606'],\n",
              " ['28602779', '748', '751', 'ALK', 'Gene', '238'],\n",
              " ['28602779', '778', '789', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['28602779', '830', '840', 'crizotinib', 'Chemical', 'MESH:D000077547'],\n",
              " ['28602779', '845', '853', 'platinum', 'Chemical', 'MESH:D010984'],\n",
              " ['28602779', '963', '971', 'patients', 'Species', '9606'],\n",
              " ['28602779', '1000', '1003', 'ALK', 'Gene', '238'],\n",
              " ['28602779',\n",
              "  '1032',\n",
              "  '1058',\n",
              "  'non-small-cell lung cancer',\n",
              "  'Disease',\n",
              "  'MESH:D002289'],\n",
              " ['28602779', '1167', '1175', 'platinum', 'Chemical', 'MESH:D010984'],\n",
              " ['28602779', '1189', '1199', 'crizotinib', 'Chemical', 'MESH:D000077547'],\n",
              " ['28602779', '1497', '1507', 'toxicities', 'Disease', 'MESH:D064420'],\n",
              " ['28602779', '1531', '1539', 'patients', 'Species', '9606'],\n",
              " ['28602779', '1663', '1673', 'metastases', 'Disease', 'MESH:D009362'],\n",
              " ['28602779', '1683', '1692', 'ceritinib', 'Chemical', 'MESH:C586847'],\n",
              " ['28602779', '1773', '1783', 'pemetrexed', 'Chemical', 'MESH:D000068437'],\n",
              " ['28602779', '1797', '1806', 'docetaxel', 'Chemical', 'MESH:D000077143'],\n",
              " ['28602779', '1855', '1863', 'Patients', 'Species', '9606'],\n",
              " ['28602779',\n",
              "  '1905',\n",
              "  '1924',\n",
              "  'progressive disease',\n",
              "  'Disease',\n",
              "  'MESH:D018450'],\n",
              " ['28602779', '1949', '1958', 'ceritinib', 'Chemical', 'MESH:C586847'],\n",
              " ['28602779', '2339', '2347', 'patients', 'Species', '9606'],\n",
              " ['28602779', '2386', '2391', 'Nov 2', 'Gene', '1490'],\n",
              " ['28602779', '2425', '2433', 'patients', 'Species', '9606'],\n",
              " ['28602779', '2448', '2457', 'ceritinib', 'Chemical', 'MESH:C586847'],\n",
              " ['28602779', '2501', '2511', 'pemetrexed', 'Chemical', 'MESH:D000068437'],\n",
              " ['28602779', '2525', '2534', 'docetaxel', 'Chemical', 'MESH:D000077143'],\n",
              " ['28602779', '2643', '2652', 'Ceritinib', 'Chemical', 'MESH:C586847'],\n",
              " ['28602779', '2782', '2791', 'ceritinib', 'Chemical', 'MESH:C586847'],\n",
              " ['28602779', '2932', '2940', 'patients', 'Species', '9606'],\n",
              " ['28602779', '2948', '2957', 'ceritinib', 'Chemical', 'MESH:C586847'],\n",
              " ['28602779', '3097', '3106', 'ceritinib', 'Chemical', 'MESH:C586847'],\n",
              " ['28602779', '3203', '3212', 'ceritinib', 'Chemical', 'MESH:C586847'],\n",
              " ['28602779', '3532', '3540', 'patients', 'Species', '9606'],\n",
              " ['28602779', '3548', '3557', 'ceritinib', 'Chemical', 'MESH:C586847'],\n",
              " ['28602779', '3678', '3686', 'patients', 'Species', '9606'],\n",
              " ['28602779', '3694', '3703', 'ceritinib', 'Chemical', 'MESH:C586847'],\n",
              " ['28602779', '3757', '3761', 'died', 'Disease', 'MESH:D003643'],\n",
              " ['28602779', '3894', '3902', 'patients', 'Species', '9606'],\n",
              " ['28602779', '3907', '3911', 'died', 'Disease', 'MESH:D003643'],\n",
              " ['28602779', '3919', '3928', 'ceritinib', 'Chemical', 'MESH:C586847'],\n",
              " ['28602779', '3935', '3939', 'died', 'Disease', 'MESH:D003643'],\n",
              " ['28602779', '3985', '3989', 'died', 'Disease', 'MESH:D003643'],\n",
              " ['28602779',\n",
              "  '4028',\n",
              "  '4052',\n",
              "  'cerebrovascular accident',\n",
              "  'Disease',\n",
              "  'MESH:D020521'],\n",
              " ['28602779',\n",
              "  '4066',\n",
              "  '4085',\n",
              "  'respiratory failure',\n",
              "  'Disease',\n",
              "  'MESH:D012131'],\n",
              " ['28602779', '4105', '4111', 'deaths', 'Disease', 'MESH:D003643'],\n",
              " ['28602779', '4187', '4193', 'deaths', 'Disease', 'MESH:D003643'],\n",
              " ['28602779', '4298', '4306', 'patients', 'Species', '9606'],\n",
              " ['28602779', '4362', '4365', 'ALK', 'Gene', '238'],\n",
              " ['28602779', '4393', '4403', 'crizotinib', 'Chemical', 'MESH:D000077547'],\n",
              " ['28602779', '4419', '4428', 'ceritinib', 'Chemical', 'MESH:C586847'],\n",
              " ['28602779', '4503', '4510', 'patient', 'Species', '9606'],\n",
              " ['30413378', '0', '10', 'Lorlatinib', 'Chemical', 'MESH:C000590786'],\n",
              " ['30413378', '14', '22', 'patients', 'Species', '9606'],\n",
              " ['30413378', '28', '31', 'ALK', 'Gene', '238'],\n",
              " ['30413378', '56', '67', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['30413378', '118', '128', 'Lorlatinib', 'Chemical', 'MESH:C000590786'],\n",
              " ['30413378', '189', '192', 'ALK', 'Gene', '238'],\n",
              " ['30413378', '197', '201', 'ROS1', 'Gene', '6098'],\n",
              " ['30413378', '242', '245', 'ALK', 'Gene', '238'],\n",
              " ['30413378', '298', '306', 'patients', 'Species', '9606'],\n",
              " ['30413378', '312', '315', 'ALK', 'Gene', '238'],\n",
              " ['30413378', '340', '351', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['30413378', '374', '384', 'metastases', 'Disease', 'MESH:D009362'],\n",
              " ['30413378', '407', '410', 'ALK', 'Gene', '238'],\n",
              " ['30413378', '501', '511', 'lorlatinib', 'Chemical', 'MESH:C000590786'],\n",
              " ['30413378', '515', '523', 'patients', 'Species', '9606'],\n",
              " ['30413378', '529', '532', 'ALK', 'Gene', '238'],\n",
              " ['30413378', '567', '578', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['30413378', '612', '620', 'patients', 'Species', '9606'],\n",
              " ['30413378', '658', '661', 'ALK', 'Gene', '238'],\n",
              " ['30413378', '674', '678', 'ROS1', 'Gene', '6098'],\n",
              " ['30413378', '714', '725', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['30413378', '747', '757', 'metastases', 'Disease', 'MESH:D009362'],\n",
              " ['30413378', '883', '891', 'Patients', 'Species', '9606'],\n",
              " ['30413378', '944', '950', 'EXP1-6', 'Gene', '7514;1434;64328;57510;23214'],\n",
              " ['30413378', '968', '971', 'ALK', 'Gene', '238'],\n",
              " ['30413378', '976', '980', 'ROS1', 'Gene', '6098'],\n",
              " ['30413378', '1025', '1035', 'lorlatinib', 'Chemical', 'MESH:C000590786'],\n",
              " ['30413378',\n",
              "  '1129',\n",
              "  '1148',\n",
              "  'intracranial tumour',\n",
              "  'Disease',\n",
              "  'MESH:D001932'],\n",
              " ['30413378', '1221', '1224', 'ALK', 'Gene', '238'],\n",
              " ['30413378', '1234', '1242', 'patients', 'Species', '9606'],\n",
              " ['30413378', '1323', '1331', 'patients', 'Species', '9606'],\n",
              " ['30413378', '1366', '1376', 'lorlatinib', 'Chemical', 'MESH:C000590786'],\n",
              " ['30413378', '1421', '1429', 'Patients', 'Species', '9606'],\n",
              " ['30413378', '1450', '1460', 'metastases', 'Disease', 'MESH:D009362'],\n",
              " ['30413378', '1583', '1593', 'lorlatinib', 'Chemical', 'MESH:C000590786'],\n",
              " ['30413378', '1616', '1619', 'ALK', 'Gene', '238'],\n",
              " ['30413378', '1629', '1637', 'patients', 'Species', '9606'],\n",
              " ['30413378', '1639', '1645', 'EXP1-5', 'Gene', '7514;1434;64328;57510'],\n",
              " ['30413378', '1685', '1693', 'patients', 'Species', '9606'],\n",
              " ['30413378', '1695', '1701', 'EXP1-6', 'Gene', '7514;1434;64328;57510;23214'],\n",
              " ['30413378', '1843', '1851', 'patients', 'Species', '9606'],\n",
              " ['30413378', '1879', '1882', 'ALK', 'Gene', '238'],\n",
              " ['30413378', '1913', '1917', 'EXP1', 'Gene', '7514'],\n",
              " ['30413378', '1932', '1935', 'ALK', 'Gene', '238'],\n",
              " ['30413378', '1993', '1997', 'EXP2', 'Gene', '1434'],\n",
              " ['30413378', '2056', '2059', 'ALK', 'Gene', '238'],\n",
              " ['30413378', '2192', '2195', 'ALK', 'Gene', '238'],\n",
              " ['30413378', '2221', '2225', 'EXP4', 'Gene', '64328'],\n",
              " ['30413378', '2243', '2247', 'EXP5', 'Gene', '57510'],\n",
              " ['30413378', '2258', '2261', 'ALK', 'Gene', '238'],\n",
              " ['30413378', '2335', '2339', 'ROS1', 'Gene', '6098'],\n",
              " ['30413378', '2378', '2382', 'EXP6', 'Gene', '23214'],\n",
              " ['30413378', '2389', '2396', 'patient', 'Species', '9606'],\n",
              " ['30413378', '2400', '2404', 'EXP4', 'Gene', '64328'],\n",
              " ['30413378', '2427', '2437', 'lorlatinib', 'Chemical', 'MESH:C000590786'],\n",
              " ['30413378', '2504', '2512', 'patients', 'Species', '9606'],\n",
              " ['30413378', '2514', '2518', 'EXP1', 'Gene', '7514'],\n",
              " ['30413378', '2594', '2602', 'patients', 'Species', '9606'],\n",
              " ['30413378', '2610', '2618', 'patients', 'Species', '9606'],\n",
              " ['30413378', '2622', '2626', 'EXP1', 'Gene', '7514'],\n",
              " ['30413378', '2782', '2785', 'ALK', 'Gene', '238'],\n",
              " ['30413378', '2795', '2803', 'patients', 'Species', '9606'],\n",
              " ['30413378', '2862', '2868', 'EXP2-5', 'Gene', '1434;64328;57510'],\n",
              " ['30413378', '2937', '2945', 'patients', 'Species', '9606'],\n",
              " ['30413378', '3059', '3067', 'patients', 'Species', '9606'],\n",
              " ['30413378', '3139', '3147', 'patients', 'Species', '9606'],\n",
              " ['30413378', '3179', '3189', 'crizotinib', 'Chemical', 'MESH:D000077547'],\n",
              " ['30413378', '3231', '3239', 'patients', 'Species', '9606'],\n",
              " ['30413378', '3262', '3272', 'crizotinib', 'Chemical', 'MESH:D000077547'],\n",
              " ['30413378', '3345', '3353', 'patients', 'Species', '9606'],\n",
              " ['30413378', '3412', '3418', 'EXP4-5', 'Gene', '64328;57510'],\n",
              " ['30413378', '3504', '3512', 'patients', 'Species', '9606'],\n",
              " ['30413378', '3553', '3557', 'EXP2', 'Gene', '1434'],\n",
              " ['30413378', '3594', '3602', 'patients', 'Species', '9606'],\n",
              " ['30413378', '3645', '3653', 'patients', 'Species', '9606'],\n",
              " ['30413378', '3657', '3663', 'EXP4-5', 'Gene', '64328;57510'],\n",
              " ['30413378', '3725', '3733', 'patients', 'Species', '9606'],\n",
              " ['30413378', '3739', '3760', 'hypercholesterolaemia', 'Disease', ''],\n",
              " ['30413378', '3779', '3787', 'patients', 'Species', '9606'],\n",
              " ['30413378', '3824', '3845', 'hypertriglyceridaemia', 'Disease', ''],\n",
              " ['30413378', '3958', '3966', 'patients', 'Species', '9606'],\n",
              " ['30413378', '3977', '3985', 'patients', 'Species', '9606'],\n",
              " ['30413378', '4092', '4098', 'deaths', 'Disease', 'MESH:D003643'],\n",
              " ['30413378', '4156', '4159', 'ALK', 'Gene', '238'],\n",
              " ['30413378', '4201', '4211', 'lorlatinib', 'Chemical', 'MESH:C000590786'],\n",
              " ['30413378', '4289', '4297', 'patients', 'Species', '9606'],\n",
              " ['30413378', '4303', '4306', 'ALK', 'Gene', '238'],\n",
              " ['30413378', '4331', '4342', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['30413378', '4379', '4389', 'crizotinib', 'Chemical', 'MESH:D000077547'],\n",
              " ['30413378', '4471', '4474', 'ALK', 'Gene', '238'],\n",
              " ['30413378', '4509', '4519', 'lorlatinib', 'Chemical', 'MESH:C000590786'],\n",
              " ['30413378', '4570', '4578', 'patients', 'Species', '9606'],\n",
              " ['30413378', '4584', '4587', 'ALK', 'Gene', '238'],\n",
              " ['30413378', '4612', '4623', 'lung cancer', 'Disease', 'MESH:D008175'],\n",
              " ['34534430', '0', '11', 'Trastuzumab', 'Chemical', 'MESH:D000068878'],\n",
              " ['34534430', '12', '22', 'Deruxtecan', 'Chemical', '-'],\n",
              " ['34534430', '26', '30', 'HER2', 'Gene', '2064'],\n",
              " ['34534430',\n",
              "  '38',\n",
              "  '64',\n",
              "  'Non-Small-Cell Lung Cancer',\n",
              "  'Disease',\n",
              "  'MESH:D002289'],\n",
              " ['34534430',\n",
              "  '78',\n",
              "  '118',\n",
              "  'Human epidermal growth factor receptor 2',\n",
              "  'Gene',\n",
              "  '2064'],\n",
              " ['34534430', '120', '124', 'HER2', 'Gene', '2064'],\n",
              " ['34534430', '172', '180', 'patients', 'Species', '9606'],\n",
              " ['34534430',\n",
              "  '186',\n",
              "  '212',\n",
              "  'non-small-cell lung cancer',\n",
              "  'Disease',\n",
              "  'MESH:D002289'],\n",
              " ['34534430', '214', '219', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['34534430', '249', '260', 'trastuzumab', 'Chemical', 'MESH:D000068878'],\n",
              " ['34534430', '261', '271', 'deruxtecan', 'Chemical', '-'],\n",
              " ['34534430', '282', '289', 'DS-8201', 'Chemical', 'MESH:C000614160'],\n",
              " ['34534430', '294', '298', 'HER2', 'Gene', '2064'],\n",
              " ['34534430', '327', '335', 'patients', 'Species', '9606'],\n",
              " ['34534430', '341', '345', 'HER2', 'Gene', '2064'],\n",
              " ['34534430', '353', '358', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['34534430', '474', '485', 'trastuzumab', 'Chemical', 'MESH:D000068878'],\n",
              " ['34534430', '486', '496', 'deruxtecan', 'Chemical', '-'],\n",
              " ['34534430', '554', '562', 'patients', 'Species', '9606'],\n",
              " ['34534430', '582', '586', 'HER2', 'Gene', '2064'],\n",
              " ['34534430', '594', '599', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['34534430', '854', '858', 'HER2', 'Gene', '2064'],\n",
              " ['34534430', '909', '917', 'patients', 'Species', '9606'],\n",
              " ['34534430', '1066', '1074', 'patients', 'Species', '9606'],\n",
              " ['34534430', '1469', '1477', 'patients', 'Species', '9606'],\n",
              " ['34534430', '1507', '1518', 'neutropenia', 'Disease', 'MESH:D009503'],\n",
              " ['34534430', '1567', '1579', 'lung disease', 'Disease', 'MESH:D008171'],\n",
              " ['34534430', '1599', '1607', 'patients', 'Species', '9606'],\n",
              " ['34534430', '1624', '1629', 'death', 'Disease', 'MESH:D003643'],\n",
              " ['34534430', '1635', '1643', 'patients', 'Species', '9606'],\n",
              " ['34534430', '1686', '1690', 'HER2', 'Gene', '2064'],\n",
              " ['34534430', '1724', '1732', 'patients', 'Species', '9606'],\n",
              " ['34534430', '1752', '1756', 'HER2', 'Gene', '2064'],\n",
              " ['34534430', '1771', '1775', 'HER2', 'Gene', '2064'],\n",
              " ['34534430', '1804', '1815', 'Trastuzumab', 'Chemical', 'MESH:D000068878'],\n",
              " ['34534430', '1816', '1826', 'deruxtecan', 'Chemical', '-'],\n",
              " ['34534430', '1865', '1873', 'patients', 'Species', '9606'],\n",
              " ['34534430', '1898', '1902', 'HER2', 'Gene', '2064'],\n",
              " ['34534430', '1910', '1915', 'NSCLC', 'Disease', 'MESH:D002289'],\n",
              " ['34534430', '1958', '1970', 'lung disease', 'Disease', 'MESH:D008171']]"
            ]
          },
          "metadata": {},
          "execution_count": 43
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "cols = [\"pmid\", \"start_offset\", \"end_offset\", \"entity_mention\", \"entity_type\", \"entity_id\"]"
      ],
      "metadata": {
        "id": "aR3vv6hGPcyp"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "entity_df = pd.DataFrame(pubtator_entities, columns=cols)"
      ],
      "metadata": {
        "id": "1kp353xHPe1v"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "entity_df.head()"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 247
        },
        "id": "7cCTu27RPgaK",
        "outputId": "10e8d353-ff70-4c9a-ac02-76d2522538a8"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "       pmid start_offset end_offset entity_mention entity_type     entity_id\n",
              "0  25765070            0          6         Cancer     Disease  MESH:D009369\n",
              "1  25765070          157        164        patient     Species          9606\n",
              "2  25765070          187        193         tumors     Disease  MESH:D009369\n",
              "3  25765070          347        359   lung cancers     Disease  MESH:D008175\n",
              "4  25765070          373        386  pembrolizumab    Chemical  MESH:C582435"
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-8df4199e-afe9-4336-8b1b-3baed27d7c77\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>pmid</th>\n",
              "      <th>start_offset</th>\n",
              "      <th>end_offset</th>\n",
              "      <th>entity_mention</th>\n",
              "      <th>entity_type</th>\n",
              "      <th>entity_id</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>25765070</td>\n",
              "      <td>0</td>\n",
              "      <td>6</td>\n",
              "      <td>Cancer</td>\n",
              "      <td>Disease</td>\n",
              "      <td>MESH:D009369</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>25765070</td>\n",
              "      <td>157</td>\n",
              "      <td>164</td>\n",
              "      <td>patient</td>\n",
              "      <td>Species</td>\n",
              "      <td>9606</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>25765070</td>\n",
              "      <td>187</td>\n",
              "      <td>193</td>\n",
              "      <td>tumors</td>\n",
              "      <td>Disease</td>\n",
              "      <td>MESH:D009369</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>25765070</td>\n",
              "      <td>347</td>\n",
              "      <td>359</td>\n",
              "      <td>lung cancers</td>\n",
              "      <td>Disease</td>\n",
              "      <td>MESH:D008175</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>25765070</td>\n",
              "      <td>373</td>\n",
              "      <td>386</td>\n",
              "      <td>pembrolizumab</td>\n",
              "      <td>Chemical</td>\n",
              "      <td>MESH:C582435</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-8df4199e-afe9-4336-8b1b-3baed27d7c77')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-8df4199e-afe9-4336-8b1b-3baed27d7c77 button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-8df4199e-afe9-4336-8b1b-3baed27d7c77');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-ff8895bb-c5ae-40dd-ac36-676ed487e36a\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-ff8895bb-c5ae-40dd-ac36-676ed487e36a')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-ff8895bb-c5ae-40dd-ac36-676ed487e36a button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "variable_name": "entity_df",
              "summary": "{\n  \"name\": \"entity_df\",\n  \"rows\": 916,\n  \"fields\": [\n    {\n      \"column\": \"pmid\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 24,\n        \"samples\": [\n          \"26554404\",\n          \"27544060\",\n          \"25765070\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"start_offset\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 772,\n        \"samples\": [\n          \"714\",\n          \"915\",\n          \"1804\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"end_offset\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 754,\n        \"samples\": [\n          \"811\",\n          \"1795\",\n          \"245\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"entity_mention\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 185,\n        \"samples\": [\n          \"NSCLC\",\n          \"erlotinib\",\n          \"progressive disease\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"entity_type\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 6,\n        \"samples\": [\n          \"Disease\",\n          \"Species\",\n          \"ProteinMutation\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"entity_id\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 119,\n        \"samples\": [\n          \"MESH:D000077547\",\n          \"5979\",\n          \"T cell\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"
            }
          },
          "metadata": {},
          "execution_count": 46
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "entity_df.shape"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "yJOvVa_EPqck",
        "outputId": "69e6324f-29af-4de7-c0df-a93cb7afbc6d"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "(916, 6)"
            ]
          },
          "metadata": {},
          "execution_count": 47
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# 4) What are the top five diseases (by count) detected by Pubtator in your list of 25 pmids? Based on these top five diseases, what disease topic or query do you think was used to obtain the initial list of pmids."
      ],
      "metadata": {
        "id": "3Ko4edxNFoiE"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "disease=entity_df[entity_df['entity_type']=='Disease']\n",
        "disease"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 424
        },
        "id": "FvpWEKixFTd4",
        "outputId": "b2ac5072-e547-42d9-8fc6-9cc9ad2f9b9b"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "         pmid start_offset end_offset entity_mention entity_type     entity_id\n",
              "0    25765070            0          6         Cancer     Disease  MESH:D009369\n",
              "2    25765070          187        193         tumors     Disease  MESH:D009369\n",
              "3    25765070          347        359   lung cancers     Disease  MESH:D008175\n",
              "5    25765070          510        516         tumors     Disease  MESH:D009369\n",
              "7    25765070          871        876          tumor     Disease  MESH:D009369\n",
              "..        ...          ...        ...            ...         ...           ...\n",
              "901  34534430         1507       1518    neutropenia     Disease  MESH:D009503\n",
              "902  34534430         1567       1579   lung disease     Disease  MESH:D008171\n",
              "904  34534430         1624       1629          death     Disease  MESH:D003643\n",
              "914  34534430         1910       1915          NSCLC     Disease  MESH:D002289\n",
              "915  34534430         1958       1970   lung disease     Disease  MESH:D008171\n",
              "\n",
              "[222 rows x 6 columns]"
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-83de7930-2340-4a01-b905-1eaa47ab6380\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>pmid</th>\n",
              "      <th>start_offset</th>\n",
              "      <th>end_offset</th>\n",
              "      <th>entity_mention</th>\n",
              "      <th>entity_type</th>\n",
              "      <th>entity_id</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>25765070</td>\n",
              "      <td>0</td>\n",
              "      <td>6</td>\n",
              "      <td>Cancer</td>\n",
              "      <td>Disease</td>\n",
              "      <td>MESH:D009369</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>25765070</td>\n",
              "      <td>187</td>\n",
              "      <td>193</td>\n",
              "      <td>tumors</td>\n",
              "      <td>Disease</td>\n",
              "      <td>MESH:D009369</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>25765070</td>\n",
              "      <td>347</td>\n",
              "      <td>359</td>\n",
              "      <td>lung cancers</td>\n",
              "      <td>Disease</td>\n",
              "      <td>MESH:D008175</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>5</th>\n",
              "      <td>25765070</td>\n",
              "      <td>510</td>\n",
              "      <td>516</td>\n",
              "      <td>tumors</td>\n",
              "      <td>Disease</td>\n",
              "      <td>MESH:D009369</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>7</th>\n",
              "      <td>25765070</td>\n",
              "      <td>871</td>\n",
              "      <td>876</td>\n",
              "      <td>tumor</td>\n",
              "      <td>Disease</td>\n",
              "      <td>MESH:D009369</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>...</th>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>901</th>\n",
              "      <td>34534430</td>\n",
              "      <td>1507</td>\n",
              "      <td>1518</td>\n",
              "      <td>neutropenia</td>\n",
              "      <td>Disease</td>\n",
              "      <td>MESH:D009503</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>902</th>\n",
              "      <td>34534430</td>\n",
              "      <td>1567</td>\n",
              "      <td>1579</td>\n",
              "      <td>lung disease</td>\n",
              "      <td>Disease</td>\n",
              "      <td>MESH:D008171</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>904</th>\n",
              "      <td>34534430</td>\n",
              "      <td>1624</td>\n",
              "      <td>1629</td>\n",
              "      <td>death</td>\n",
              "      <td>Disease</td>\n",
              "      <td>MESH:D003643</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>914</th>\n",
              "      <td>34534430</td>\n",
              "      <td>1910</td>\n",
              "      <td>1915</td>\n",
              "      <td>NSCLC</td>\n",
              "      <td>Disease</td>\n",
              "      <td>MESH:D002289</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>915</th>\n",
              "      <td>34534430</td>\n",
              "      <td>1958</td>\n",
              "      <td>1970</td>\n",
              "      <td>lung disease</td>\n",
              "      <td>Disease</td>\n",
              "      <td>MESH:D008171</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>222 rows Ã— 6 columns</p>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-83de7930-2340-4a01-b905-1eaa47ab6380')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-83de7930-2340-4a01-b905-1eaa47ab6380 button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-83de7930-2340-4a01-b905-1eaa47ab6380');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-0771d93d-ce56-4adf-b275-05e5bff77e0f\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-0771d93d-ce56-4adf-b275-05e5bff77e0f')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-0771d93d-ce56-4adf-b275-05e5bff77e0f button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "variable_name": "disease",
              "summary": "{\n  \"name\": \"disease\",\n  \"rows\": 222,\n  \"fields\": [\n    {\n      \"column\": \"pmid\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 24,\n        \"samples\": [\n          \"26554404\",\n          \"27544060\",\n          \"25765070\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"start_offset\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 212,\n        \"samples\": [\n          \"167\",\n          \"892\",\n          \"1689\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"end_offset\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 211,\n        \"samples\": [\n          \"192\",\n          \"1334\",\n          \"1863\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"entity_mention\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 68,\n        \"samples\": [\n          \"squamous cell cancer\",\n          \"colorectal cancer\",\n          \"lung cancer\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"entity_type\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 1,\n        \"samples\": [\n          \"Disease\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"entity_id\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 36,\n        \"samples\": [\n          \"MESH:D008171\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"
            }
          },
          "metadata": {},
          "execution_count": 48
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "disease_counts = disease['entity_mention'].value_counts(normalize=False).sort_values(ascending=False)"
      ],
      "metadata": {
        "id": "x6QUsXN8Qc1E"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "print(\"Unique Values and Counts (Descending):\")\n",
        "print(disease_counts)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "-LXLSuzUQx6V",
        "outputId": "53d4c41b-c1d7-4193-b9f5-5e3301a9bb9b"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Unique Values and Counts (Descending):\n",
            "entity_mention\n",
            "NSCLC                                              54\n",
            "lung cancer                                        31\n",
            "tumor                                              10\n",
            "tumors                                             10\n",
            "SCC                                                 8\n",
            "                                                   ..\n",
            "Non-Small-Cell Lung Cancer                          1\n",
            "neutropenia                                         1\n",
            "squamous cell cancer                                1\n",
            "Lung Adenocarcinoma and Squamous Cell Carcinoma     1\n",
            "death                                               1\n",
            "Name: count, Length: 68, dtype: int64\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "**- Top 5 diseases are:** <br>\n",
        "1) NSCLC (Non-Small Cell Lung Cancer) <br>\n",
        "2) Lung cancer <br>\n",
        "3) Tumor <br>\n",
        "4) Tumors <br>\n",
        "5) SCC (Squamous Cell Carcinoma) <br>\n",
        "<br>\n",
        "\n",
        "**Based on the analysis results, it can be assumed that the list is comprised of articles related to cancer with high portion related to lung cancer.**\n",
        "\n"
      ],
      "metadata": {
        "id": "nZcJMC-wQ04u"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "# 5) What are the top five genes (by count) detected by Pubtator in your list of 25 pmids?"
      ],
      "metadata": {
        "id": "2ninFXEuRpEs"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "unique_entity_types = entity_df['entity_type'].unique()\n",
        "print(unique_entity_types)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "oa86iOBCQ4V2",
        "outputId": "5709d203-a927-4b41-f455-22add0fef42e"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "['Disease' 'Species' 'Chemical' 'CellLine' 'Gene' 'ProteinMutation']\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "gene=entity_df[entity_df['entity_type']=='Gene']\n",
        "gene"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 424
        },
        "id": "G0Nuv9i8R0mK",
        "outputId": "391ce91a-dcc2-4682-e8e3-9ac25af29194"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "         pmid start_offset end_offset           entity_mention entity_type  \\\n",
              "12   25891174          114        137  programmed cell death 1        Gene   \n",
              "13   25891174          139        143                     PD-1        Gene   \n",
              "17   25891174          322        335            PD-1 ligand 1        Gene   \n",
              "18   25891174          337        342                    PD-L1        Gene   \n",
              "23   25891174          668        673                    PD-L1        Gene   \n",
              "..        ...          ...        ...                      ...         ...   \n",
              "897  34534430          854        858                     HER2        Gene   \n",
              "906  34534430         1686       1690                     HER2        Gene   \n",
              "908  34534430         1752       1756                     HER2        Gene   \n",
              "909  34534430         1771       1775                     HER2        Gene   \n",
              "913  34534430         1898       1902                     HER2        Gene   \n",
              "\n",
              "    entity_id  \n",
              "12       5133  \n",
              "13       5133  \n",
              "17      29126  \n",
              "18      29126  \n",
              "23      29126  \n",
              "..        ...  \n",
              "897      2064  \n",
              "906      2064  \n",
              "908      2064  \n",
              "909      2064  \n",
              "913      2064  \n",
              "\n",
              "[325 rows x 6 columns]"
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-1fde8d7a-3e86-4b1e-a502-5c99a5c015ee\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>pmid</th>\n",
              "      <th>start_offset</th>\n",
              "      <th>end_offset</th>\n",
              "      <th>entity_mention</th>\n",
              "      <th>entity_type</th>\n",
              "      <th>entity_id</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>12</th>\n",
              "      <td>25891174</td>\n",
              "      <td>114</td>\n",
              "      <td>137</td>\n",
              "      <td>programmed cell death 1</td>\n",
              "      <td>Gene</td>\n",
              "      <td>5133</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>13</th>\n",
              "      <td>25891174</td>\n",
              "      <td>139</td>\n",
              "      <td>143</td>\n",
              "      <td>PD-1</td>\n",
              "      <td>Gene</td>\n",
              "      <td>5133</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>17</th>\n",
              "      <td>25891174</td>\n",
              "      <td>322</td>\n",
              "      <td>335</td>\n",
              "      <td>PD-1 ligand 1</td>\n",
              "      <td>Gene</td>\n",
              "      <td>29126</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>18</th>\n",
              "      <td>25891174</td>\n",
              "      <td>337</td>\n",
              "      <td>342</td>\n",
              "      <td>PD-L1</td>\n",
              "      <td>Gene</td>\n",
              "      <td>29126</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>23</th>\n",
              "      <td>25891174</td>\n",
              "      <td>668</td>\n",
              "      <td>673</td>\n",
              "      <td>PD-L1</td>\n",
              "      <td>Gene</td>\n",
              "      <td>29126</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>...</th>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>897</th>\n",
              "      <td>34534430</td>\n",
              "      <td>854</td>\n",
              "      <td>858</td>\n",
              "      <td>HER2</td>\n",
              "      <td>Gene</td>\n",
              "      <td>2064</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>906</th>\n",
              "      <td>34534430</td>\n",
              "      <td>1686</td>\n",
              "      <td>1690</td>\n",
              "      <td>HER2</td>\n",
              "      <td>Gene</td>\n",
              "      <td>2064</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>908</th>\n",
              "      <td>34534430</td>\n",
              "      <td>1752</td>\n",
              "      <td>1756</td>\n",
              "      <td>HER2</td>\n",
              "      <td>Gene</td>\n",
              "      <td>2064</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>909</th>\n",
              "      <td>34534430</td>\n",
              "      <td>1771</td>\n",
              "      <td>1775</td>\n",
              "      <td>HER2</td>\n",
              "      <td>Gene</td>\n",
              "      <td>2064</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>913</th>\n",
              "      <td>34534430</td>\n",
              "      <td>1898</td>\n",
              "      <td>1902</td>\n",
              "      <td>HER2</td>\n",
              "      <td>Gene</td>\n",
              "      <td>2064</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>325 rows Ã— 6 columns</p>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-1fde8d7a-3e86-4b1e-a502-5c99a5c015ee')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-1fde8d7a-3e86-4b1e-a502-5c99a5c015ee button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-1fde8d7a-3e86-4b1e-a502-5c99a5c015ee');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-95963234-5730-4aad-ad96-e80c7e3ac6f8\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-95963234-5730-4aad-ad96-e80c7e3ac6f8')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-95963234-5730-4aad-ad96-e80c7e3ac6f8 button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "variable_name": "gene",
              "summary": "{\n  \"name\": \"gene\",\n  \"rows\": 325,\n  \"fields\": [\n    {\n      \"column\": \"pmid\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 23,\n        \"samples\": [\n          \"27544060\",\n          \"26771872\",\n          \"25891174\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"start_offset\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 298,\n        \"samples\": [\n          \"1180\",\n          \"1639\",\n          \"197\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"end_offset\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 297,\n        \"samples\": [\n          \"359\",\n          \"345\",\n          \"53\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"entity_mention\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 47,\n        \"samples\": [\n          \"FGFR1\",\n          \"EXP2\",\n          \"Anaplastic lymphoma kinase\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"entity_type\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 1,\n        \"samples\": [\n          \"Gene\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"entity_id\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 36,\n        \"samples\": [\n          \"2064\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"
            }
          },
          "metadata": {},
          "execution_count": 53
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "gene_counts = gene['entity_mention'].value_counts(normalize=False).sort_values(ascending=False)\n",
        "print(\"Unique Values and Counts Genes (Descending):\")\n",
        "print(gene_counts)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "NhKb5MXBR5oW",
        "outputId": "d5eab1cc-d4e2-4ad3-be7a-a29eabebed92"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Unique Values and Counts Genes (Descending):\n",
            "entity_mention\n",
            "ALK                                         54\n",
            "EGFR                                        49\n",
            "KRAS                                        35\n",
            "PD-L1                                       22\n",
            "MET                                         18\n",
            "RET                                         15\n",
            "LKB1                                        13\n",
            "mTOR                                        12\n",
            "HER2                                        10\n",
            "TP53                                        10\n",
            "ABL1                                        10\n",
            "CD8                                          8\n",
            "PIM1                                         8\n",
            "ROS1                                         6\n",
            "BRAF                                         6\n",
            "FGFR1                                        4\n",
            "anaplastic lymphoma kinase                   3\n",
            "EXP1                                         3\n",
            "c-MYC                                        3\n",
            "STAT3                                        3\n",
            "AKT                                          2\n",
            "EXP4                                         2\n",
            "EXP2                                         2\n",
            "EXP1-6                                       2\n",
            "epidermal growth factor receptor             2\n",
            "EXP4-5                                       2\n",
            "FGFR                                         1\n",
            "PD-1 ligand 1                                1\n",
            "4EBP1                                        1\n",
            "S6K                                          1\n",
            "Epidermal Growth Factor Receptor             1\n",
            "ABL                                          1\n",
            "programmed cell death-ligand 1               1\n",
            "PD-1                                         1\n",
            "Anaplastic lymphoma kinase                   1\n",
            "Fibroblast growth factor receptor 1          1\n",
            "MEK1/2                                       1\n",
            "MEK                                          1\n",
            "programmed cell death 1                      1\n",
            "Odd                                          1\n",
            "Nov 2                                        1\n",
            "programmed death-ligand 1                    1\n",
            "EXP6                                         1\n",
            "EXP1-5                                       1\n",
            "EXP5                                         1\n",
            "EXP2-5                                       1\n",
            "Human epidermal growth factor receptor 2     1\n",
            "Name: count, dtype: int64\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "**- Top 5 genes are:** <br>\n",
        "1) ALK <br>\n",
        "2) EGFR <br>\n",
        "3) KRAS <br>\n",
        "4) PD-L1 <br>\n",
        "5) MET <br>"
      ],
      "metadata": {
        "id": "2WljfhpxSDnQ"
      }
    }
  ]
}